<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Nina Yifan Y." />

<meta name="date" content="2024-10-04" />

<title>cTWAS-European Lung+Liver+Lymphocytes//eQTL+sQTL</title>

<script src="site_libs/header-attrs-2.28/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Multigroup_cTWAS</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/yy896/Multigroup_cTWAS">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">cTWAS-European
Lung+Liver+Lymphocytes//eQTL+sQTL</h1>
<h4 class="author">Nina Yifan Y.</h4>
<h4 class="date">2024-10-04</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-10-04
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Multigroup_cTWAS/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20241004code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20241004)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20241004code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20241004)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomyy896MultigroupcTWAStree6cba1b353d61ea8aef119e174c1c376eb728a20atargetblank6cba1b3a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/yy896/Multigroup_cTWAS/tree/6cba1b353d61ea8aef119e174c1c376eb728a20a" target="_blank">6cba1b3</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomyy896MultigroupcTWAStree6cba1b353d61ea8aef119e174c1c376eb728a20atargetblank6cba1b3a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/yy896/Multigroup_cTWAS/tree/6cba1b353d61ea8aef119e174c1c376eb728a20a" target="_blank">6cba1b3</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory

Untracked files:
    Untracked:  analysis/cTWAS_noLD_european_lymphocytes_output.Rmd
    Untracked:  analysis/cTWAS_noLD_european_prostatetestis_output.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown
(<code>analysis/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd</code>)
and HTML
(<code>docs/cTWAS_noLD_european_lungliverlymphocytes_output.html</code>)
files. If you’ve configured a remote Git repository (see
<code>?wflow_git_remote</code>), click on the hyperlinks in the table
below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cba1b353d61ea8aef119e174c1c376eb728a20a/analysis/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd" target="_blank">6cba1b3</a>
</td>
<td>
yy896
</td>
<td>
2024-10-04
</td>
<td>
Add my first analysis
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="introduction" class="section level2">
<h2>Introduction</h2>
</div>
<div id="data-summary" class="section level1">
<h1>Data Summary</h1>
<div id="group-size" class="section level3">
<h3>Group Size</h3>
<table>
<colgroup>
<col width="7%" />
<col width="13%" />
<col width="12%" />
<col width="14%" />
<col width="12%" />
<col width="18%" />
<col width="16%" />
<col width="5%" />
</colgroup>
<thead>
<tr class="header">
<th align="left"></th>
<th align="left">V1</th>
<th align="left">V2</th>
<th align="left">V3</th>
<th align="left">V4</th>
<th align="left">V5</th>
<th align="left">V6</th>
<th align="left">V7</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">group</td>
<td align="left">lung|expression</td>
<td align="left">lung|splicing</td>
<td align="left">liver|expression</td>
<td align="left">liver|splicing</td>
<td align="left">lymphocytes|expression</td>
<td align="left">lymphocytes|splicing</td>
<td align="left">SNP</td>
</tr>
<tr class="even">
<td align="left">group_size</td>
<td align="left">10630</td>
<td align="left">32464</td>
<td align="left">8721</td>
<td align="left">19025</td>
<td align="left">8478</td>
<td align="left">26609</td>
<td align="left">7146350</td>
</tr>
</tbody>
</table>
</div>
<div id="piechart-with-att_pve-on" class="section level3">
<h3>Piechart with att_pve on</h3>
<p><img src="figure/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd/Piechart-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="combined-pips" class="section level3">
<h3>Combined PIPs</h3>
<pre><code>2024-10-04 17:55:33.08414 INFO::Limit gene results to credible sets</code></pre>
<table style="width:100%;">
<colgroup>
<col width="5%" />
<col width="13%" />
<col width="12%" />
<col width="14%" />
<col width="13%" />
<col width="17%" />
<col width="16%" />
<col width="7%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">gene_name</th>
<th align="right">lung|expression_pip</th>
<th align="right">lung|splicing_pip</th>
<th align="right">liver|expression_pip</th>
<th align="right">liver|splicing_pip</th>
<th align="right">lymphocytes|expression_pip</th>
<th align="right">lymphocytes|splicing_pip</th>
<th align="right">combined_pip</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">BIK</td>
<td align="right">1.0000000</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">1.0000000</td>
</tr>
<tr class="even">
<td align="left">MSMB</td>
<td align="right">1.0000000</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">1.0000000</td>
</tr>
<tr class="odd">
<td align="left">MMP7</td>
<td align="right">0.7169050</td>
<td align="right">NA</td>
<td align="right">0.2825474</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.9994524</td>
</tr>
<tr class="even">
<td align="left">ZDHHC7</td>
<td align="right">0.9987771</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.9987771</td>
</tr>
<tr class="odd">
<td align="left">PPP1R14A</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.9977660</td>
<td align="right">NA</td>
<td align="right">0.9977660</td>
</tr>
<tr class="even">
<td align="left">MLPH</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.9962567</td>
<td align="right">NA</td>
<td align="right">0.9962567</td>
</tr>
<tr class="odd">
<td align="left">RAB13</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.9936266</td>
<td align="right">NA</td>
<td align="right">0.9936266</td>
</tr>
<tr class="even">
<td align="left">PDLIM5</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.9927937</td>
<td align="right">0.9927937</td>
</tr>
<tr class="odd">
<td align="left">MBOAT7</td>
<td align="right">0.9924853</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.9924853</td>
</tr>
<tr class="even">
<td align="left">PIK3C2B</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.9904477</td>
<td align="right">NA</td>
<td align="right">0.9904477</td>
</tr>
<tr class="odd">
<td align="left">CCDC88C</td>
<td align="right">NA</td>
<td align="right">0.2928383</td>
<td align="right">NA</td>
<td align="right">0.3035177</td>
<td align="right">NA</td>
<td align="right">0.3849813</td>
<td align="right">0.9813373</td>
</tr>
<tr class="even">
<td align="left">JADE2</td>
<td align="right">0.9579950</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.9579950</td>
</tr>
<tr class="odd">
<td align="left">VAMP5</td>
<td align="right">NA</td>
<td align="right">0.9497607</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.9497607</td>
</tr>
<tr class="even">
<td align="left">PXMP4</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.9183020</td>
<td align="right">NA</td>
<td align="right">0.9183020</td>
</tr>
<tr class="odd">
<td align="left">TACC1</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.9167993</td>
<td align="right">0.9167993</td>
</tr>
<tr class="even">
<td align="left">ARNT2</td>
<td align="right">0.9117966</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.9117966</td>
</tr>
<tr class="odd">
<td align="left">RNF217</td>
<td align="right">NA</td>
<td align="right">0.9071159</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.9071159</td>
</tr>
<tr class="even">
<td align="left">TOR1A</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.0197818</td>
<td align="right">0.7995323</td>
<td align="right">0.0821688</td>
<td align="right">0.9014829</td>
</tr>
<tr class="odd">
<td align="left">SIVA1</td>
<td align="right">0.0592498</td>
<td align="right">0.0678977</td>
<td align="right">NA</td>
<td align="right">0.0355445</td>
<td align="right">0.0798153</td>
<td align="right">0.6508764</td>
<td align="right">0.8933837</td>
</tr>
<tr class="even">
<td align="left">TBX1</td>
<td align="right">NA</td>
<td align="right">0.8871766</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.8871766</td>
</tr>
<tr class="odd">
<td align="left">ZBTB38</td>
<td align="right">NA</td>
<td align="right">0.2990790</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.5840135</td>
<td align="right">0.8830925</td>
</tr>
<tr class="even">
<td align="left">CASP8</td>
<td align="right">NA</td>
<td align="right">0.2871709</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.5942137</td>
<td align="right">0.8813845</td>
</tr>
<tr class="odd">
<td align="left">MYO5C</td>
<td align="right">0.0339134</td>
<td align="right">NA</td>
<td align="right">0.1277896</td>
<td align="right">NA</td>
<td align="right">0.7127089</td>
<td align="right">NA</td>
<td align="right">0.8744119</td>
</tr>
<tr class="even">
<td align="left">SOD2</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.8687264</td>
<td align="right">0.8687264</td>
</tr>
<tr class="odd">
<td align="left">IKZF2</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.8571084</td>
<td align="right">0.0077898</td>
<td align="right">0.8648983</td>
</tr>
<tr class="even">
<td align="left">CHMP4C</td>
<td align="right">0.2146880</td>
<td align="right">0.2422590</td>
<td align="right">0.1368719</td>
<td align="right">0.2413628</td>
<td align="right">0.0184096</td>
<td align="right">NA</td>
<td align="right">0.8535913</td>
</tr>
<tr class="odd">
<td align="left">L3MBTL3</td>
<td align="right">0.8099578</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">NA</td>
<td align="right">0.8099578</td>
</tr>
</tbody>
</table>
</div>
<div id="scatter-plot" class="section level3">
<h3>Scatter Plot</h3>
<p><img src="figure/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd/unnamed-chunk-3-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="ctwas-output" class="section level1">
<h1>cTWAS output</h1>
<pre class="r"><code>make_convergence_plots(param, gwas_n)</code></pre>
<p><img src="figure/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd/cTWAS%20output-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ctwas_parameters &lt;- summarize_param(param, gwas_n)
ctwas_parameters</code></pre>
<pre><code>$group_size
       lung|expression          lung|splicing       liver|expression 
                 10630                  32464                   8721 
        liver|splicing lymphocytes|expression   lymphocytes|splicing 
                 19025                   8478                  26609 
                   SNP 
               7146350 

$group_prior
       lung|expression          lung|splicing       liver|expression 
          0.0029718106           0.0034900103           0.0018946444 
        liver|splicing lymphocytes|expression   lymphocytes|splicing 
          0.0034770994           0.0087024176           0.0072215260 
                   SNP 
          0.0001646342 

$group_prior_var
       lung|expression          lung|splicing       liver|expression 
              43.84128               10.62075               43.84128 
        liver|splicing lymphocytes|expression   lymphocytes|splicing 
              10.62075               43.84128               10.62075 
                   SNP 
              16.75032 

$enrichment
       lung|expression          lung|splicing       liver|expression 
              18.05099               21.19858               11.50821 
        liver|splicing lymphocytes|expression   lymphocytes|splicing 
              21.12015               52.85912               43.86407 

$group_pve
       lung|expression          lung|splicing       liver|expression 
          0.0019054870           0.0016555873           0.0009966567 
        liver|splicing lymphocytes|expression   lymphocytes|splicing 
          0.0009666408           0.0044502553           0.0028078950 
                   SNP 
          0.0271141431 

$total_pve
[1] 0.03989667

$attributable_pve
       lung|expression          lung|splicing       liver|expression 
            0.04776056             0.04149688             0.02498095 
        liver|splicing lymphocytes|expression   lymphocytes|splicing 
            0.02422861             0.11154454             0.07037919 
                   SNP 
            0.67960926 </code></pre>
</div>
<div id="locus-plot-from-highest-pip-to-lowest" class="section level1">
<h1>Locus plot (from highest PIP to lowest)</h1>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;22_42294812_43318194&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:55:34.136801 INFO::Limit to protein coding genes
2024-10-04 17:55:34.141293 INFO::focal id: ENSG00000100290.2|lung_expression
2024-10-04 17:55:34.141581 INFO::focal molecular trait: BIK lung expression
2024-10-04 17:55:34.141889 INFO::Range of locus: chr22:42295132-43317762</code></pre>
<pre><code>2024-10-04 17:55:34.239708 INFO::focal molecular trait QTL positions: 43104206,43110629
2024-10-04 17:55:34.254348 INFO::Limit PIPs to credible sets</code></pre>
<p><img src="figure/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd/locus%20plot-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;10_44234627_46978910&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:55:34.9891 INFO::Limit to protein coding genes
2024-10-04 17:55:34.989749 INFO::focal id: ENSG00000263639.5|lung_expression
2024-10-04 17:55:34.99001 INFO::focal molecular trait: MSMB lung expression
2024-10-04 17:55:34.990308 INFO::Range of locus: chr10:44234762-46910781</code></pre>
<pre><code>2024-10-04 17:55:35.08859 INFO::focal molecular trait QTL positions: 46046326
2024-10-04 17:55:35.102286 INFO::Limit PIPs to credible sets</code></pre>
<p><img src="figure/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd/locus%20plot-2.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;16_84412277_85115079&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:55:35.821034 INFO::Limit to protein coding genes
2024-10-04 17:55:35.821613 INFO::focal id: ENSG00000153786.12|lung_expression
2024-10-04 17:55:35.82186 INFO::focal molecular trait: ZDHHC7 lung expression
2024-10-04 17:55:35.822128 INFO::Range of locus: chr16:84412178-85114834</code></pre>
<pre><code>2024-10-04 17:55:35.869185 INFO::focal molecular trait QTL positions: 84988377,85011068,85011497,85011520
2024-10-04 17:55:35.882545 INFO::Limit PIPs to credible sets</code></pre>
<p><img src="figure/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd/locus%20plot-3.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;19_37036131_39679979&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:55:36.581276 INFO::Limit to protein coding genes
2024-10-04 17:55:36.581982 INFO::focal id: ENSG00000167641.10|lymphocytes_expression
2024-10-04 17:55:36.582274 INFO::focal molecular trait: PPP1R14A lymphocytes expression
2024-10-04 17:55:36.582574 INFO::Range of locus: chr19:37036031-39682468</code></pre>
<pre><code>2024-10-04 17:55:36.652281 INFO::focal molecular trait QTL positions: 38256477
2024-10-04 17:55:36.665245 INFO::Limit PIPs to credible sets</code></pre>
<p><img src="figure/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd/locus%20plot-4.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;2_236540389_237852338&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:55:37.874795 INFO::Limit to protein coding genes
2024-10-04 17:55:37.875413 INFO::focal id: ENSG00000115648.13|lymphocytes_expression
2024-10-04 17:55:37.875672 INFO::focal molecular trait: MLPH lymphocytes expression
2024-10-04 17:55:37.87596 INFO::Range of locus: chr2:236541810-237868276</code></pre>
<pre><code>2024-10-04 17:55:37.929244 INFO::focal molecular trait QTL positions: 237484645,237484791
2024-10-04 17:55:37.942579 INFO::Limit PIPs to credible sets</code></pre>
<p><img src="figure/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd/locus%20plot-5.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;1_153208353_154797927&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:55:38.61419 INFO::Limit to protein coding genes
2024-10-04 17:55:38.61473 INFO::focal id: ENSG00000143545.8|lymphocytes_expression
2024-10-04 17:55:38.614975 INFO::focal molecular trait: RAB13 lymphocytes expression
2024-10-04 17:55:38.615246 INFO::Range of locus: chr1:153208786-154797763</code></pre>
<pre><code>2024-10-04 17:55:38.67695 INFO::focal molecular trait QTL positions: 153985724,153989308
2024-10-04 17:55:38.689915 INFO::Limit PIPs to credible sets</code></pre>
<p><img src="figure/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd/locus%20plot-6.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;19_54099074_55216712&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:55:39.566953 INFO::Limit to protein coding genes
2024-10-04 17:55:39.567597 INFO::focal id: ENSG00000125505.16|lung_expression
2024-10-04 17:55:39.567859 INFO::focal molecular trait: MBOAT7 lung expression
2024-10-04 17:55:39.568128 INFO::Range of locus: chr19:54098580-55244488</code></pre>
<pre><code>2024-10-04 17:55:39.629755 INFO::focal molecular trait QTL positions: 54189591,54190017
2024-10-04 17:55:39.642808 INFO::Limit PIPs to credible sets</code></pre>
<p><img src="figure/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd/locus%20plot-7.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;1_203365606_204711940&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:55:40.555343 INFO::Limit to protein coding genes
2024-10-04 17:55:40.555935 INFO::focal id: ENSG00000133056.13|lymphocytes_expression
2024-10-04 17:55:40.556181 INFO::focal molecular trait: PIK3C2B lymphocytes expression
2024-10-04 17:55:40.556444 INFO::Range of locus: chr1:203181224-204711982</code></pre>
<pre><code>2024-10-04 17:55:40.613321 INFO::focal molecular trait QTL positions: 204495053
2024-10-04 17:55:40.626138 INFO::Limit PIPs to credible sets</code></pre>
<p><img src="figure/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd/locus%20plot-8.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;5_132803957_135441711&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:55:41.201778 INFO::Limit to protein coding genes
2024-10-04 17:55:41.202485 INFO::focal id: ENSG00000043143.20|lung_expression
2024-10-04 17:55:41.202774 INFO::focal molecular trait: JADE2 lung expression
2024-10-04 17:55:41.203075 INFO::Range of locus: chr5:132778929-135441452</code></pre>
<pre><code>2024-10-04 17:55:41.258979 INFO::focal molecular trait QTL positions: 134524410
2024-10-04 17:55:41.272334 INFO::Limit PIPs to credible sets</code></pre>
<p><img src="figure/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd/locus%20plot-9.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;20_33027017_34225635&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:55:42.195362 INFO::Limit to protein coding genes
2024-10-04 17:55:42.195925 INFO::focal id: ENSG00000101417.11|lymphocytes_expression
2024-10-04 17:55:42.196184 INFO::focal molecular trait: PXMP4 lymphocytes expression
2024-10-04 17:55:42.196447 INFO::Range of locus: chr20:33027532-34224952</code></pre>
<pre><code>2024-10-04 17:55:42.244182 INFO::focal molecular trait QTL positions: 33720469
2024-10-04 17:55:42.257774 INFO::Limit PIPs to credible sets</code></pre>
<p><img src="figure/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd/locus%20plot-10.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;15_78223711_80568637&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:55:42.939446 INFO::Limit to protein coding genes
2024-10-04 17:55:42.940106 INFO::focal id: ENSG00000172379.20|lung_expression
2024-10-04 17:55:42.94039 INFO::focal molecular trait: ARNT2 lung expression
2024-10-04 17:55:42.94069 INFO::Range of locus: chr15:78206478-80568240</code></pre>
<pre><code>2024-10-04 17:55:42.994704 INFO::focal molecular trait QTL positions: 80404427
2024-10-04 17:55:43.007838 INFO::Limit PIPs to credible sets</code></pre>
<p><img src="figure/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd/locus%20plot-11.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;8_37521871_39081287&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:55:43.844256 INFO::Limit to protein coding genes
2024-10-04 17:55:43.844822 INFO::focal id: intron_8_38787743_38788704|lymphocytes_splicing
2024-10-04 17:55:43.84506 INFO::focal molecular trait: TACC1 lymphocytes splicing
2024-10-04 17:55:43.845322 INFO::Range of locus: chr8:37522177-39081209</code></pre>
<pre><code>2024-10-04 17:55:43.899462 INFO::focal molecular trait QTL positions: 38787396
2024-10-04 17:55:43.91293 INFO::Limit PIPs to credible sets</code></pre>
<p><img src="figure/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd/locus%20plot-12.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;6_123535036_125103237&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:55:44.544207 INFO::Limit to protein coding genes
2024-10-04 17:55:44.54482 INFO::focal id: intron_6_124971598_125045211|lung_splicing
2024-10-04 17:55:44.545085 INFO::focal molecular trait: RNF217 lung splicing
2024-10-04 17:55:44.545368 INFO::Range of locus: chr6:123426616-125103086</code></pre>
<pre><code>2024-10-04 17:55:44.718065 INFO::focal molecular trait QTL positions: 124972588
2024-10-04 17:55:44.731163 INFO::Limit PIPs to credible sets</code></pre>
<p><img src="figure/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd/locus%20plot-13.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;22_17813322_19924835&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:55:45.168258 INFO::Limit to protein coding genes
2024-10-04 17:55:45.168897 INFO::focal id: intron_22_19761280_19763241|lung_splicing
2024-10-04 17:55:45.169174 INFO::focal molecular trait: TBX1 lung splicing
2024-10-04 17:55:45.169467 INFO::Range of locus: chr22:17813372-19925414</code></pre>
<pre><code>2024-10-04 17:55:45.22329 INFO::focal molecular trait QTL positions: 19762002
2024-10-04 17:55:45.236901 INFO::Limit PIPs to credible sets</code></pre>
<p><img src="figure/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd/locus%20plot-14.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;2_213149558_214709071&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:55:45.969076 INFO::Limit to protein coding genes
2024-10-04 17:55:45.969641 INFO::focal id: ENSG00000030419.16|lymphocytes_expression
2024-10-04 17:55:45.969892 INFO::focal molecular trait: IKZF2 lymphocytes expression
2024-10-04 17:55:45.970171 INFO::Range of locus: chr2:213075959-215420129</code></pre>
<pre><code>2024-10-04 17:55:46.028119 INFO::focal molecular trait QTL positions: 213153600
2024-10-04 17:55:46.041754 INFO::Limit PIPs to credible sets</code></pre>
<p><img src="figure/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd/locus%20plot-15.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;6_129539886_130614309&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:55:46.4944 INFO::Limit to protein coding genes
2024-10-04 17:55:46.495028 INFO::focal id: ENSG00000198945.7|lung_expression
2024-10-04 17:55:46.495288 INFO::focal molecular trait: L3MBTL3 lung expression
2024-10-04 17:55:46.495566 INFO::Range of locus: chr6:129539786-130613726</code></pre>
<pre><code>2024-10-04 17:55:46.668157 INFO::focal molecular trait QTL positions: 130027974
2024-10-04 17:55:46.681189 INFO::Limit PIPs to credible sets</code></pre>
<p><img src="figure/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd/locus%20plot-16.png" width="960" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.1 (2024-06-14)
Platform: aarch64-apple-darwin20
Running under: macOS 15.0.1

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: America/New_York
tzcode source: internal

attached base packages:
[1] stats4    stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] data.table_1.16.0         lubridate_1.9.3          
 [3] forcats_1.0.0             stringr_1.5.1            
 [5] dplyr_1.1.4               purrr_1.0.2              
 [7] tidyr_1.3.1               tibble_3.2.1             
 [9] tidyverse_2.0.0           ggrepel_0.9.6            
[11] ggplot2_3.5.1             EnsDb.Hsapiens.v86_2.99.0
[13] ensembldb_2.28.1          AnnotationFilter_1.28.0  
[15] GenomicFeatures_1.56.0    AnnotationDbi_1.66.0     
[17] Biobase_2.64.0            GenomicRanges_1.56.1     
[19] GenomeInfoDb_1.40.1       IRanges_2.38.1           
[21] S4Vectors_0.42.1          BiocGenerics_0.50.0      
[23] ctwas_0.4.14              readr_2.1.5              
[25] workflowr_1.7.1          

loaded via a namespace (and not attached):
  [1] RColorBrewer_1.1-3          rstudioapi_0.16.0          
  [3] jsonlite_1.8.9              magrittr_2.0.3             
  [5] farver_2.1.2                rmarkdown_2.28             
  [7] fs_1.6.4                    BiocIO_1.14.0              
  [9] zlibbioc_1.50.0             vctrs_0.6.5                
 [11] memoise_2.0.1               Rsamtools_2.20.0           
 [13] RCurl_1.98-1.16             mixsqp_0.3-54              
 [15] htmltools_0.5.8.1           S4Arrays_1.4.1             
 [17] curl_5.2.3                  SparseArray_1.4.8          
 [19] sass_0.4.9                  bslib_0.8.0                
 [21] htmlwidgets_1.6.4           plotly_4.10.4              
 [23] zoo_1.8-12                  cachem_1.1.0               
 [25] gggrid_0.2-0                GenomicAlignments_1.40.0   
 [27] whisker_0.4.1               lifecycle_1.0.4            
 [29] pkgconfig_2.0.3             Matrix_1.7-0               
 [31] R6_2.5.1                    fastmap_1.2.0              
 [33] GenomeInfoDbData_1.2.12     MatrixGenerics_1.16.0      
 [35] digest_0.6.37               colorspace_2.1-1           
 [37] ps_1.8.0                    rprojroot_2.0.4            
 [39] irlba_2.3.5.1               RSQLite_2.3.7              
 [41] labeling_0.4.3              pgenlibr_0.3.7             
 [43] timechange_0.3.0            fansi_1.0.6                
 [45] httr_1.4.7                  abind_1.4-8                
 [47] compiler_4.4.1              bit64_4.0.5                
 [49] withr_3.0.1                 BiocParallel_1.38.0        
 [51] DBI_1.2.3                   highr_0.11                 
 [53] DelayedArray_0.30.1         rjson_0.2.23               
 [55] formattable_0.2.1           tools_4.4.1                
 [57] httpuv_1.6.15               glue_1.7.0                 
 [59] restfulr_0.0.15             callr_3.7.6                
 [61] promises_1.3.0              grid_4.4.1                 
 [63] getPass_0.2-4               generics_0.1.3             
 [65] LDlinkR_1.4.0               gtable_0.3.5               
 [67] tzdb_0.4.0                  locuszoomr_0.3.5           
 [69] hms_1.1.3                   utf8_1.2.4                 
 [71] XVector_0.44.0              pillar_1.9.0               
 [73] later_1.3.2                 logging_0.10-108           
 [75] lattice_0.22-6              rtracklayer_1.64.0         
 [77] bit_4.5.0                   tidyselect_1.2.1           
 [79] Biostrings_2.72.1           knitr_1.48                 
 [81] git2r_0.33.0.9000           ProtGenerics_1.36.0        
 [83] SummarizedExperiment_1.34.0 xfun_0.47                  
 [85] matrixStats_1.4.1           stringi_1.8.4              
 [87] UCSC.utils_1.0.0            lazyeval_0.2.2             
 [89] yaml_2.3.10                 evaluate_1.0.0             
 [91] codetools_0.2-20            cli_3.6.3                  
 [93] munsell_0.5.1               processx_3.8.4             
 [95] jquerylib_0.1.4             Rcpp_1.0.13                
 [97] png_0.1-8                   XML_3.99-0.17              
 [99] parallel_4.4.1              blob_1.2.4                 
[101] bitops_1.0-8                viridisLite_0.4.2          
[103] scales_1.3.0                crayon_1.5.3               
[105] rlang_1.1.4                 cowplot_1.1.3              
[107] KEGGREST_1.44.1            </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
