<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Nina Yifan Yang" />

<meta name="date" content="2024-11-03" />

<title>Prostate Cancer ~ Predictdb eQTL&amp;sQTL (Prostate,Testis,Lymphocytes) &amp; RefZ prostate cohort &amp; TCGA prostate tumor cohort</title>

<script src="site_libs/header-attrs-2.28/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.33/datatables.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Multigroup_cTWAS</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/yy896/Multigroup_cTWAS">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">Prostate Cancer ~ Predictdb eQTL&amp;sQTL
(Prostate,Testis,Lymphocytes) &amp; RefZ prostate cohort &amp; TCGA
prostate tumor cohort</h1>
<h4 class="author">Nina Yifan Yang</h4>
<h4 class="date">2024-11-03</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-11-03
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Multigroup_cTWAS/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20241004code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20241004)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20241004code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20241004)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomyy896MultigroupcTWAStree6c7cf9df8d3916c6f29d86e893f8cf8479f6f772targetblank6c7cf9da">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/yy896/Multigroup_cTWAS/tree/6c7cf9df8d3916c6f29d86e893f8cf8479f6f772" target="_blank">6c7cf9d</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomyy896MultigroupcTWAStree6c7cf9df8d3916c6f29d86e893f8cf8479f6f772targetblank6c7cf9da"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/yy896/Multigroup_cTWAS/tree/6c7cf9df8d3916c6f29d86e893f8cf8479f6f772" target="_blank">6c7cf9d</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    analysis/.DS_Store
    Ignored:    analysis/.Rhistory

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown
(<code>analysis/cTWAS_LD_european_combined3_output.Rmd</code>) and HTML
(<code>docs/cTWAS_LD_european_combined3_output.html</code>) files. If
you’ve configured a remote Git repository (see
<code>?wflow_git_remote</code>), click on the hyperlinks in the table
below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6c7cf9df8d3916c6f29d86e893f8cf8479f6f772/analysis/cTWAS_LD_european_combined3_output.Rmd" target="_blank">6c7cf9d</a>
</td>
<td>
yy896
</td>
<td>
2024-11-03
</td>
<td>
UPDATED RESULTS
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/yy896/Multigroup_cTWAS/075e972669362b36a4f002a7cb77a7efed609c0a/docs/cTWAS_LD_european_combined3_output.html" target="_blank">075e972</a>
</td>
<td>
yy896
</td>
<td>
2024-10-31
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/4c2b617fc0ebf92e5a16eaaf495d6266d7d9880d/analysis/cTWAS_LD_european_combined3_output.Rmd" target="_blank">4c2b617</a>
</td>
<td>
yy896
</td>
<td>
2024-10-31
</td>
<td>
UPDATED RESULTS
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/yy896/Multigroup_cTWAS/099102dbbcddba5380e899c2d37db5ecfc27e20d/docs/cTWAS_LD_european_combined3_output.html" target="_blank">099102d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/80c479265aee7359be435f163de964ca698647b0/analysis/cTWAS_LD_european_combined3_output.Rmd" target="_blank">80c4792</a>
</td>
<td>
yy896
</td>
<td>
2024-10-30
</td>
<td>
UPDATED RESULTS
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p>Here presenting a cTWAS results report based on real data analysis.
The analyzed trait is prostate cancer, the prediction models are
Prostate &amp; Testis &amp; Lymphocytes gene expression and splicing
models trained on GTEx v8 in the PredictDB format, and RefZ FUSION
weights expression data under NIH accession number phs000985.v2.p1, and
TCGA FUSION weights expression data derived from mRNA-seq.</p>
<div id="analysis-settings" class="section level2">
<h2>Analysis Settings</h2>
<div id="input-data" class="section level3">
<h3>Input Data</h3>
<ul>
<li>GWAS Z-scores</li>
</ul>
<p>The summary statistics for prostate cancer are downloaded from <a
href="https://www.ebi.ac.uk/gwas/downloads/summary-statistics"
class="uri">https://www.ebi.ac.uk/gwas/downloads/summary-statistics</a>,
using dataset ID: GCST90274714.</p>
<p>The sample size is</p>
<pre><code>[1] &quot;gwas_n = 726828&quot;</code></pre>
<ul>
<li>Prediction models</li>
</ul>
<p>The prediction models used in this analysis are prostate &amp; testis
&amp; lymphocytes gene expression and splicing models, trained on GTEx
v8 in the PredictDB format. These models can be downloaded from <a
href="https://predictdb.org/post/2021/07/21/gtex-v8-models-on-eqtl-and-sqtl/"
class="uri">https://predictdb.org/post/2021/07/21/gtex-v8-models-on-eqtl-and-sqtl/</a>.
The RefZ weights model can be downloaded from <a
href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000985.v2.p1"
class="uri">https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000985.v2.p1</a>.
The TCGA weights model can be found here. <a
href="http://gusevlab.org/projects/fusion/"
class="uri">http://gusevlab.org/projects/fusion/</a></p>
<ul>
<li>Reference data</li>
</ul>
<p>We did include ukb38 whole genome LD reference in the analysis.</p>
</div>
<div id="data-processing-and-harmonization" class="section level3">
<h3>Data processing and harmonization</h3>
<p>We map the reference SNPs to regions following the instructions from
the cTWAS tutorial.</p>
<p>When processing z-scores, we exclude multi-allelic and
strand-ambiguous variants by setting
<code>drop_multiallelic = TRUE</code> and
<code>drop_strand_ambig = TRUE</code>.</p>
<p>The process can be divided into steps below, users can expand the
code snippets below to view the exact code used.</p>
<ul>
<li>Input and output settings</li>
</ul>
<pre class="r"><code>weight_files &lt;- c(&quot;/project2/xinhe/shared_data/multigroup_ctwas/weights/expression_models/expression_Prostate.db&quot;,
                  &quot;/project2/xinhe/shared_data/multigroup_ctwas/weights/splicing_models/splicing_Prostate.db&quot;,
                  &quot;/project2/xinhe/shared_data/multigroup_ctwas/weights/expression_models/expression_Testis.db&quot;,
                  &quot;/project2/xinhe/shared_data/multigroup_ctwas/weights/splicing_models/splicing_Testis.db&quot;,
                  &quot;/project2/xinhe/shared_data/multigroup_ctwas/weights/expression_models/expression_Cells_EBV-transformed_lymphocytes.db&quot;,
                  &quot;/project2/xinhe/shared_data/multigroup_ctwas/weights/splicing_models/splicing_Cells_EBV-transformed_lymphocytes.db&quot;)

z_snp &lt;- vroom::vroom(&quot;/project2/xinhe/yifany/GCST90274714.mod.tsv.gz&quot;, col_names = TRUE, show_col_types = FALSE)
z_snp$Z &lt;- z_snp$beta / z_snp$standard_error

# gwas_n &lt;- as.numeric(names(sort(table(z_snp$base_pair_location), decreasing = TRUE)[1]))
# cat(&quot;gwas_n =&quot;, gwas_n, &quot;\n&quot;)
gwas_n &lt;- 726828

z_snp &lt;- z_snp[, c(&quot;rsid&quot;, &quot;other_allele&quot;, &quot;effect_allele&quot;, &quot;Z&quot;)]
colnames(z_snp) &lt;- c(&quot;id&quot;, &quot;A1&quot;, &quot;A2&quot;, &quot;z&quot;)

region_file &lt;- system.file(&quot;extdata/ldetect&quot;, &quot;EUR.b38.ldetect.regions.RDS&quot;, package = &quot;ctwas&quot;)
region_info &lt;- readRDS(region_file)

# output dir
outputdir &lt;- &quot;/project/xinhe/yifany/multigroup_cTWAS&quot;
dir.create(outputdir, showWarnings=F, recursive=T)

# Number of cores
ncore &lt;- 5</code></pre>
<ul>
<li>Preprocessing GWAS</li>
</ul>
<pre class="r"><code>### Preprocess SNP_map and LD_map
genome_version &lt;- &quot;b38&quot;
region_file &lt;- system.file(&quot;extdata/ldetect&quot;, paste0(&quot;EUR.&quot;, genome_version, &quot;.ldetect.regions.RDS&quot;), package = &quot;ctwas&quot;)
region_info &lt;- readRDS(region_file)

LD_dir &lt;- &quot;/project2/mstephens/wcrouse/UKB_LDR_0.1&quot;

LD_filestem &lt;- sprintf(&quot;ukb_%s_0.1_chr%s.R_snp.%s_%s&quot;, genome_version, region_info$chrom, region_info$start, region_info$stop)

region_metatable &lt;- region_info
region_metatable$LD_file &lt;- file.path(LD_dir, paste0(LD_filestem, &quot;.RDS&quot;))
region_metatable$SNP_file &lt;- file.path(LD_dir, paste0(LD_filestem, &quot;.Rvar&quot;))

res &lt;- create_snp_LD_map(region_metatable)
region_info &lt;- res$region_info
snp_map &lt;- res$snp_map
LD_map &lt;- res$LD_map

z_snp &lt;- preprocess_z_snp(z_snp, snp_map, 
                          drop_multiallelic = TRUE, 
                          drop_strand_ambig = TRUE)</code></pre>
<ul>
<li>Preprocessing weights</li>
</ul>
<pre class="r"><code>weights_prostate_expression &lt;- preprocess_weights(weight_files[1],
                                                  region_info,
                                                  gwas_snp_ids = z_snp$id,
                                                  snp_map = snp_map,
                                                  type = &quot;eQTL&quot;,
                                                  context = &quot;prostate&quot;,
                                                  weight_format = &quot;PredictDB&quot;,
                                                  drop_strand_ambig = TRUE,
                                                  scale_predictdb_weights = TRUE,
                                                  load_predictdb_LD = TRUE,
                                                  filter_protein_coding_genes = TRUE,
                                                  ncore = 5)

weights_prostate_splicing &lt;- preprocess_weights(weight_files[2],
                                                region_info,
                                                gwas_snp_ids = z_snp$id,
                                                snp_map = snp_map,
                                                type = &quot;sQTL&quot;,
                                                context = &quot;prostate&quot;,
                                                weight_format = &quot;PredictDB&quot;,
                                                drop_strand_ambig = TRUE,
                                                scale_predictdb_weights = TRUE,
                                                load_predictdb_LD = TRUE,
                                                filter_protein_coding_genes = TRUE,
                                                ncore = 5)

weights_testis_expression &lt;- preprocess_weights(weight_files[3],
                                                region_info,
                                                gwas_snp_ids = z_snp$id,
                                                snp_map = snp_map,
                                                type = &quot;eQTL&quot;,
                                                context = &quot;testis&quot;,
                                                weight_format = &quot;PredictDB&quot;,
                                                drop_strand_ambig = TRUE,
                                                scale_predictdb_weights = TRUE,
                                                load_predictdb_LD = TRUE,
                                                filter_protein_coding_genes = TRUE,
                                                ncore = 5)

weights_testis_splicing &lt;- preprocess_weights(weight_files[4],
                                              region_info,
                                              gwas_snp_ids = z_snp$id,
                                              snp_map = snp_map,
                                              type = &quot;sQTL&quot;,
                                              context = &quot;testis&quot;,
                                              weight_format = &quot;PredictDB&quot;,
                                              drop_strand_ambig = TRUE,
                                              scale_predictdb_weights = TRUE,
                                              load_predictdb_LD = TRUE,
                                              filter_protein_coding_genes = TRUE,
                                              ncore = 5)

weights_lymphocytes_expression &lt;- preprocess_weights(weight_files[5],
                                                     region_info,
                                                     gwas_snp_ids = z_snp$id,
                                                     snp_map = snp_map,
                                                     type = &quot;eQTL&quot;,
                                                     context = &quot;lymphocytes&quot;,
                                                     weight_format = &quot;PredictDB&quot;,
                                                     drop_strand_ambig = TRUE,
                                                     scale_predictdb_weights = TRUE,
                                                     load_predictdb_LD = TRUE,
                                                     filter_protein_coding_genes = TRUE,
                                                     ncore = 5)

weights_lymphocytes_splicing &lt;- preprocess_weights(weight_files[6],
                                                   region_info,
                                                   gwas_snp_ids = z_snp$id,
                                                   snp_map = snp_map,
                                                   type = &quot;sQTL&quot;,
                                                   context = &quot;lymphocytes&quot;,
                                                   weight_format = &quot;PredictDB&quot;,
                                                   drop_strand_ambig = TRUE,
                                                   scale_predictdb_weights = TRUE,
                                                   load_predictdb_LD = TRUE,
                                                   filter_protein_coding_genes = TRUE,
                                                   ncore = 5)

weights_RefZ_prostate &lt;- preprocess_weights(&quot;/project2/xinhe/yifany/RefZ_prostate/weights&quot;,
                                            region_info,
                                            gwas_snp_ids = z_snp$id,
                                            snp_map = snp_map,
                                            LD_map = LD_map,
                                            type = &quot;RefZ&quot;,
                                            context = &quot;prostate&quot;,                         
                                            weight_format = &quot;FUSION&quot;,
                                            fusion_method = &quot;lasso&quot;,
                                            fusion_genome_version = &quot;b38&quot;,
                                            top_n_snps = NULL,
                                            drop_strand_ambig = TRUE,
                                            filter_protein_coding_genes = FALSE,
                                            scale_predictdb_weights = FALSE,
                                            load_predictdb_LD = FALSE,
                                            ncore = 5)

weights_TCGA_prostate &lt;- preprocess_weights(&quot;/project2/xinhe/yifany/TCGA_prostate/TCGA-PRAD.TUMOR&quot;,
                                            region_info,
                                            gwas_snp_ids = z_snp$id,
                                            snp_map = snp_map,
                                            LD_map = LD_map,
                                            type = &quot;TCGA&quot;,
                                            context = &quot;prostate&quot;,                         
                                            weight_format = &quot;FUSION&quot;,
                                            fusion_method = &quot;lasso&quot;,
                                            fusion_genome_version = &quot;b38&quot;,
                                            top_n_snps = NULL,
                                            drop_strand_ambig = TRUE,
                                            filter_protein_coding_genes = FALSE,
                                            scale_predictdb_weights = FALSE,
                                            load_predictdb_LD = FALSE,
                                            ncore = 5)

weights &lt;- c(
  weights_prostate_expression,
  weights_prostate_splicing,
  weights_testis_expression,
  weights_testis_splicing,
  weights_lymphocytes_expression,
  weights_lymphocytes_splicing,
  weights_RefZ_prostate,
  weights_TCGA_prostate)</code></pre>
</div>
</div>
<div id="running-ctwas-analysis" class="section level2">
<h2>Running cTWAS analysis</h2>
<p>We use the ctwas main function <code>ctwas_sumstats</code> to run the
cTWAS analysis with LD. For more details on this function, refer to the
cTWAS tutorial: <a
href="https://xinhe-lab.github.io/multigroup_ctwas/articles/running_ctwas_analysis.html"
class="uri">https://xinhe-lab.github.io/multigroup_ctwas/articles/running_ctwas_analysis.html</a></p>
<p>Users can expand the code snippets below to view the exact code
used.</p>
<pre class="r"><code>ctwas_res &lt;- ctwas_sumstats_noLD(z_snp,
                                 weights,
                                 region_info,
                                 snp_map,
                                 thin = 0.1, 
                                 ncore = 5)</code></pre>
</div>
<div id="parameter-estimation" class="section level2">
<h2>Parameter estimation</h2>
<p><code>ctwas_res</code> is the object contains the outputs of
cTWAS</p>
<p>We extract the estimated parameters by
<code>param &lt;- ctwas_res$param</code></p>
<p>We make plots using the function
<code>make_convergence_plots(param, gwas_n)</code> to see how estimated
parameters converge during the execution of the program:</p>
<p><img src="figure/cTWAS_LD_european_combined3_output.Rmd/unnamed-chunk-7-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-7-1">
Past versions of unnamed-chunk-7-1.png
</button>
</p>
<div id="fig-unnamed-chunk-7-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/099102dbbcddba5380e899c2d37db5ecfc27e20d/docs/figure/cTWAS_LD_european_combined3_output.Rmd/unnamed-chunk-7-1.png" target="_blank">099102d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>These plots show the estimated prior inclusion probability, prior
effect size variance, enrichment and proportion of variance explained
(PVE) over the iterations of parameter estimation. The enrichment is
defined as the ratio of the prior inclusion probability of molecular
traits over the prior inclusion probability of variants. We generally
expect molecular traits to have higher prior inclusion probability than
variants. Enrichment values typically range from 20 - 100 for expression
traits.</p>
<p>Then, we use <code>summarize_param(param, gwas_n)</code> to obtain
estimated parameters (from the last iteration) and to compute the PVE by
variants and molecular traits.</p>
<pre><code>[1] &quot;The number of genes/introns/SNPs used in the analysis is:&quot;</code></pre>
<pre><code>   prostate|eQTL    prostate|sQTL      testis|eQTL      testis|sQTL 
            9693            27939            11361            41889 
lymphocytes|eQTL lymphocytes|sQTL    prostate|RefZ    prostate|TCGA 
            8478            24989            12731             4650 
             SNP 
         6646020 </code></pre>
<p><code>ctwas_parameters$prop_heritability</code> contains the
proportion of heritability mediated by molecular traits and variants, we
visualize it using pie chart.</p>
<pre class="r"><code>ctwas_parameters &lt;- summarize_param(param, gwas_n)
data &lt;- data.frame(
  category = names(ctwas_parameters$prop_heritability),
  percentage = ctwas_parameters$prop_heritability
)
# Calculate percentage labels for the chart
data$percentage_label &lt;- paste0(round(data$percentage * 100), &quot;%&quot;)

ggplot(data, aes(x = &quot;&quot;, y = percentage, fill = category)) +
  geom_bar(stat = &quot;identity&quot;, width = 1) +
  coord_polar(&quot;y&quot;, start = 0) +
  theme_void() +  # Remove background and axes
  geom_text(aes(label = percentage_label), 
            position = position_stack(vjust = 0.5)) +
  scale_fill_brewer(palette = &quot;Pastel1&quot;) +
  labs(fill = &quot;Category&quot;) +
  ggtitle(&quot;Percent of heritability&quot;)</code></pre>
<p><img src="figure/cTWAS_LD_european_combined3_output.Rmd/piechart-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-piechart-1">
Past versions of piechart-1.png
</button>
</p>
<div id="fig-piechart-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/075e972669362b36a4f002a7cb77a7efed609c0a/docs/figure/cTWAS_LD_european_combined3_output.Rmd/piechart-1.png" target="_blank">075e972</a>
</td>
<td>
yy896
</td>
<td>
2024-10-31
</td>
</tr>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/099102dbbcddba5380e899c2d37db5ecfc27e20d/docs/figure/cTWAS_LD_european_combined3_output.Rmd/piechart-1.png" target="_blank">099102d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="diagnosis-plots" class="section level2">
<h2>Diagnosis plots</h2>
<p>For all genes analyzed, we compare the z-scores and fine-mapping
PIPs”</p>
<pre class="r"><code>ggplot(data = finemap_res[finemap_res$type!=&quot;SNP&quot;,], aes(x = abs(z), y = susie_pip)) +
  geom_point() +
  labs(x = &quot;abs(z-scores)&quot;, y = &quot;PIPs&quot;) +
  theme_minimal()</code></pre>
<p><img src="figure/cTWAS_LD_european_combined3_output.Rmd/unnamed-chunk-9-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-9-1">
Past versions of unnamed-chunk-9-1.png
</button>
</p>
<div id="fig-unnamed-chunk-9-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/099102dbbcddba5380e899c2d37db5ecfc27e20d/docs/figure/cTWAS_LD_european_combined3_output.Rmd/unnamed-chunk-9-1.png" target="_blank">099102d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="fine-mapping-results" class="section level2">
<h2>Fine-mapping results</h2>
<p>We process the fine-mapping results here.</p>
<p>We first add gene annotations to cTWAS results.</p>
<p><code>mapping_table</code> is a combination of eQTL, sQTL, RefZ
expression and TCGA expression data.</p>
<pre class="r"><code>finemap_res &lt;- anno_finemap_res(finemap_res,
                                snp_map = snp_map,
                                mapping_table = mapping_table,
                                add_gene_annot = TRUE,
                                map_by = &quot;molecular_id&quot;,
                                drop_unmapped = TRUE,
                                add_position = TRUE,
                                use_gene_pos = &quot;mid&quot;)</code></pre>
<p>The annotated fine-mapping results, ones within credible sets are
shown</p>
<pre class="r"><code>finemap_res_show &lt;- subset(finemap_res, group != &quot;SNP&quot; &amp; gene_type == &quot;protein_coding&quot; &amp; susie_pip &gt; 0.8 &amp; !is.na(cs))

DT::datatable(finemap_res_show,caption = htmltools::tags$caption( style = &#39;caption-side: topleft; text-align = left; color:black;&#39;,&#39;The annotated fine-mapping results, ones within credible sets are shown&#39;),options = list(pageLength = 5) )</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-232a8e63d6aacd296363" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-232a8e63d6aacd296363">{"x":{"filter":"none","vertical":false,"caption":"<caption style=\"caption-side: topleft; text-align = left; color:black;\">The annotated fine-mapping results, ones within credible sets are shown<\/caption>","data":[["5372","9353","10980","12615","15171","16235","16580","17900","20192","24963","25511","30544","37063","37459","38486","40891","42263","44797","53864","58391","63210","67078","72350","78234","79495","89279","90630","93227","93396"],["LMO4|prostate_RefZ","ENSG00000133056.13|lymphocytes_eQTL","GDF7|prostate_RefZ","TGFA|prostate_RefZ","intron_2_207165706_207166757|lymphocytes_sQTL","ENSG00000115648.13|lymphocytes_eQTL","intron_2_241195845_241199316|prostate_sQTL","SATB1|prostate_RefZ","CHMP2B|prostate_RefZ","intron_4_94654640_94657427|lymphocytes_sQTL","LOC100505545|prostate_RefZ","HLA-DPB1|prostate_RefZ","WNT2|prostate_RefZ","TRIM24|prostate_RefZ","ZNF395|prostate_RefZ","UBE2R2|prostate_TCGA","C9orf152|prostate_RefZ","LOC283070|prostate_RefZ","PKNOX2|prostate_RefZ","TBX5|prostate_RefZ","ITPK1|prostate_RefZ","IGF1R|prostate_RefZ","ENSG00000179409.10|prostate_eQTL","LOC100506779|prostate_RefZ","FOXJ1|prostate_RefZ","ENSG00000105327.17|prostate_eQTL","ENSG00000170892.10|testis_eQTL","ENSG00000101417.11|lymphocytes_eQTL","ENSG00000126001.15|testis_eQTL"],["LMO4","ENSG00000133056.13","GDF7","TGFA","intron_2_207165706_207166757","ENSG00000115648.13","intron_2_241195845_241199316","SATB1","CHMP2B","intron_4_94654640_94657427","LOC100505545","HLA-DPB1","WNT2","TRIM24","ZNF395","UBE2R2","C9orf152","LOC283070","PKNOX2","TBX5","ITPK1","IGF1R","ENSG00000179409.10","LOC100506779","FOXJ1","ENSG00000105327.17","ENSG00000170892.10","ENSG00000101417.11","ENSG00000126001.15"],["RefZ","eQTL","RefZ","RefZ","sQTL","eQTL","sQTL","RefZ","RefZ","sQTL","RefZ","RefZ","RefZ","RefZ","RefZ","TCGA","RefZ","RefZ","RefZ","RefZ","RefZ","RefZ","eQTL","RefZ","RefZ","eQTL","eQTL","eQTL","eQTL"],["prostate","lymphocytes","prostate","prostate","lymphocytes","lymphocytes","prostate","prostate","prostate","lymphocytes","prostate","prostate","prostate","prostate","prostate","prostate","prostate","prostate","prostate","prostate","prostate","prostate","prostate","prostate","prostate","prostate","testis","lymphocytes","testis"],["prostate|RefZ","lymphocytes|eQTL","prostate|RefZ","prostate|RefZ","lymphocytes|sQTL","lymphocytes|eQTL","prostate|sQTL","prostate|RefZ","prostate|RefZ","lymphocytes|sQTL","prostate|RefZ","prostate|RefZ","prostate|RefZ","prostate|RefZ","prostate|RefZ","prostate|TCGA","prostate|RefZ","prostate|RefZ","prostate|RefZ","prostate|RefZ","prostate|RefZ","prostate|RefZ","prostate|eQTL","prostate|RefZ","prostate|RefZ","prostate|eQTL","testis|eQTL","lymphocytes|eQTL","testis|eQTL"],["1_87662948_89600744","1_203365606_204711940","2_19492643_20850730","2_70528066_72947719","2_205870980_207780675","2_236540389_237852338","2_240620667_241210506","3_17849626_19083652","3_85533081_87360582","4_93312148_95261037","4_145257035_147600722","6_32714887_33268720","7_117140124_118711527","7_137191815_139059933","8_28304875_29470379","9_33578336_34642246","9_110015744_112068802","10_10207433_12544798","11_124625632_126441425","12_113548904_114598797","14_92665954_93858939","15_97487680_98700830","17_150263_1269105","17_57280180_59410151","17_74676064_76379479","19_46646825_48778970","19_54099074_55216712","20_33027017_34225635","20_34225635_36332043"],[8.87652774617216,14.44444444444444,13.30267040791562,4.61077907926688,-6.192982456140351,-10.35733908051552,13.791151879786,5.475864119276906,-20.74844613061263,-14.33072946069555,5.652693739574281,6.383922732319628,-5.593138590979815,-5.594993758786543,-5.214984046812426,7.625322933995303,-6.156709167079851,4.912402169448627,10.11848170491887,13.45393968764112,5.262154828528049,-6.21189016706335,14.40350877192982,-7.383055621966613,4.993475117543033,5.136986301369863,-5.317204335134762,-6.141025641025641,23.11759293172035],[0.9969149804224952,0.94979148733383,0.9991410378341971,0.9814497003902883,0.8822241365813874,0.9872055572673457,0.9999999999089298,0.8530623066560352,1,0.9951631824960016,0.9920512926102973,0.8401216835536892,0.9178035879617265,0.9909848564343262,0.9833551181363075,0.8743292579789752,0.986215721553089,0.9355791174654586,0.9415083194307692,0.9979144404841779,0.9951165125355645,0.9892004245510464,0.9905754108786723,0.9957559259072205,0.910736586491682,0.8806524761640139,0.9926430082705089,0.9068377806513906,1],[76.36521426906259,194.489770509355,170.2903545445792,21.31853757025626,33.71691821977918,100.4659498934163,163.54416143862,29.66718390374582,412.8686338031247,176.5180388434354,31.5499852751521,39.97107759804948,30.90918672539366,30.929045452046,26.99870562781476,49.41350748846637,37.24483637653737,24.06679391082752,98.93646440182478,174.1628635156612,27.47155887687527,37.897847668535,193.3944125133435,53.13158059659626,24.83518003384114,25.43997061992937,27.18751337805008,35.9431897844289,496.6693207268939],["L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1"],["LMO4","PIK3C2B","GDF7","TGFA","KLF7","MLPH","ANO7","SATB1","CHMP2B","PDLIM5","LOC100505545","HLA-DPB1","WNT2","TRIM24","ZNF395","UBE2R2","C9orf152","LOC283070","PKNOX2","TBX5","ITPK1","IGF1R","GEMIN4","LOC100506779","FOXJ1","BBC3","TSEN34","PXMP4","CEP250"],["protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding"],["1","1","2","2","2","2","2","3","3","4","4","6","7","7","8","9","9","10","11","12","14","15","17","17","17","19","19","20","20"],["87662948","204422628","19492643","70528066","207165706","237485428","241195845","17849626","85533081","94654640","145257035","32714887","117140124","137191815","28304875","33578336","110015744","10207433","124625632","113548904","92665954","97487680","744414","57280180","74676064","47220824","54189938","33702754","35455164"],["89600744","204494724","20850730","72947719","207166757","237555318","241199316","19083652","87360582","94657427","147600722","33268720","118711527","139059933","29470379","34642246","112068802","12544798","126441425","114598797","93858939","98700830","752659","59410151","76379479","47232766","54194536","33720319","35519280"],[88631846,204458676,20171686,71737892,207166232,237520373,241197580,18466639,86446832,94656034,146428878,32991804,117925826,138125874,28887627,34110291,111042273,11376116,125533528,114073850,93262446,98094255,748536,58345166,75527772,47226795,54192237,33711536,35487222]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>id<\/th>\n      <th>molecular_id<\/th>\n      <th>type<\/th>\n      <th>context<\/th>\n      <th>group<\/th>\n      <th>region_id<\/th>\n      <th>z<\/th>\n      <th>susie_pip<\/th>\n      <th>mu2<\/th>\n      <th>cs<\/th>\n      <th>gene_name<\/th>\n      <th>gene_type<\/th>\n      <th>chrom<\/th>\n      <th>start<\/th>\n      <th>end<\/th>\n      <th>pos<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":[7,8,9,16]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"id","targets":1},{"name":"molecular_id","targets":2},{"name":"type","targets":3},{"name":"context","targets":4},{"name":"group","targets":5},{"name":"region_id","targets":6},{"name":"z","targets":7},{"name":"susie_pip","targets":8},{"name":"mu2","targets":9},{"name":"cs","targets":10},{"name":"gene_name","targets":11},{"name":"gene_type","targets":12},{"name":"chrom","targets":13},{"name":"start","targets":14},{"name":"end","targets":15},{"name":"pos","targets":16}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script>
<p>Next, we compute gene combined-PIPs across different types of
molecular traits, where only genes within credible sets are shown.</p>
<pre class="r"><code>susie_alpha_res &lt;- anno_susie_alpha_res(susie_alpha_res,
                                        mapping_table = mapping_table,
                                        map_by = &quot;molecular_id&quot;,
                                        drop_unmapped = TRUE)</code></pre>
<pre class="r"><code>combined_pip_by_group &lt;- combine_gene_pips(susie_alpha_res,
                                          group_by = &quot;gene_name&quot;,
                                          by = &quot;group&quot;,
                                          method = &quot;combine_cs&quot;,
                                          filter_cs = TRUE,
                                          include_cs_id = TRUE)
combined_pip_by_group$sQTL_pip_partition &lt;- sapply(combined_pip_by_group$gene_name, function(gene) {
  # Find rows in finemap_res_show matching the gene_name
  matching_rows &lt;- finemap_res_show %&gt;%
    dplyr::filter(gene_name == gene, type == &quot;sQTL&quot;)  # Match gene_name and filter by type == &quot;sQTL&quot;
  # If no matching rows, return NA
  if (nrow(matching_rows) == 0) {
    return(NA)
  }
  # Create the desired string format: molecular_id-round(susie_pip, digits = 4)
  paste(matching_rows$molecular_id, &quot;:PIP=&quot;, round(matching_rows$susie_pip, digits = 4), sep = &quot;&quot;, collapse = &quot;, &quot;)
})

combined_pip_table &lt;- subset(combined_pip_by_group, combined_pip &gt; 0.8)
DT::datatable(combined_pip_table, caption = htmltools::tags$caption(
                style = &#39;caption-side: topleft; text-align = left; color:black;&#39;,
                &#39;Gene PIPs, only genes within credible sets are shown&#39;),
              options = list(pageLength = 5))</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-2112c1f1dbc2c6427f4c" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-2112c1f1dbc2c6427f4c">{"x":{"filter":"none","vertical":false,"caption":"<caption style=\"caption-side: topleft; text-align = left; color:black;\">Gene PIPs, only genes within credible sets are shown<\/caption>","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86"],["CEP250","CHMP2B","ANO7","MMP7","BHLHA15","MYO9B","PKNOX2","GDF7","TBX5","LMO4","LOC100506779","TMPRSS2","PDLIM5","ITPK1","RGS17","PPP1R14A","LGR4","TSEN34","LOC100505545","AGPAT3","TRIM24","GEMIN4","IGF1R","STXBP1","MLPH","C9orf152","GRHL3","CHMP4C","ARNT2","ZNF395","TGM2","ACPP","TGFA","EBF2","ADGRG1","SECTM1","CCDC88C","MYO5C","FMN1","UBE2E2","BCL2L14","SLC39A8","TBX1","PWWP2A","RPS6KL1","TACC1","PIK3C2B","SMAD2","LOC283070","FBXO38","CAPN9","TOMM7","WNT2","FOXJ1","ZDHHC7","ZBTB38","PXMP4","MBNL1","ARAP1","RP11-77H9.1","COL2A1","ZNF236","HAUS6","MAPKBP1","KLF7","BBC3","UBE2R2","GRHL1","DNASE2B","FAM162A","EFHD1","SATB1","BTBD2","HMGCS1","HLA-DPB1","FAM118A","GIGYF2","ZNF568","C9orf64","SENP5","SLC38A1","RCCD1","CHD3","ARMC1","PIM1","CTBP2"],["20_34225635_36332043.L1","3_85533081_87360582.L1","2_240620667_241210506.L1","11_101460390_104088908.L1","7_97470258_99117851.L1","19_16263605_18299052.L1","11_124625632_126441425.L1","2_19492643_20850730.L1","12_113548904_114598797.L1","1_85989068_87662948.L1,1_87662948_89600744.L1","17_57280180_59410151.L1","21_40017600_41901419.L1","4_93312148_95261037.L1","14_92665954_93858939.L1","6_152773361_154652986.L1","19_37036131_39679979.L1","11_26998914_28460046.L1","19_54099074_55216712.L1","4_145257035_147600722.L1","21_43086158_44757190.L1","7_137191815_139059933.L1","17_150263_1269105.L1","15_97487680_98700830.L1","9_127293231_129403191.L1","2_236540389_237852338.L1","9_110015744_112068802.L1","1_23594100_25190354.L1","8_81044160_81781504.L1","15_78223711_80568637.L1","8_26825008_28304875.L1,8_28304875_29470379.L1","20_36332043_38281128.L1","3_132117672_133533329.L1","2_70528066_72947719.L1","8_25625938_26825008.L1","16_57630418_59011501.L1","17_82076532_83247139.L1","14_90830516_92665954.L1","15_51385363_52776899.L1","15_32149610_33723224.L1","3_22162752_23763374.L1","12_11601912_12580594.L1","4_102300199_104384137.L1","22_17813322_19924835.L1","5_159398690_160546937.L1","14_72422907_75978424.L1","8_37521871_39081287.L1","1_203365606_204711940.L1","18_46719283_48413361.L1","10_10207433_12544798.L1","5_147802435_149283061.L1","1_230549509_231954506.L1","7_22468010_23431823.L1","7_117140124_118711527.L1","17_74676064_76379479.L1","16_84412277_85115079.L1","3_140235755_141620255.L1","20_33027017_34225635.L1","3_151630942_153538782.L1","11_72574973_74701306.L1","16_8288517_9705562.L1","12_47321010_48608083.L1","18_75815032_77101859.L1","9_18661053_20463536.L1","15_40885316_42484201.L1","2_205870980_207780675.L1","19_46646825_48778970.L1","9_33578336_34642246.L1","2_8282017_10158342.L1","1_84378812_85989068.L1","3_122255250_123798921.L1","2_230978674_232685293.L1","3_17849626_19083652.L1","19_610729_2098397.L1","5_41888608_43983397.L1","6_32714887_33268720.L1","22_44599428_46074615.L1","2_232685293_234242343.L1","19_35978393_37036131.L1","9_82825886_84323281.L1","3_196244537_197349116.L1","12_45630446_47321010.L1","15_89932319_91621162.L1","17_7414079_8403107.L1","8_65576855_68950822.L1","6_35487979_37604820.L1","10_124109830_126312529.L1"],[1,1,0.9999999999089298,0.9999997938817149,0.9995172877299063,0.9993578577226783,0.9992465069088894,0.9991410378341971,0.9979144404841779,0.9969571999716287,0.9957559259072205,0.9952046732283537,0.9951631824960016,0.9951165125355645,0.9950524210688071,0.9947748760007911,0.9939811849880033,0.9926430082705089,0.9920512926102973,0.9916082933787448,0.9909848564343262,0.9905754108786723,0.9892004245510464,0.9888455778071907,0.9872055572673457,0.986215721553089,0.9862076294233734,0.9852574769381721,0.9851152080393533,0.983386809145084,0.9826844955893879,0.9825853733857625,0.9814497003902883,0.9724708598003707,0.9723761313146767,0.9705890865480412,0.9697345618709604,0.9695358049967499,0.9670502616325477,0.965097562634179,0.9617544715106918,0.96056706905232,0.9581971811971841,0.9569582869898468,0.9562140622048158,0.9529601409810315,0.94979148733383,0.9413379526391698,0.9355791174654586,0.9318374336538175,0.9284074300191616,0.920281130443382,0.9178035879617265,0.910736586491682,0.9105744555071841,0.9093302527348391,0.9068377806513906,0.9059389146426077,0.9040744656449466,0.9031451134244735,0.9020100211904601,0.8909050878134713,0.8905265670797509,0.8899867765699263,0.8822241365813874,0.8806524761640139,0.8743292579789752,0.867649911360909,0.8670391029572899,0.8668698164297988,0.8546676115409579,0.8530623066560352,0.8481474290771971,0.8455204946756157,0.8401216835536892,0.8370825696901748,0.8310672081639633,0.8271828527143636,0.8127926771754768,0.8104532179494045,0.8102161027083427,0.8097814828817715,0.8093960650432177,0.8058822486302315,0.8055643571028663,0.8007839667505662],[null,null,null,null,null,null,"11_124625632_126441425.L1",null,null,null,null,null,null,null,null,null,null,null,null,null,null,"17_150263_1269105.L1",null,null,null,null,null,"8_81044160_81781504.L1","15_78223711_80568637.L1",null,null,null,null,null,null,null,null,null,null,null,null,null,"22_17813322_19924835.L1",null,"14_72422907_75978424.L1",null,null,null,null,"5_147802435_149283061.L1",null,"7_22468010_23431823.L1",null,null,"16_84412277_85115079.L1",null,null,null,null,null,null,"18_75815032_77101859.L1",null,null,null,"19_46646825_48778970.L1",null,"2_8282017_10158342.L1",null,null,null,null,"19_610729_2098397.L1",null,null,"22_44599428_46074615.L1",null,null,null,null,null,null,null,"8_65576855_68950822.L1",null,null],[null,null,null,null,null,null,0.05773818747812021,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0.9905754108786723,null,null,null,null,null,0.03890038156208908,0.2439161018791921,null,null,null,null,null,null,null,null,null,null,null,null,null,0.2214309367088567,null,0.08021733529108332,null,null,null,null,0.07029274524210338,null,0.03812463481028028,null,null,0.2924804497256203,null,null,null,null,null,null,0.01158500163367626,null,null,null,0.8806524761640139,null,0.5165207914544616,null,null,null,null,0.03471985193407112,null,null,0.01506482690066291,null,null,null,null,null,null,null,0.0002971929417111552,null,null],[null,null,"2_240620667_241210506.L1",null,null,null,null,null,null,"1_85989068_87662948.L1",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"16_57630418_59011501.L1",null,"14_90830516_92665954.L1",null,null,null,null,null,null,"5_159398690_160546937.L1","14_72422907_75978424.L1","8_37521871_39081287.L1",null,null,null,"5_147802435_149283061.L1",null,"7_22468010_23431823.L1",null,null,null,"3_140235755_141620255.L1",null,null,"11_72574973_74701306.L1",null,null,null,null,"15_40885316_42484201.L1",null,null,null,"2_8282017_10158342.L1","1_84378812_85989068.L1","3_122255250_123798921.L1",null,null,null,null,null,"22_44599428_46074615.L1","2_232685293_234242343.L1","19_35978393_37036131.L1",null,"3_196244537_197349116.L1","12_45630446_47321010.L1",null,"17_7414079_8403107.L1",null,null,null],[null,null,0.9999999999089298,null,null,null,null,null,null,0.006535978686597721,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0.4073987581915592,null,0.5504842307630851,null,null,null,null,null,null,0.05074845241556147,0.0528437448106629,0.3602867665882177,null,null,null,0.07314856991356478,null,0.04052209069805357,null,null,null,0.04629834563259194,null,null,0.1502575859175613,null,null,null,null,0.5255632789653809,null,null,null,0.1421300106871762,0.08435618587031635,0.04172071952804968,null,null,null,null,null,0.4082828775795692,0.009427041393616609,0.02052910804720165,null,0.373979748234569,0.0008208854331668292,null,0.4092904726583009,null,null,null],["20_34225635_36332043.L1",null,null,null,null,null,null,null,null,null,null,null,null,null,null,"19_37036131_39679979.L1",null,"19_54099074_55216712.L1",null,null,null,null,null,null,null,null,null,null,"15_78223711_80568637.L1",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"14_72422907_75978424.L1",null,null,"18_46719283_48413361.L1",null,"5_147802435_149283061.L1",null,null,null,null,null,null,null,null,null,"16_8288517_9705562.L1",null,null,null,null,null,null,null,null,null,null,null,null,"19_610729_2098397.L1",null,null,"22_44599428_46074615.L1",null,null,null,null,null,null,null,null,null,null],[1,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0.09526426138477573,null,0.9926430082705089,null,null,null,null,null,null,null,null,null,null,0.176511390077594,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0.0580497689634385,null,null,0.02211153509114372,null,0.01430254537633202,null,null,null,null,null,null,null,null,null,0.5439482964160625,null,null,null,null,null,null,null,null,null,null,null,null,0.01164139111460438,null,null,0.0115485035031978,null,null,null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null,null,"1_85989068_87662948.L1",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"16_57630418_59011501.L1",null,null,null,null,null,null,null,null,"5_159398690_160546937.L1","14_72422907_75978424.L1","8_37521871_39081287.L1",null,"18_46719283_48413361.L1",null,"5_147802435_149283061.L1",null,null,null,null,null,"3_140235755_141620255.L1",null,null,"11_72574973_74701306.L1","16_8288517_9705562.L1",null,"18_75815032_77101859.L1",null,"15_40885316_42484201.L1",null,null,null,null,null,"3_122255250_123798921.L1",null,null,"19_610729_2098397.L1",null,null,"22_44599428_46074615.L1","2_232685293_234242343.L1","19_35978393_37036131.L1",null,"3_196244537_197349116.L1",null,null,"17_7414079_8403107.L1","8_65576855_68950822.L1",null,null],[null,null,null,null,null,null,null,null,null,0.004532021753741233,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0.5649773731231174,null,null,null,null,null,null,null,null,0.07037755211400509,0.01224611746001503,0.2647683863354146,null,0.006574038505188229,null,0.02154833411948598,null,null,null,null,null,0.5260825526360882,null,null,0.6615121900907103,0.359196817008411,null,0.08798534877340547,null,0.3644234976045455,null,null,null,null,null,0.09136299639550305,null,null,0.003278030054515768,null,null,0.3265666673706527,0.006420701838719678,0.02029040146182071,null,0.1333360298184747,null,null,0.2111801800858664,0.007921490293497979,null,null],[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"19_37036131_39679979.L1",null,null,null,null,null,null,null,null,"2_236540389_237852338.L1",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"14_72422907_75978424.L1",null,"1_203365606_204711940.L1",null,null,"5_147802435_149283061.L1",null,null,null,null,null,null,"20_33027017_34225635.L1","3_151630942_153538782.L1",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"19_35978393_37036131.L1","9_82825886_84323281.L1",null,null,"15_89932319_91621162.L1",null,"8_65576855_68950822.L1",null,null],[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0.1530184571299622,null,null,null,null,null,null,null,null,0.9872055572673457,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0.007536161620041448,null,0.94979148733383,null,null,0.08900452453118268,null,null,null,null,null,null,0.9068377806513906,0.08498606092982142,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0.7699215223769088,0.03610842717617391,null,null,0.08280594319482504,null,0.0007489397160256317,null,null],[null,null,null,null,null,null,null,null,null,"1_85989068_87662948.L1",null,null,"4_93312148_95261037.L1",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"8_26825008_28304875.L1",null,null,null,null,null,null,"14_90830516_92665954.L1",null,null,null,null,null,null,"5_159398690_160546937.L1",null,"8_37521871_39081287.L1",null,null,null,"5_147802435_149283061.L1",null,null,null,null,null,"3_140235755_141620255.L1",null,"3_151630942_153538782.L1","11_72574973_74701306.L1",null,null,"18_75815032_77101859.L1",null,null,"2_205870980_207780675.L1",null,null,null,null,"3_122255250_123798921.L1",null,null,null,null,null,"22_44599428_46074615.L1","2_232685293_234242343.L1","19_35978393_37036131.L1",null,"3_196244537_197349116.L1","12_45630446_47321010.L1",null,"17_7414079_8403107.L1","8_65576855_68950822.L1",null,null],[null,null,null,null,null,null,null,null,null,0.002617341928748407,null,null,0.9951631824960016,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0.001903949156029539,null,null,null,null,null,null,0.4192503311078754,null,null,null,null,null,null,0.04064457939518862,null,0.2885546926577855,null,null,null,0.03892280247783142,null,null,null,null,null,0.3369493544661591,null,0.03721680332985478,0.09230468963667493,null,null,0.1016269360446366,null,null,0.8822241365813874,null,null,null,null,0.005870998341226286,null,null,null,null,null,0.07561969433609228,0.003952476864602428,0.01644182082843249,null,0.299521440473554,0.0003287247351849709,null,0.1889254122990505,0.009002776192427286,null,null],[null,"3_85533081_87360582.L1",null,"11_101460390_104088908.L1","7_97470258_99117851.L1","19_16263605_18299052.L1","11_124625632_126441425.L1","2_19492643_20850730.L1","12_113548904_114598797.L1","1_87662948_89600744.L1","17_57280180_59410151.L1",null,null,"14_92665954_93858939.L1","6_152773361_154652986.L1","19_37036131_39679979.L1","11_26998914_28460046.L1",null,"4_145257035_147600722.L1","21_43086158_44757190.L1","7_137191815_139059933.L1",null,"15_97487680_98700830.L1","9_127293231_129403191.L1",null,"9_110015744_112068802.L1",null,"8_81044160_81781504.L1","15_78223711_80568637.L1","8_28304875_29470379.L1","20_36332043_38281128.L1","3_132117672_133533329.L1","2_70528066_72947719.L1",null,null,null,null,null,null,"3_22162752_23763374.L1","12_11601912_12580594.L1",null,"22_17813322_19924835.L1","5_159398690_160546937.L1","14_72422907_75978424.L1","8_37521871_39081287.L1",null,null,"10_10207433_12544798.L1","5_147802435_149283061.L1","1_230549509_231954506.L1","7_22468010_23431823.L1","7_117140124_118711527.L1","17_74676064_76379479.L1","16_84412277_85115079.L1",null,null,"3_151630942_153538782.L1",null,null,"12_47321010_48608083.L1","18_75815032_77101859.L1","9_18661053_20463536.L1",null,null,null,null,null,"1_84378812_85989068.L1",null,null,"3_17849626_19083652.L1","19_610729_2098397.L1","5_41888608_43983397.L1","6_32714887_33268720.L1",null,null,null,null,"3_196244537_197349116.L1","12_45630446_47321010.L1","15_89932319_91621162.L1",null,"8_65576855_68950822.L1","6_35487979_37604820.L1","10_124109830_126312529.L1"],[null,1,null,0.1549992034629429,0.9995172877299063,0.9993578577226783,0.9415083194307692,0.9991410378341971,0.9979144404841779,0.9969149804224952,0.9957559259072205,null,null,0.9951165125355645,0.9950524210688071,0.7464921574860531,0.8912417851716635,null,0.9920512926102973,0.4329220945063733,0.9909848564343262,null,0.9892004245510464,0.9053443463527049,null,0.986215721553089,null,0.1293907077289669,0.5646877160825672,0.9833551181363075,0.0922468748559464,0.05950100026463645,0.9814497003902883,null,null,null,null,null,null,0.965097562634179,0.1069543818417863,null,0.7367662444883274,0.112360706725559,0.2626283198666179,0.01932341535544113,null,null,0.9355791174654586,0.6246179119933175,0.06443285756849704,0.0657285706652202,0.9178035879617265,0.910736586491682,0.6180940057815638,null,null,0.7792899251866459,null,null,0.8757337893240539,0.2982466590358102,0.131398401948982,null,null,null,null,null,0.009081475948130913,null,null,0.8530623066560352,0.6152422665320409,0.8455204946756157,0.8401216835536892,null,null,null,null,0.003615999422806659,0.01966804657103294,0.00713241173794843,null,0.7879118494865693,0.1110839190167932,0.8007839667505662],[null,null,null,"11_101460390_104088908.L1",null,null,null,null,null,null,null,"21_40017600_41901419.L1",null,null,null,null,"11_26998914_28460046.L1",null,null,"21_43086158_44757190.L1",null,null,null,"9_127293231_129403191.L1",null,null,"1_23594100_25190354.L1","8_81044160_81781504.L1",null,null,"20_36332043_38281128.L1","3_132117672_133533329.L1",null,"8_25625938_26825008.L1",null,"17_82076532_83247139.L1",null,"15_51385363_52776899.L1","15_32149610_33723224.L1",null,"12_11601912_12580594.L1","4_102300199_104384137.L1",null,"5_159398690_160546937.L1","14_72422907_75978424.L1","8_37521871_39081287.L1",null,"18_46719283_48413361.L1",null,null,"1_230549509_231954506.L1","7_22468010_23431823.L1",null,null,null,null,null,"3_151630942_153538782.L1",null,null,"12_47321010_48608083.L1","18_75815032_77101859.L1","9_18661053_20463536.L1",null,null,null,"9_33578336_34642246.L1","2_8282017_10158342.L1","1_84378812_85989068.L1","3_122255250_123798921.L1","2_230978674_232685293.L1",null,"19_610729_2098397.L1",null,null,null,"2_232685293_234242343.L1",null,"9_82825886_84323281.L1",null,"12_45630446_47321010.L1","15_89932319_91621162.L1",null,null,"6_35487979_37604820.L1",null],[null,null,null,0.845000590418772,null,null,null,null,null,null,null,0.9952046732283537,null,null,null,null,0.1027393998163397,null,null,0.5586861988723715,null,null,null,0.08350123145448585,null,null,0.9862076294233734,0.8169663876471159,null,null,0.8904376207334416,0.923084373121126,null,0.9724708598003707,null,0.9705890865480412,null,0.9695358049967499,0.9670502616325477,null,0.8548000896689055,0.96056706905232,null,0.6828269963395325,0.4826926141929566,0.02002688004417263,null,0.9126523790428378,null,null,0.8639745724506646,0.7759058342698278,null,null,null,null,null,0.004446125196285511,null,null,0.02627623186640626,0.3914611423259429,0.7591281651307689,null,null,null,0.8743292579789752,0.2089991092192711,0.7736014411388425,0.7279151021650198,0.8546676115409579,null,0.1832658894419648,null,null,null,0.8112669880670246,null,0.7766842499993029,null,0.789398445968958,0.7198431279489982,null,null,0.6944804380860731,null],[null,null,"intron_2_241195845_241199316:PIP=1",null,null,null,null,null,null,null,null,null,"intron_4_94654640_94657427:PIP=0.9952",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"intron_2_207165706_207166757:PIP=0.8822",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>gene_name<\/th>\n      <th>combined_cs_id<\/th>\n      <th>combined_pip<\/th>\n      <th>prostate|eQTL_cs_id<\/th>\n      <th>prostate|eQTL_pip<\/th>\n      <th>prostate|sQTL_cs_id<\/th>\n      <th>prostate|sQTL_pip<\/th>\n      <th>testis|eQTL_cs_id<\/th>\n      <th>testis|eQTL_pip<\/th>\n      <th>testis|sQTL_cs_id<\/th>\n      <th>testis|sQTL_pip<\/th>\n      <th>lymphocytes|eQTL_cs_id<\/th>\n      <th>lymphocytes|eQTL_pip<\/th>\n      <th>lymphocytes|sQTL_cs_id<\/th>\n      <th>lymphocytes|sQTL_pip<\/th>\n      <th>prostate|RefZ_cs_id<\/th>\n      <th>prostate|RefZ_pip<\/th>\n      <th>prostate|TCGA_cs_id<\/th>\n      <th>prostate|TCGA_pip<\/th>\n      <th>sQTL_pip_partition<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":[3,5,7,9,11,13,15,17,19]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"gene_name","targets":1},{"name":"combined_cs_id","targets":2},{"name":"combined_pip","targets":3},{"name":"prostate|eQTL_cs_id","targets":4},{"name":"prostate|eQTL_pip","targets":5},{"name":"prostate|sQTL_cs_id","targets":6},{"name":"prostate|sQTL_pip","targets":7},{"name":"testis|eQTL_cs_id","targets":8},{"name":"testis|eQTL_pip","targets":9},{"name":"testis|sQTL_cs_id","targets":10},{"name":"testis|sQTL_pip","targets":11},{"name":"lymphocytes|eQTL_cs_id","targets":12},{"name":"lymphocytes|eQTL_pip","targets":13},{"name":"lymphocytes|sQTL_cs_id","targets":14},{"name":"lymphocytes|sQTL_pip","targets":15},{"name":"prostate|RefZ_cs_id","targets":16},{"name":"prostate|RefZ_pip","targets":17},{"name":"prostate|TCGA_cs_id","targets":18},{"name":"prostate|TCGA_pip","targets":19},{"name":"sQTL_pip_partition","targets":20}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="ideogram" class="section level2">
<h2>Ideogram</h2>
<p><img src="figure/cTWAS_LD_european_combined3_output.Rmd/Ideogram-1.png" width="3840" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Ideogram-1">
Past versions of Ideogram-1.png
</button>
</p>
<div id="fig-Ideogram-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/099102dbbcddba5380e899c2d37db5ecfc27e20d/docs/figure/cTWAS_LD_european_combined3_output.Rmd/Ideogram-1.png" target="_blank">099102d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="locus-plots-only-showing-top-10-here" class="section level2">
<h2>Locus plots (ONLY showing top 10 here)</h2>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;20_34225635_36332043&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-11-03 13:54:10.484952 INFO::Limit to protein coding genes
2024-11-03 13:54:10.489316 INFO::focal id: ENSG00000126001.15|testis_eQTL
2024-11-03 13:54:10.493374 INFO::focal molecular trait: CEP250 testis eQTL
2024-11-03 13:54:10.493646 INFO::Range of locus: chr20:34225783-36331600</code></pre>
<pre><code>chromosome 20, position 34225783 to 36331600</code></pre>
<pre><code>3896 SNPs/datapoints</code></pre>
<pre><code>2024-11-03 13:54:10.759109 INFO::focal molecular trait QTL positions: 35445740,35454222,35456125
2024-11-03 13:54:10.775387 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 458 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined3_output.Rmd/locus%20plot-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-1">
Past versions of "locus plot-1.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/099102dbbcddba5380e899c2d37db5ecfc27e20d/docs/figure/cTWAS_LD_european_combined3_output.Rmd/locus plot-1.png" target="_blank">099102d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;3_85533081_87360582&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-11-03 13:54:11.8094 INFO::Limit to protein coding genes
2024-11-03 13:54:11.81 INFO::focal id: CHMP2B|prostate_RefZ
2024-11-03 13:54:11.810271 INFO::focal molecular trait: CHMP2B prostate RefZ
2024-11-03 13:54:11.810539 INFO::Range of locus: chr3:85532981-87360172</code></pre>
<pre><code>chromosome 3, position 85532981 to 87360172</code></pre>
<pre><code>4787 SNPs/datapoints</code></pre>
<pre><code>2024-11-03 13:54:11.930319 INFO::focal molecular trait QTL positions: 87122674,87149239,87149381,87155929,87159999,87160086,87172220,87214347,87235251,87246499,87252295
2024-11-03 13:54:11.944524 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 7 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<pre><code>Warning: ggrepel: 11 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined3_output.Rmd/locus%20plot-2.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-2">
Past versions of "locus plot-2.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/099102dbbcddba5380e899c2d37db5ecfc27e20d/docs/figure/cTWAS_LD_european_combined3_output.Rmd/locus plot-2.png" target="_blank">099102d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;2_240620667_241210506&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-11-03 13:54:12.622044 INFO::Limit to protein coding genes
2024-11-03 13:54:12.622624 INFO::focal id: intron_2_241195845_241199316|prostate_sQTL
2024-11-03 13:54:12.622874 INFO::focal molecular trait: ANO7 prostate sQTL
2024-11-03 13:54:12.623102 INFO::Range of locus: chr2:240618245-241608572</code></pre>
<pre><code>chromosome 2, position 240618245 to 241608572</code></pre>
<pre><code>2092 SNPs/datapoints</code></pre>
<pre><code>2024-11-03 13:54:12.730194 INFO::focal molecular trait QTL positions: 241195850,241202304
2024-11-03 13:54:12.743983 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 133 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined3_output.Rmd/locus%20plot-3.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-3">
Past versions of "locus plot-3.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/099102dbbcddba5380e899c2d37db5ecfc27e20d/docs/figure/cTWAS_LD_european_combined3_output.Rmd/locus plot-3.png" target="_blank">099102d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;11_101460390_104088908&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-11-03 13:54:13.355854 INFO::Limit to protein coding genes
2024-11-03 13:54:13.356567 INFO::focal id: MMP7|prostate_TCGA
2024-11-03 13:54:13.356899 INFO::focal molecular trait: MMP7,MMP7 prostate,prostate TCGA,TCGA
2024-11-03 13:54:13.357211 INFO::Range of locus: chr11:101460301-104087928</code></pre>
<pre><code>chromosome 11, position 101460301 to 104087928</code></pre>
<pre><code>8434 SNPs/datapoints</code></pre>
<pre><code>2024-11-03 13:54:13.646554 INFO::focal molecular trait QTL positions: 102474543,102518756,102530930,102536460,102553159,102553897,102557982,102559377,102573296,102573400,102577525,102579220
2024-11-03 13:54:13.659889 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 80 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<pre><code>Warning: ggrepel: 12 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined3_output.Rmd/locus%20plot-4.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-4">
Past versions of "locus plot-4.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/099102dbbcddba5380e899c2d37db5ecfc27e20d/docs/figure/cTWAS_LD_european_combined3_output.Rmd/locus plot-4.png" target="_blank">099102d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;7_97470258_99117851&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-11-03 13:54:14.254081 INFO::Limit to protein coding genes
2024-11-03 13:54:14.255366 INFO::focal id: BHLHA15|prostate_RefZ
2024-11-03 13:54:14.255682 INFO::focal molecular trait: BHLHA15,BHLHA15 prostate,prostate RefZ,RefZ
2024-11-03 13:54:14.255949 INFO::Range of locus: chr7:97470327-99117511</code></pre>
<pre><code>chromosome 7, position 97470327 to 99117511</code></pre>
<pre><code>4641 SNPs/datapoints</code></pre>
<pre><code>2024-11-03 13:54:14.330652 INFO::focal molecular trait QTL positions: 97925967,98028065,98028141,98046947,98144500,98145547,98149264,98154749,98156188,98156970,98160039,98320338,98349597,98391265,98490538
2024-11-03 13:54:14.344255 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 92 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<pre><code>Warning: ggrepel: 8 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined3_output.Rmd/locus%20plot-5.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-5">
Past versions of "locus plot-5.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/099102dbbcddba5380e899c2d37db5ecfc27e20d/docs/figure/cTWAS_LD_european_combined3_output.Rmd/locus plot-5.png" target="_blank">099102d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;19_16263605_18299052&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-11-03 13:54:15.130492 INFO::Limit to protein coding genes
2024-11-03 13:54:15.131835 INFO::focal id: MYO9B|prostate_RefZ
2024-11-03 13:54:15.13215 INFO::focal molecular trait: MYO9B,MYO9B prostate,prostate RefZ,RefZ
2024-11-03 13:54:15.132423 INFO::Range of locus: chr19:16263732-18298377</code></pre>
<pre><code>chromosome 19, position 16263732 to 18298377</code></pre>
<pre><code>6504 SNPs/datapoints</code></pre>
<pre><code>2024-11-03 13:54:15.237553 INFO::focal molecular trait QTL positions: 16744454,17030435,17096346,17097041,17099048,17100758,17101372,17101600,17104392,17108295,17117744,17118629,17118822,17415324,17537541,17554084,17554444,17649423
2024-11-03 13:54:15.250853 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 516 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<pre><code>Warning: ggrepel: 12 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined3_output.Rmd/locus%20plot-6.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-6">
Past versions of "locus plot-6.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/099102dbbcddba5380e899c2d37db5ecfc27e20d/docs/figure/cTWAS_LD_european_combined3_output.Rmd/locus plot-6.png" target="_blank">099102d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;11_124625632_126441425&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-11-03 13:54:16.572134 INFO::Limit to protein coding genes
2024-11-03 13:54:16.573295 INFO::focal id: PKNOX2|prostate_RefZ
2024-11-03 13:54:16.573568 INFO::focal molecular trait: PKNOX2 prostate RefZ
2024-11-03 13:54:16.573834 INFO::Range of locus: chr11:124627414-126441118</code></pre>
<pre><code>chromosome 11, position 124627414 to 126441118</code></pre>
<pre><code>5331 SNPs/datapoints</code></pre>
<pre><code>2024-11-03 13:54:16.648119 INFO::focal molecular trait QTL positions: 125184897,125274530
2024-11-03 13:54:16.661113 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 345 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined3_output.Rmd/locus%20plot-7.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-7">
Past versions of "locus plot-7.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/099102dbbcddba5380e899c2d37db5ecfc27e20d/docs/figure/cTWAS_LD_european_combined3_output.Rmd/locus plot-7.png" target="_blank">099102d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;2_19492643_20850730&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-11-03 13:54:17.866917 INFO::Limit to protein coding genes
2024-11-03 13:54:17.867525 INFO::focal id: GDF7|prostate_RefZ
2024-11-03 13:54:17.867778 INFO::focal molecular trait: GDF7 prostate RefZ
2024-11-03 13:54:17.868024 INFO::Range of locus: chr2:19352676-20849469</code></pre>
<pre><code>chromosome 2, position 19352676 to 20849469</code></pre>
<pre><code>4125 SNPs/datapoints</code></pre>
<pre><code>2024-11-03 13:54:17.963925 INFO::focal molecular trait QTL positions: 20241617,20677676,20678345,20678393,20681073
2024-11-03 13:54:17.977211 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 82 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined3_output.Rmd/locus%20plot-8.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-8">
Past versions of "locus plot-8.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/099102dbbcddba5380e899c2d37db5ecfc27e20d/docs/figure/cTWAS_LD_european_combined3_output.Rmd/locus plot-8.png" target="_blank">099102d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;12_113548904_114598797&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-11-03 13:54:18.555406 INFO::Limit to protein coding genes
2024-11-03 13:54:18.556038 INFO::focal id: TBX5|prostate_RefZ
2024-11-03 13:54:18.55632 INFO::focal molecular trait: TBX5 prostate RefZ
2024-11-03 13:54:18.556599 INFO::Range of locus: chr12:113550218-114597138</code></pre>
<pre><code>chromosome 12, position 113550218 to 114597138</code></pre>
<pre><code>3483 SNPs/datapoints</code></pre>
<pre><code>2024-11-03 13:54:18.624949 INFO::focal molecular trait QTL positions: 114070107,114247766
2024-11-03 13:54:18.639712 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 31 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined3_output.Rmd/locus%20plot-9.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-9">
Past versions of "locus plot-9.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/099102dbbcddba5380e899c2d37db5ecfc27e20d/docs/figure/cTWAS_LD_european_combined3_output.Rmd/locus plot-9.png" target="_blank">099102d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;1_85989068_87662948&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-11-03 13:54:19.299301 INFO::Limit to protein coding genes
2024-11-03 13:54:19.300422 INFO::focal id: LOC339524|prostate_TCGA
2024-11-03 13:54:19.30073 INFO::focal molecular trait: LOC339524,LOC339524 prostate,prostate TCGA,TCGA
2024-11-03 13:54:19.300982 INFO::Range of locus: chr1:85942988-87662356</code></pre>
<pre><code>chromosome 1, position 85942988 to 87662356</code></pre>
<pre><code>4622 SNPs/datapoints</code></pre>
<pre><code>2024-11-03 13:54:19.397865 INFO::focal molecular trait QTL positions: 87118956,87134014,87148492
2024-11-03 13:54:19.411571 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 87 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<pre><code>Warning: ggrepel: 92 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined3_output.Rmd/locus%20plot-10.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-10">
Past versions of "locus plot-10.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-10" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/099102dbbcddba5380e899c2d37db5ecfc27e20d/docs/figure/cTWAS_LD_european_combined3_output.Rmd/locus plot-10.png" target="_blank">099102d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="gsea-gene-ontology" class="section level2">
<h2>GSEA-Gene Ontology</h2>
<pre class="r"><code>dbs &lt;- c(&quot;GO_Biological_Process_2021&quot;, &quot;GO_Cellular_Component_2021&quot;, &quot;GO_Molecular_Function_2021&quot;)
genes &lt;- combined_pip_by_group$gene_name[combined_pip_by_group$combined_pip &gt;0.8]

#number of genes for gene set enrichment
sprintf(&quot;The number of genes used in enrichment analysis = %s&quot;, length(genes))</code></pre>
<pre><code>[1] &quot;The number of genes used in enrichment analysis = 86&quot;</code></pre>
<pre class="r"><code>GO_enrichment &lt;- enrichr(genes, dbs)</code></pre>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2021... Done.
  Querying GO_Cellular_Component_2021... Done.
  Querying GO_Molecular_Function_2021... Done.
Parsing results... Done.</code></pre>
<pre class="r"><code>print(&quot;GO_Biological_Process_2021&quot;)</code></pre>
<pre><code>[1] &quot;GO_Biological_Process_2021&quot;</code></pre>
<pre class="r"><code>db &lt;- &quot;GO_Biological_Process_2021&quot;
df &lt;- GO_enrichment[[db]]
print(plotEnrich(GO_enrichment[[db]]))</code></pre>
<p><img src="figure/cTWAS_LD_european_combined3_output.Rmd/GO_Biological_Process_2021-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-GO_Biological_Process_2021-1">
Past versions of GO_Biological_Process_2021-1.png
</button>
</p>
<div id="fig-GO_Biological_Process_2021-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/099102dbbcddba5380e899c2d37db5ecfc27e20d/docs/figure/cTWAS_LD_european_combined3_output.Rmd/GO_Biological_Process_2021-1.png" target="_blank">099102d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>df &lt;- df[df$Adjusted.P.value&lt;0.05,c(&quot;Term&quot;, &quot;Overlap&quot;, &quot;Adjusted.P.value&quot;, &quot;Genes&quot;)]

DT::datatable(df,caption = htmltools::tags$caption( style = &#39;caption-side: topleft; text-align = left; color:black;&#39;,&#39;Enriched pathways from GO_Biological_Process_2021&#39;),options = list(pageLength = 5) )</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-57a5e797e447beab7be8" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-57a5e797e447beab7be8">{"x":{"filter":"none","vertical":false,"caption":"<caption style=\"caption-side: topleft; text-align = left; color:black;\">Enriched pathways from GO_Biological_Process_2021<\/caption>","data":[[],[],[],[],[]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":3},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Term","targets":1},{"name":"Overlap","targets":2},{"name":"Adjusted.P.value","targets":3},{"name":"Genes","targets":4}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>print(&quot;GO_Cellular_Component_2021&quot;)</code></pre>
<pre><code>[1] &quot;GO_Cellular_Component_2021&quot;</code></pre>
<pre class="r"><code>db &lt;- &quot;GO_Cellular_Component_2021&quot;
df &lt;- GO_enrichment[[db]]
print(plotEnrich(GO_enrichment[[db]]))</code></pre>
<p><img src="figure/cTWAS_LD_european_combined3_output.Rmd/GO_Cellular_Component_2021-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-GO_Cellular_Component_2021-1">
Past versions of GO_Cellular_Component_2021-1.png
</button>
</p>
<div id="fig-GO_Cellular_Component_2021-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/099102dbbcddba5380e899c2d37db5ecfc27e20d/docs/figure/cTWAS_LD_european_combined3_output.Rmd/GO_Cellular_Component_2021-1.png" target="_blank">099102d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>df &lt;- df[df$Adjusted.P.value&lt;0.05,c(&quot;Term&quot;, &quot;Overlap&quot;, &quot;Adjusted.P.value&quot;, &quot;Genes&quot;)]

DT::datatable(df,caption = htmltools::tags$caption( style = &#39;caption-side: topleft; text-align = left; color:black;&#39;,&#39;Enriched pathways from GO_Cellular_Component_2021&#39;),options = list(pageLength = 5) )</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-0a34d9ca313023088b60" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-0a34d9ca313023088b60">{"x":{"filter":"none","vertical":false,"caption":"<caption style=\"caption-side: topleft; text-align = left; color:black;\">Enriched pathways from GO_Cellular_Component_2021<\/caption>","data":[[],[],[],[],[]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":3},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Term","targets":1},{"name":"Overlap","targets":2},{"name":"Adjusted.P.value","targets":3},{"name":"Genes","targets":4}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>print(&quot;GO_Molecular_Function_2021&quot;)</code></pre>
<pre><code>[1] &quot;GO_Molecular_Function_2021&quot;</code></pre>
<pre class="r"><code>db &lt;- &quot;GO_Molecular_Function_2021&quot;
df &lt;- GO_enrichment[[db]]
print(plotEnrich(GO_enrichment[[db]]))</code></pre>
<p><img src="figure/cTWAS_LD_european_combined3_output.Rmd/GO_Molecular_Function_2021-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-GO_Molecular_Function_2021-1">
Past versions of GO_Molecular_Function_2021-1.png
</button>
</p>
<div id="fig-GO_Molecular_Function_2021-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/099102dbbcddba5380e899c2d37db5ecfc27e20d/docs/figure/cTWAS_LD_european_combined3_output.Rmd/GO_Molecular_Function_2021-1.png" target="_blank">099102d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>df &lt;- df[df$Adjusted.P.value&lt;0.05,c(&quot;Term&quot;, &quot;Overlap&quot;, &quot;Adjusted.P.value&quot;, &quot;Genes&quot;)]

DT::datatable(df,caption = htmltools::tags$caption( style = &#39;caption-side: topleft; text-align = left; color:black;&#39;,&#39;Enriched pathways from GO_Molecular_Function_2021&#39;),options = list(pageLength = 5) )</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-38f6575cbe0384702c9e" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-38f6575cbe0384702c9e">{"x":{"filter":"none","vertical":false,"caption":"<caption style=\"caption-side: topleft; text-align = left; color:black;\">Enriched pathways from GO_Molecular_Function_2021<\/caption>","data":[["1","2","3","4"],["RNA polymerase II cis-regulatory region sequence-specific DNA binding (GO:0000978)","cis-regulatory region sequence-specific DNA binding (GO:0000987)","RNA polymerase II transcription regulatory region sequence-specific DNA binding (GO:0000977)","DNA-binding transcription activator activity, RNA polymerase II-specific (GO:0001228)"],["17/1149","16/1149","16/1359","7/333"],[0.001044343674859992,0.002147871543011444,0.01037234400610471,0.02293438435546223],["TBX1;SMAD2;ARNT2;ZNF395;SATB1;ZBTB38;EBF2;FOXJ1;GRHL3;BHLHA15;TBX5;GRHL1;KLF7;TRIM24;ZNF568;ZNF236;PKNOX2","TBX1;SMAD2;ARNT2;ZNF395;SATB1;ZBTB38;EBF2;FOXJ1;GRHL3;BHLHA15;TBX5;GRHL1;KLF7;ZNF568;ZNF236;PKNOX2","TBX1;SMAD2;ARNT2;ZNF395;SATB1;ZBTB38;EBF2;FOXJ1;GRHL3;BHLHA15;TBX5;GRHL1;KLF7;ZNF568;ZNF236;PKNOX2","SMAD2;KLF7;ZNF395;FOXJ1;GRHL3;TBX5;GRHL1"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":3},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Term","targets":1},{"name":"Overlap","targets":2},{"name":"Adjusted.P.value","targets":3},{"name":"Genes","targets":4}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.1 (2024-06-14)
Platform: aarch64-apple-darwin20
Running under: macOS 15.1

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: America/New_York
tzcode source: internal

attached base packages:
[1] stats4    stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] biomaRt_2.60.1            karyoploteR_1.30.0       
 [3] regioneR_1.36.0           htmltools_0.5.8.1        
 [5] DT_0.33                   enrichR_3.2              
 [7] EnsDb.Hsapiens.v86_2.99.0 ensembldb_2.28.1         
 [9] AnnotationFilter_1.28.0   GenomicFeatures_1.56.0   
[11] AnnotationDbi_1.66.0      Biobase_2.64.0           
[13] GenomicRanges_1.56.2      GenomeInfoDb_1.40.1      
[15] IRanges_2.38.1            S4Vectors_0.42.1         
[17] BiocGenerics_0.50.0       ctwas_0.4.18             
[19] data.table_1.16.2         lubridate_1.9.3          
[21] forcats_1.0.0             stringr_1.5.1            
[23] dplyr_1.1.4               purrr_1.0.2              
[25] readr_2.1.5               tidyr_1.3.1              
[27] tibble_3.2.1              ggplot2_3.5.1            
[29] tidyverse_2.0.0          

loaded via a namespace (and not attached):
  [1] later_1.3.2                 BiocIO_1.14.0              
  [3] bitops_1.0-9                filelock_1.0.3             
  [5] XML_3.99-0.17               rpart_4.1.23               
  [7] lifecycle_1.0.4             httr2_1.0.5                
  [9] mixsqp_0.3-54               rprojroot_2.0.4            
 [11] lattice_0.22-6              crosstalk_1.2.1            
 [13] backports_1.5.0             magrittr_2.0.3             
 [15] locuszoomr_0.3.5            Hmisc_5.2-0                
 [17] plotly_4.10.4               sass_0.4.9                 
 [19] rmarkdown_2.28              jquerylib_0.1.4            
 [21] yaml_2.3.10                 httpuv_1.6.15              
 [23] cowplot_1.1.3               DBI_1.2.3                  
 [25] RColorBrewer_1.1-3          abind_1.4-8                
 [27] zlibbioc_1.50.0             pgenlibr_0.3.7             
 [29] biovizBase_1.52.0           RCurl_1.98-1.16            
 [31] nnet_7.3-19                 WriteXLS_6.7.0             
 [33] VariantAnnotation_1.50.0    rappdirs_0.3.3             
 [35] git2r_0.35.0                GenomeInfoDbData_1.2.12    
 [37] ggrepel_0.9.6               irlba_2.3.5.1              
 [39] codetools_0.2-20            DelayedArray_0.30.1        
 [41] xml2_1.3.6                  tidyselect_1.2.1           
 [43] UCSC.utils_1.0.0            farver_2.1.2               
 [45] matrixStats_1.4.1           BiocFileCache_2.12.0       
 [47] base64enc_0.1-3             bamsignals_1.36.0          
 [49] GenomicAlignments_1.40.0    jsonlite_1.8.9             
 [51] Formula_1.2-5               tools_4.4.1                
 [53] progress_1.2.3              Rcpp_1.0.13                
 [55] glue_1.8.0                  gridExtra_2.3              
 [57] SparseArray_1.4.8           xfun_0.48                  
 [59] MatrixGenerics_1.16.0       withr_3.0.2                
 [61] fastmap_1.2.0               LDlinkR_1.4.0              
 [63] fansi_1.0.6                 digest_0.6.37              
 [65] timechange_0.3.0            R6_2.5.1                   
 [67] colorspace_2.1-1            dichromat_2.0-0.1          
 [69] RSQLite_2.3.7               utf8_1.2.4                 
 [71] generics_0.1.3              rtracklayer_1.64.0         
 [73] prettyunits_1.2.0           httr_1.4.7                 
 [75] htmlwidgets_1.6.4           S4Arrays_1.4.1             
 [77] whisker_0.4.1               pkgconfig_2.0.3            
 [79] gtable_0.3.6                blob_1.2.4                 
 [81] workflowr_1.7.1             XVector_0.44.0             
 [83] gggrid_0.2-0                ProtGenerics_1.36.0        
 [85] scales_1.3.0                logging_0.10-108           
 [87] png_0.1-8                   knitr_1.48                 
 [89] rstudioapi_0.17.1           tzdb_0.4.0                 
 [91] rjson_0.2.23                checkmate_2.3.2            
 [93] curl_5.2.3                  cachem_1.1.0               
 [95] zoo_1.8-12                  parallel_4.4.1             
 [97] foreign_0.8-87              restfulr_0.0.15            
 [99] pillar_1.9.0                grid_4.4.1                 
[101] vctrs_0.6.5                 promises_1.3.0             
[103] dbplyr_2.5.0                cluster_2.1.6              
[105] htmlTable_2.4.3             evaluate_1.0.1             
[107] cli_3.6.3                   compiler_4.4.1             
[109] bezier_1.1.2                Rsamtools_2.20.0           
[111] rlang_1.1.4                 crayon_1.5.3               
[113] labeling_0.4.3              fs_1.6.5                   
[115] stringi_1.8.4               viridisLite_0.4.2          
[117] BiocParallel_1.38.0         munsell_0.5.1              
[119] Biostrings_2.72.1           lazyeval_0.2.2             
[121] Matrix_1.7-1                BSgenome_1.72.0            
[123] hms_1.1.3                   bit64_4.5.2                
[125] KEGGREST_1.44.1             SummarizedExperiment_1.34.0
[127] highr_0.11                  memoise_2.0.1              
[129] bslib_0.8.0                 bit_4.5.0                  </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
