<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Nina Yifan Yang" />

<meta name="date" content="2024-10-21" />

<title>Prostate Cancer ~ Predictdb eQTL&amp;sQTL from Prostate &amp; Testis &amp; Lymphocytes &amp; RefZ expression prostate cohort Chr 1+2+22 TEST RUN</title>

<script src="site_libs/header-attrs-2.28/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.33/datatables.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Multigroup_cTWAS</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/yy896/Multigroup_cTWAS">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">Prostate Cancer ~ Predictdb eQTL&amp;sQTL
from Prostate &amp; Testis &amp; Lymphocytes &amp; RefZ expression
prostate cohort Chr 1+2+22 TEST RUN</h1>
<h4 class="author">Nina Yifan Yang</h4>
<h4 class="date">2024-10-21</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-10-21
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Multigroup_cTWAS/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20241004code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20241004)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20241004code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20241004)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomyy896MultigroupcTWAStree3e3c45fc4ba1a457ea566dd35f6803ba80ffea49targetblank3e3c45fa">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/yy896/Multigroup_cTWAS/tree/3e3c45fc4ba1a457ea566dd35f6803ba80ffea49" target="_blank">3e3c45f</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomyy896MultigroupcTWAStree3e3c45fc4ba1a457ea566dd35f6803ba80ffea49targetblank3e3c45fa"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/yy896/Multigroup_cTWAS/tree/3e3c45fc4ba1a457ea566dd35f6803ba80ffea49" target="_blank">3e3c45f</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    analysis/.DS_Store
    Ignored:    analysis/.Rhistory

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown
(<code>analysis/cTWAS_LD_european_combined_output.Rmd</code>) and HTML
(<code>docs/cTWAS_LD_european_combined_output.html</code>) files. If
you’ve configured a remote Git repository (see
<code>?wflow_git_remote</code>), click on the hyperlinks in the table
below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/3e3c45fc4ba1a457ea566dd35f6803ba80ffea49/analysis/cTWAS_LD_european_combined_output.Rmd" target="_blank">3e3c45f</a>
</td>
<td>
yy896
</td>
<td>
2024-10-21
</td>
<td>
Update TEST RUN
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/yy896/Multigroup_cTWAS/6cc8b8cd13debb32ccae908ef010901528d52252/docs/cTWAS_LD_european_combined_output.html" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/bd0a6f3ab33dd36fd365cb451c8c99983b70e37b/analysis/cTWAS_LD_european_combined_output.Rmd" target="_blank">bd0a6f3</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
<td>
Update TEST RUN
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/yy896/Multigroup_cTWAS/7e4a02a27d0cc98583887f656d9ed222ac8070e5/docs/cTWAS_LD_european_combined_output.html" target="_blank">7e4a02a</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/f68bc7a2508f7e1f0cd961a8da4c9863e4a29e38/analysis/cTWAS_LD_european_combined_output.Rmd" target="_blank">f68bc7a</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
<td>
Updated TEST RUN
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/yy896/Multigroup_cTWAS/b6b2e9d193f8b35da919ac778a1f1e88634beb74/docs/cTWAS_LD_european_combined_output.html" target="_blank">b6b2e9d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/b8000e941ffbe88d3a5668150840cc956cd57c5f/analysis/cTWAS_LD_european_combined_output.Rmd" target="_blank">b8000e9</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
<td>
Updated TEST RUN
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/yy896/Multigroup_cTWAS/b53f41c3468e7e78241db3f085a7fc17baa86757/docs/cTWAS_LD_european_combined_output.html" target="_blank">b53f41c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/cea5fa5f015986dafbb7747d962dc26876310f8b/analysis/cTWAS_LD_european_combined_output.Rmd" target="_blank">cea5fa5</a>
</td>
<td>
yy896
</td>
<td>
2024-10-16
</td>
<td>
Update TEST RUN
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p>Here presenting a cTWAS results report based on real data analysis.
The analyzed trait is prostate cancer, the prediction models are
Prostate &amp; Testis &amp; Lymphocytes gene expression and splicing
models trained on GTEx v8 in the PredictDB format, and RefZ FUSION
weights expression data under NIH accession number phs000985.v2.p1.</p>
<div id="analysis-settings" class="section level2">
<h2>Analysis Settings</h2>
<div id="input-data" class="section level3">
<h3>Input Data</h3>
<ul>
<li>GWAS Z-scores</li>
</ul>
<p>The summary statistics for prostate cancer are downloaded from <a
href="https://www.ebi.ac.uk/gwas/downloads/summary-statistics"
class="uri">https://www.ebi.ac.uk/gwas/downloads/summary-statistics</a>,
using dataset ID: GCST90274714.</p>
<p>The sample size is</p>
<pre><code>[1] &quot;gwas_n = 726828&quot;</code></pre>
<ul>
<li>Prediction models</li>
</ul>
<p>The prediction models used in this analysis are prostate &amp; testis
&amp; lymphocytes gene expression and splicing models, trained on GTEx
v8 in the PredictDB format. These models can be downloaded from <a
href="https://predictdb.org/post/2021/07/21/gtex-v8-models-on-eqtl-and-sqtl/"
class="uri">https://predictdb.org/post/2021/07/21/gtex-v8-models-on-eqtl-and-sqtl/</a>.
The RefZ weights model can be downloaded from <a
href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000985.v2.p1"
class="uri">https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000985.v2.p1</a>.</p>
<ul>
<li>Reference data</li>
</ul>
<p>We did include ukb38 whole genome LD reference in the analysis.</p>
</div>
<div id="data-processing-and-harmonization" class="section level3">
<h3>Data processing and harmonization</h3>
<p>We map the reference SNPs to regions following the instructions from
the cTWAS tutorial.</p>
<p>When processing z-scores, we exclude multi-allelic and
strand-ambiguous variants by setting
<code>drop_multiallelic = TRUE</code> and
<code>drop_strand_ambig = TRUE</code>.</p>
<p>The process can be divided into steps below, users can expand the
code snippets below to view the exact code used.</p>
<ul>
<li>Input and output settings</li>
</ul>
<pre class="r"><code>weight_prostate_expression_file &lt;- system.file(&quot;extdata/project_data&quot;, 
                                               &quot;mashr_Prostate_e.db&quot;, package = &quot;ctwas&quot;)
weight_prostate_splicing_file &lt;- system.file(&quot;extdata/project_data&quot;, 
                                             &quot;mashr_Prostate_s.db&quot;, package = &quot;ctwas&quot;)
weight_testis_expression_file &lt;- system.file(&quot;extdata/project_data&quot;, 
                                             &quot;mashr_Testis_e.db&quot;, package = &quot;ctwas&quot;)
weight_testis_splicing_file &lt;- system.file(&quot;extdata/project_data&quot;, 
                                           &quot;mashr_Testis_s.db&quot;, package = &quot;ctwas&quot;)
weight_lymphocytes_expression_file &lt;- system.file(&quot;extdata/project_data&quot;, 
                                                  &quot;mashr_Cells_EBV-transformed_lymphocytes_e.db&quot;, package = &quot;ctwas&quot;)
weight_lymphocytes_splicing_file &lt;- system.file(&quot;extdata/project_data&quot;, 
                                                &quot;mashr_Cells_EBV-transformed_lymphocytes_s.db&quot;, package = &quot;ctwas&quot;)

z_snp &lt;- vroom::vroom(&quot;~/Desktop/Prostate Cancer Project/GCST90274714.mod.tsv.gz&quot;, col_names = TRUE, show_col_types = FALSE)
z_snp$Z &lt;- z_snp$beta / z_snp$standard_error

z_snp &lt;- z_snp[, c(&quot;rsid&quot;, &quot;other_allele&quot;, &quot;effect_allele&quot;, &quot;Z&quot;)]
colnames(z_snp) &lt;- c(&quot;id&quot;, &quot;A1&quot;, &quot;A2&quot;, &quot;z&quot;)

region_file &lt;- system.file(&quot;extdata/ldetect&quot;, &quot;EUR.b38.ldetect.regions.RDS&quot;, package = &quot;ctwas&quot;)
region_info &lt;- readRDS(region_file)

## other parameters
ncore &lt;- 6</code></pre>
<ul>
<li>Preprocessing GWAS</li>
</ul>
<pre class="r"><code>### Preprocess SNP_map and LD_map
LD_dir &lt;- &quot;~/Desktop/Prostate Cancer Project/LD_b38&quot;
genome_version &lt;- &quot;b38&quot;

LD_filestem &lt;- sprintf(&quot;ukb_%s_0.1_chr%s.R_snp.%s_%s&quot;, genome_version, region_info$chrom, region_info$start, region_info$stop)

region_metatable &lt;- region_info
region_metatable$LD_file &lt;- file.path(LD_dir, paste0(LD_filestem, &quot;.RDS&quot;))
region_metatable$SNP_file &lt;- file.path(LD_dir, paste0(LD_filestem, &quot;.Rvar&quot;))

res &lt;- create_snp_LD_map(region_metatable)
region_info &lt;- res$region_info
snp_map &lt;- res$snp_map
LD_map &lt;- res$LD_map

### Preprocess GWAS z-scores
z_snp &lt;- preprocess_z_snp(z_snp, snp_map, 
                          drop_multiallelic = TRUE, 
                          drop_strand_ambig = TRUE)</code></pre>
<ul>
<li>Preprocessing weights</li>
</ul>
<pre class="r"><code>weights_prostate_expression &lt;- preprocess_weights(weight_prostate_expression_file,
                                                  region_info,
                                                  gwas_snp_ids = z_snp$id,
                                                  snp_map = snp_map,
                                                  type = &quot;expression&quot;,
                                                  context = &quot;prostate&quot;,
                                                  weight_name = &quot;prostate_expression&quot;,
                                                  weight_format = &quot;PredictDB&quot;,
                                                  drop_strand_ambig = TRUE,
                                                  scale_predictdb_weights = TRUE,
                                                  load_predictdb_LD = TRUE,
                                                  filter_protein_coding_genes = TRUE,
                                                  ncore = 6)
weights_prostate_splicing &lt;- preprocess_weights(weight_prostate_splicing_file,
                                                region_info,
                                                gwas_snp_ids = z_snp$id,
                                                snp_map = snp_map,
                                                type = &quot;splicing&quot;,
                                                context = &quot;prostate&quot;,
                                                weight_name = &quot;prostate_splicing&quot;,
                                                weight_format = &quot;PredictDB&quot;,
                                                drop_strand_ambig = TRUE,
                                                scale_predictdb_weights = TRUE,
                                                load_predictdb_LD = TRUE,
                                                filter_protein_coding_genes = TRUE,
                                                ncore = 6)

weights_testis_expression &lt;- preprocess_weights(weight_testis_expression_file,
                                                region_info,
                                                gwas_snp_ids = z_snp$id,
                                                snp_map = snp_map,
                                                type = &quot;expression&quot;,
                                                context = &quot;testis&quot;,
                                                weight_name = &quot;testis_expression&quot;,
                                                weight_format = &quot;PredictDB&quot;,
                                                drop_strand_ambig = TRUE,
                                                scale_predictdb_weights = TRUE,
                                                load_predictdb_LD = TRUE,
                                                filter_protein_coding_genes = TRUE,
                                                ncore = 6)
weights_testis_splicing &lt;- preprocess_weights(weight_testis_splicing_file,
                                              region_info,
                                              gwas_snp_ids = z_snp$id,
                                              snp_map = snp_map,
                                              type = &quot;splicing&quot;,
                                              context = &quot;testis&quot;,
                                              weight_name = &quot;testis_splicing&quot;,
                                              weight_format = &quot;PredictDB&quot;,
                                              drop_strand_ambig = TRUE,
                                              scale_predictdb_weights = TRUE,
                                              load_predictdb_LD = TRUE,
                                              filter_protein_coding_genes = TRUE,
                                              ncore = 6)

weights_lymphocytes_expression &lt;- preprocess_weights(weight_lymphocytes_expression_file,
                                                     region_info,
                                                     gwas_snp_ids = z_snp$id,
                                                     snp_map = snp_map,
                                                     type = &quot;expression&quot;,
                                                     context = &quot;lymphocytes&quot;,
                                                     weight_name = &quot;lymphocytes_expression&quot;,
                                                     weight_format = &quot;PredictDB&quot;,
                                                     drop_strand_ambig = TRUE,
                                                     scale_predictdb_weights = TRUE,
                                                     load_predictdb_LD = TRUE,
                                                     filter_protein_coding_genes = TRUE,
                                                     ncore = 6)
weights_lymphocytes_splicing &lt;- preprocess_weights(weight_lymphocytes_splicing_file,
                                                   region_info,
                                                   gwas_snp_ids = z_snp$id,
                                                   snp_map = snp_map,
                                                   type = &quot;splicing&quot;,
                                                   context = &quot;lymphocytes&quot;,
                                                   weight_name = &quot;lymphocytes_splicing&quot;,
                                                   weight_format = &quot;PredictDB&quot;,
                                                   drop_strand_ambig = TRUE,
                                                   scale_predictdb_weights = TRUE,
                                                   load_predictdb_LD = TRUE,
                                                   filter_protein_coding_genes = TRUE,
                                                   ncore = 6)

weights_RefZ_prostate &lt;- preprocess_weights(&quot;~/Desktop/Prostate Cancer Project/MAYO/weights&quot;,
                              region_info,
                              gwas_snp_ids = z_snp$id,
                              snp_map = snp_map,
                              LD_map = LD_map,
                              type = &quot;RefZ_expression&quot;,
                              context = &quot;prostate&quot;,                         
                              weight_format = &quot;FUSION&quot;,
                              fusion_method = &quot;lasso&quot;,
                              fusion_genome_version = &quot;b38&quot;,
                              top_n_snps = NULL,
                              drop_strand_ambig = TRUE,
                              filter_protein_coding_genes = FALSE,
                              scale_predictdb_weights = FALSE,
                              load_predictdb_LD = FALSE,
                              ncore = 4)

weights_RefZ_prostate &lt;- preprocess_weights(&quot;~/Desktop/Prostate Cancer Project/MAYO/weights&quot;,
                              region_info,
                              gwas_snp_ids = z_snp$id,
                              snp_map = snp_map,
                              LD_map = LD_map,
                              type = &quot;RefZ_expression&quot;,
                              context = &quot;prostate&quot;,                         
                              weight_format = &quot;FUSION&quot;,
                              fusion_method = &quot;lasso&quot;,
                              fusion_genome_version = &quot;b38&quot;,
                              top_n_snps = NULL,
                              drop_strand_ambig = TRUE,
                              filter_protein_coding_genes = FALSE,
                              scale_predictdb_weights = FALSE,
                              load_predictdb_LD = FALSE,
                              ncore = 4)

weights &lt;- c(
  weights_prostate_expression,
  weights_prostate_splicing,
  weights_testis_expression,
  weights_testis_splicing,
  weights_lymphocytes_expression,
  weights_lymphocytes_splicing,
  weights_RefZ_prostate)</code></pre>
</div>
</div>
<div id="running-ctwas-analysis" class="section level2">
<h2>Running cTWAS analysis</h2>
<p>We use the ctwas main function <code>ctwas_sumstats</code> to run the
cTWAS analysis with LD. For more details on this function, refer to the
cTWAS tutorial: <a
href="https://xinhe-lab.github.io/multigroup_ctwas/articles/running_ctwas_analysis.html"
class="uri">https://xinhe-lab.github.io/multigroup_ctwas/articles/running_ctwas_analysis.html</a></p>
<p>All arguments are set to their default values.</p>
<p>Users can expand the code snippets below to view the exact code
used.</p>
<pre class="r"><code>ctwas_res &lt;- ctwas_sumstats(z_snp, 
                            weights, 
                            region_info, 
                            LD_map, 
                            snp_map, 
                            thin = 0.1,
                            maxSNP = 20000,
                            group_prior_var_structure = &quot;shared_type&quot;, 
                            filter_L = TRUE,
                            filter_nonSNP_PIP = FALSE,
                            min_nonSNP_PIP = 0.5,
                            min_abs_corr = 0.1, 
                            ncore = 6, 
                            ncore_LD = 4,
                            save_cor = TRUE,
                            cor_dir = &quot;./cor_matrix&quot;,
                            force_compute_cor = FALSE)</code></pre>
</div>
<div id="parameter-estimation" class="section level2">
<h2>Parameter estimation</h2>
<p><code>ctwas_res</code> is the object contains the outputs of
cTWAS</p>
<p>We extract the estimated parameters by
<code>param &lt;- ctwas_res$param</code></p>
<p>We make plots using the function
<code>make_convergence_plots(param, gwas_n)</code> to see how estimated
parameters converge during the execution of the program:</p>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/unnamed-chunk-7-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-7-1">
Past versions of unnamed-chunk-7-1.png
</button>
</p>
<div id="fig-unnamed-chunk-7-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/unnamed-chunk-7-1.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/b53f41c3468e7e78241db3f085a7fc17baa86757/docs/figure/cTWAS_LD_european_combined_output.Rmd/unnamed-chunk-7-1.png" target="_blank">b53f41c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>These plots show the estimated prior inclusion probability, prior
effect size variance, enrichment and proportion of variance explained
(PVE) over the iterations of parameter estimation. The enrichment is
defined as the ratio of the prior inclusion probability of molecular
traits over the prior inclusion probability of variants. We generally
expect molecular traits to have higher prior inclusion probability than
variants. Enrichment values typically range from 20 - 100 for expression
traits.</p>
<p>Then, we use <code>summarize_param(param, gwas_n)</code> to obtain
estimated parameters (from the last iteration) and to compute the PVE by
variants and molecular traits.</p>
<pre><code>[1] &quot;The number of genes/introns/SNPs used in the analysis is:&quot;</code></pre>
<pre><code>   prostate|mashr_expression      prostate|mashr_splicing 
                        1902                         5813 
     testis|mashr_expression        testis|mashr_splicing 
                        2232                         9515 
lymphocytes|mashr_expression   lymphocytes|mashr_splicing 
                        1650                         5115 
    prostate|RefZ_expression                          SNP 
                        2519                      1246380 </code></pre>
<p><code>ctwas_parameters$prop_heritability</code> contains the
proportion of heritability mediated by molecular traits and variants, we
visualize it using pie chart.</p>
<pre class="r"><code>ctwas_parameters &lt;- summarize_param(param, gwas_n)
data &lt;- data.frame(
  category = names(ctwas_parameters$prop_heritability),
  percentage = ctwas_parameters$prop_heritability
)
# Calculate percentage labels for the chart
data$percentage_label &lt;- paste0(round(data$percentage * 100, 1), &quot;%&quot;)

ggplot(data, aes(x = &quot;&quot;, y = percentage, fill = category)) +
  geom_bar(stat = &quot;identity&quot;, width = 1) +
  coord_polar(&quot;y&quot;, start = 0) +
  theme_void() +  # Remove background and axes
  geom_text(aes(label = percentage_label), 
            position = position_stack(vjust = 0.5)) +
  scale_fill_brewer(palette = &quot;Pastel1&quot;) +
  labs(fill = &quot;Category&quot;) +
  ggtitle(&quot;Percent of heritability&quot;)</code></pre>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/piechart-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-piechart-1">
Past versions of piechart-1.png
</button>
</p>
<div id="fig-piechart-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/piechart-1.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/b53f41c3468e7e78241db3f085a7fc17baa86757/docs/figure/cTWAS_LD_european_combined_output.Rmd/piechart-1.png" target="_blank">b53f41c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="diagnosis-plots" class="section level2">
<h2>Diagnosis plots</h2>
<p>For all genes analyzed, we compare the z-scores and fine-mapping
PIPs”</p>
<pre class="r"><code>ggplot(data = finemap_res[finemap_res$type!=&quot;SNP&quot;,], aes(x = abs(z), y = susie_pip)) +
  geom_point() +
  labs(x = &quot;abs(z-scores)&quot;, y = &quot;PIPs&quot;) +
  theme_minimal()</code></pre>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/unnamed-chunk-9-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-9-1">
Past versions of unnamed-chunk-9-1.png
</button>
</p>
<div id="fig-unnamed-chunk-9-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/unnamed-chunk-9-1.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/b53f41c3468e7e78241db3f085a7fc17baa86757/docs/figure/cTWAS_LD_european_combined_output.Rmd/unnamed-chunk-9-1.png" target="_blank">b53f41c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="fine-mapping-results" class="section level2">
<h2>Fine-mapping results</h2>
<p>We process the fine-mapping results here.</p>
<p>We first add gene annotations to cTWAS results.</p>
<p><code>mapping_table</code> is a combination of expression, splicing
models and RefZ expression data.</p>
<pre class="r"><code>finemap_res &lt;- anno_finemap_res(finemap_res,
                                snp_map = snp_map,
                                mapping_table = mapping_table,
                                add_gene_annot = TRUE,
                                map_by = &quot;molecular_id&quot;,
                                drop_unmapped = TRUE,
                                add_position = TRUE,
                                use_gene_pos = &quot;mid&quot;)</code></pre>
<p>The annotated fine-mapping results, ones within credible sets are
shown</p>
<pre class="r"><code>finemap_res_show &lt;- subset(finemap_res, group != &quot;SNP&quot; &amp; gene_type == &quot;protein_coding&quot; &amp; susie_pip &gt; 0.8 &amp; !is.na(cs))

DT::datatable(finemap_res_show,caption = htmltools::tags$caption( style = &#39;caption-side: topleft; text-align = left; color:black;&#39;,&#39;The annotated fine-mapping results, ones within credible sets are shown&#39;),options = list(pageLength = 5) )</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-232a8e63d6aacd296363" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-232a8e63d6aacd296363">{"x":{"filter":"none","vertical":false,"caption":"<caption style=\"caption-side: topleft; text-align = left; color:black;\">The annotated fine-mapping results, ones within credible sets are shown<\/caption>","data":[["913","3010","3894","4987","7241","7434","7486","9878","11236","14923","14981","15365","15430","15634","16086"],["C1orf64|prostate_RefZ_expression","GPBP1L1|prostate_RefZ_expression","LMO4|prostate_RefZ_expression","ENSG00000143545.8|lymphocytes_expression","ENSG00000133056.13|lymphocytes_expression","NUCKS1|prostate_RefZ_expression","intron_1_206604729_206605400|prostate_splicing","GDF7|prostate_RefZ_expression","TGFA|prostate_RefZ_expression","intron_2_207165706_207166757|lymphocytes_splicing","ENSG00000030419.16|lymphocytes_expression","ENSG00000115648.13|lymphocytes_expression","intron_2_241195845_241199316|prostate_splicing","ENSG00000168385.17|prostate_expression","FARP2|prostate_RefZ_expression"],["C1orf64","GPBP1L1","LMO4","ENSG00000143545.8","ENSG00000133056.13","NUCKS1","intron_1_206604729_206605400","GDF7","TGFA","intron_2_207165706_207166757","ENSG00000030419.16","ENSG00000115648.13","intron_2_241195845_241199316","ENSG00000168385.17","FARP2"],["RefZ_expression","RefZ_expression","RefZ_expression","mashr_expression","mashr_expression","RefZ_expression","mashr_splicing","RefZ_expression","RefZ_expression","mashr_splicing","mashr_expression","mashr_expression","mashr_splicing","mashr_expression","RefZ_expression"],["prostate","prostate","prostate","lymphocytes","lymphocytes","prostate","prostate","prostate","prostate","lymphocytes","lymphocytes","lymphocytes","prostate","prostate","prostate"],["prostate|RefZ_expression","prostate|RefZ_expression","prostate|RefZ_expression","lymphocytes|mashr_expression","lymphocytes|mashr_expression","prostate|RefZ_expression","prostate|mashr_splicing","prostate|RefZ_expression","prostate|RefZ_expression","lymphocytes|mashr_splicing","lymphocytes|mashr_expression","lymphocytes|mashr_expression","prostate|mashr_splicing","prostate|mashr_expression","prostate|RefZ_expression"],["1_14565015_16571235","1_44503511_46433829","1_87662948_89600744","1_153208353_154797927","1_203365606_204711940","1_204711940_206268085","1_206268085_208237019","2_19492643_20850730","2_70528066_72947719","2_205870980_207780675","2_213149558_214709071","2_236540389_237852338","2_240620667_241210506","2_241210506_242147200","2_241210506_242147200"],[-4.989644773803413,-6.555406983168245,8.87652774617216,12.14221990404148,14.44444444444444,-7.243798963735523,-4.497963794928934,13.30267040791562,4.610779079266881,-6.192982456140351,-4.63076923076923,-10.35733908051552,13.791151879786,10.20768686024468,5.618475411033651],[0.8599377000063757,0.9492881022605367,0.9591952560965179,0.9999977920731681,0.9902064448313793,0.93408959373028,0.8419950340348963,0.9966082541790784,0.9902032044575367,0.9325821977439218,0.9231172470687152,0.9999972722896547,0.9999999999999999,0.9152335361384245,0.9999999618160373],[24.66694559310701,2005.568950986748,65.12775998677347,192.4495537142843,141.7641826125766,48.72085259117521,226.647747701363,245.3159125368183,20.30680344864259,37.67969395626817,21.79368011117873,666.2242819960173,226.7722245421143,157.0715359162332,81.6131077208276],["L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L3","L2"],[6.049042523626976e-07,5.549032902243167e-11,6.897993299585749e-19,6.309078982351151e-34,2.717395018514352e-47,4.362879835858363e-13,6.86073768458958e-06,2.233465266839708e-40,4.0116268606296e-06,5.903630251842007e-10,3.643097322721326e-06,3.875947638287908e-25,2.881223371161483e-43,1.83179643941102e-24,1.92649805251018e-08],["C1orf64","GPBP1L1","LMO4","RAB13","PIK3C2B","NUCKS1","EIF2D","GDF7","TGFA","MIR7845","IKZF2","MLPH","ANO7","SEPT2","FARP2"],["protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding"],["1","1","1","1","1","1","1","2","2","2","2","2","2","2","2"],["14565015","44503511","87662948","153981617","204422628","204711940","206604729","19492643","70528066","207165706","212999691","237485428","241195845","241315100","241210506"],["16571235","46433829","89600744","153986358","204494724","206268085","206605400","20850730","72947719","207166757","213152427","237555318","241199316","241354027","242147200"],[15568125,45468670,88631846,153983988,204458676,205490012,206605064,20171686,71737892,207166232,213076059,237520373,241197580,241334564,241678853]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>id<\/th>\n      <th>molecular_id<\/th>\n      <th>type<\/th>\n      <th>context<\/th>\n      <th>group<\/th>\n      <th>region_id<\/th>\n      <th>z<\/th>\n      <th>susie_pip<\/th>\n      <th>mu2<\/th>\n      <th>cs<\/th>\n      <th>pval<\/th>\n      <th>gene_name<\/th>\n      <th>gene_type<\/th>\n      <th>chrom<\/th>\n      <th>start<\/th>\n      <th>end<\/th>\n      <th>pos<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":[7,8,9,11,17]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"id","targets":1},{"name":"molecular_id","targets":2},{"name":"type","targets":3},{"name":"context","targets":4},{"name":"group","targets":5},{"name":"region_id","targets":6},{"name":"z","targets":7},{"name":"susie_pip","targets":8},{"name":"mu2","targets":9},{"name":"cs","targets":10},{"name":"pval","targets":11},{"name":"gene_name","targets":12},{"name":"gene_type","targets":13},{"name":"chrom","targets":14},{"name":"start","targets":15},{"name":"end","targets":16},{"name":"pos","targets":17}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script>
<p>Next, we compute gene PIPs across different types of molecular
traits.</p>
<pre class="r"><code>library(dplyr)

susie_alpha_res &lt;- ctwas_res$susie_alpha_res

susie_alpha_res &lt;- anno_susie_alpha_res(susie_alpha_res,
                                        mapping_table = mapping_table,
                                        map_by = &quot;molecular_id&quot;,
                                        drop_unmapped = TRUE)</code></pre>
<p>Gene PIPs, only genes within credible sets are shown</p>
<pre class="r"><code>library(dplyr)
susie_alpha_res &lt;- anno_susie_alpha_res(susie_alpha_res,
                                        mapping_table = mapping_table,
                                        map_by = &quot;molecular_id&quot;,
                                        drop_unmapped = TRUE)</code></pre>
<pre class="r"><code>combined_pip_by_group &lt;- combine_gene_pips(susie_alpha_res,
                                          group_by = &quot;gene_name&quot;,
                                          by = &quot;group&quot;,
                                          method = &quot;combine_cs&quot;,
                                          filter_cs = TRUE,
                                          include_cs_id = TRUE)
combined_pip_table &lt;- subset(combined_pip_by_group, combined_pip &gt; 0.8)
DT::datatable(combined_pip_table)</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-2112c1f1dbc2c6427f4c" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-2112c1f1dbc2c6427f4c">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23"],["ANO7","EVI5","FARP2","RAB13","MLPH","GPBP1L1","EIF2D","HSCB","NOL10","GDF7","TBX1","PIK3C2B","TGFA","MCAT","LMO4","CASP8","NUCKS1","MIR7845","LDAH","IKZF2","SEPT2","C1orf64","GRHL1"],["2_240620667_241210506.L1","1_91420760_93070902.L1,L2,1_91420760_93070902.L2,1_91420760_93070902.L1","2_241210506_242147200.L2","1_153208353_154797927.L1","2_236540389_237852338.L3,2_236540389_237852338.L1","1_44503511_46433829.L4,1_44503511_46433829.L1","1_206268085_208237019.L3,1_206268085_208237019.L1","22_27438791_29255810.L1","2_10158342_12279024.L3,2_10158342_12279024.L1","2_19492643_20850730.L1","22_17813322_19924835.L1","1_203365606_204711940.L1","2_70528066_72947719.L1","22_42294812_43318194.L2","1_87662948_89600744.L1","2_200711561_201953914.L1","1_204711940_206268085.L1","2_205870980_207780675.L1","2_19492643_20850730.L3","2_213149558_214709071.L1","2_241210506_242147200.L3","1_14565015_16571235.L1","2_8282017_10158342.L1"],[0.9999999999999999,0.9999999995590165,0.9999999618160373,0.9999977920731681,0.9999973343155524,0.9999930587932997,0.9999930124145712,0.9994774573351181,0.999056121931262,0.9966082541790784,0.9959470148557381,0.9902064448313793,0.9902032044575367,0.9812691606524069,0.9591952560965179,0.9342459990456946,0.93408959373028,0.9325821977439218,0.9267823849228,0.9231172470687152,0.9152335361384245,0.8599377000063757,0.8376129560439618],[null,"1_91420760_93070902.L1,L2",null,null,null,null,"1_206268085_208237019.L3",null,null,null,"22_17813322_19924835.L1",null,null,"22_42294812_43318194.L2",null,null,null,null,null,null,"2_241210506_242147200.L3",null,"2_8282017_10158342.L1"],[null,0.3761051476899059,null,null,null,null,0.1732919674189728,null,null,null,0.3367611125625962,null,null,0.2123234487854923,null,null,null,null,null,null,0.9152335361384245,null,0.5711261644379302],["2_240620667_241210506.L1","1_91420760_93070902.L2",null,null,null,"1_44503511_46433829.L4","1_206268085_208237019.L3,1_206268085_208237019.L1","22_27438791_29255810.L1","2_10158342_12279024.L3",null,null,null,null,null,null,null,null,null,"2_19492643_20850730.L3",null,null,null,"2_8282017_10158342.L1"],[0.9999999999999999,0.8700360454593817,null,null,null,0.07761534526395319,0.9998816027696937,0.3884125247965647,0.9990543404462718,null,null,null,null,null,null,null,null,null,0.3609000450461165,null,null,null,0.2672966279184327],[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],[null,null,null,null,"2_236540389_237852338.L3",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],[null,null,null,null,0.02273622150004129,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],[null,"1_91420760_93070902.L1",null,"1_153208353_154797927.L1","2_236540389_237852338.L1","1_44503511_46433829.L4",null,null,null,null,null,"1_203365606_204711940.L1",null,"22_42294812_43318194.L2",null,null,null,null,null,"2_213149558_214709071.L1",null,null,null],[null,0.6872230672980115,null,0.9999977920731681,0.9999972722896547,0.09976583449079957,null,null,null,null,null,0.9902064448313793,null,0.7689457118669146,null,null,null,null,null,0.9231172470687152,null,null,null],[null,null,null,null,null,"1_44503511_46433829.L4","1_206268085_208237019.L3","22_27438791_29255810.L1","2_10158342_12279024.L1",null,null,null,null,null,null,"2_200711561_201953914.L1",null,"2_205870980_207780675.L1","2_19492643_20850730.L3",null,null,null,null],[null,null,null,null,null,0.8232688070745292,0.4792922291363756,0.6110649325385534,0.001883854483639258,null,null,null,null,null,null,0.9342459990456946,null,0.9325821977439218,0.5673581500311765,null,null,null,null],[null,null,"2_241210506_242147200.L2",null,null,"1_44503511_46433829.L1",null,null,null,"2_19492643_20850730.L1","22_17813322_19924835.L1",null,"2_70528066_72947719.L1",null,"1_87662948_89600744.L1",null,"1_204711940_206268085.L1",null,null,null,null,"1_14565015_16571235.L1",null],[null,null,0.9999999618160373,null,null,0.9492881022605367,null,null,null,0.9966082541790784,0.6591859022931419,null,0.9902032044575367,null,0.9591952560965179,null,0.93408959373028,null,null,null,null,0.8599377000063757,null]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>gene_name<\/th>\n      <th>combined_cs_id<\/th>\n      <th>combined_pip<\/th>\n      <th>prostate|mashr_expression_cs_id<\/th>\n      <th>prostate|mashr_expression_pip<\/th>\n      <th>prostate|mashr_splicing_cs_id<\/th>\n      <th>prostate|mashr_splicing_pip<\/th>\n      <th>testis|mashr_expression_cs_id<\/th>\n      <th>testis|mashr_expression_pip<\/th>\n      <th>testis|mashr_splicing_cs_id<\/th>\n      <th>testis|mashr_splicing_pip<\/th>\n      <th>lymphocytes|mashr_expression_cs_id<\/th>\n      <th>lymphocytes|mashr_expression_pip<\/th>\n      <th>lymphocytes|mashr_splicing_cs_id<\/th>\n      <th>lymphocytes|mashr_splicing_pip<\/th>\n      <th>prostate|RefZ_expression_cs_id<\/th>\n      <th>prostate|RefZ_expression_pip<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[3,5,7,9,11,13,15,17]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"gene_name","targets":1},{"name":"combined_cs_id","targets":2},{"name":"combined_pip","targets":3},{"name":"prostate|mashr_expression_cs_id","targets":4},{"name":"prostate|mashr_expression_pip","targets":5},{"name":"prostate|mashr_splicing_cs_id","targets":6},{"name":"prostate|mashr_splicing_pip","targets":7},{"name":"testis|mashr_expression_cs_id","targets":8},{"name":"testis|mashr_expression_pip","targets":9},{"name":"testis|mashr_splicing_cs_id","targets":10},{"name":"testis|mashr_splicing_pip","targets":11},{"name":"lymphocytes|mashr_expression_cs_id","targets":12},{"name":"lymphocytes|mashr_expression_pip","targets":13},{"name":"lymphocytes|mashr_splicing_cs_id","targets":14},{"name":"lymphocytes|mashr_splicing_pip","targets":15},{"name":"prostate|RefZ_expression_cs_id","targets":16},{"name":"prostate|RefZ_expression_pip","targets":17}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="gsea-gene-ontology" class="section level2">
<h2>GSEA-Gene Ontology</h2>
<pre class="r"><code>library(enrichR)

dbs &lt;- c(&quot;GO_Biological_Process_2021&quot;, &quot;GO_Cellular_Component_2021&quot;, &quot;GO_Molecular_Function_2021&quot;)
genes &lt;- combined_pip_by_group$gene_name[combined_pip_by_group$combined_pip &gt;0.8]

#number of genes for gene set enrichment
sprintf(&quot;The number of genes used in enrichment analysis = %s&quot;, length(genes))</code></pre>
<pre><code>[1] &quot;The number of genes used in enrichment analysis = 23&quot;</code></pre>
<pre class="r"><code>GO_enrichment &lt;- enrichr(genes, dbs)</code></pre>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2021... Done.
  Querying GO_Cellular_Component_2021... Done.
  Querying GO_Molecular_Function_2021... Done.
Parsing results... Done.</code></pre>
<pre class="r"><code>print(&quot;GO_Biological_Process_2021&quot;)</code></pre>
<pre><code>[1] &quot;GO_Biological_Process_2021&quot;</code></pre>
<pre class="r"><code>db &lt;- &quot;GO_Biological_Process_2021&quot;
df &lt;- GO_enrichment[[db]]
print(plotEnrich(GO_enrichment[[db]]))</code></pre>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/GO_Biological_Process_2021-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-GO_Biological_Process_2021-1">
Past versions of GO_Biological_Process_2021-1.png
</button>
</p>
<div id="fig-GO_Biological_Process_2021-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/GO_Biological_Process_2021-1.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/b53f41c3468e7e78241db3f085a7fc17baa86757/docs/figure/cTWAS_LD_european_combined_output.Rmd/GO_Biological_Process_2021-1.png" target="_blank">b53f41c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>df &lt;- df[df$Adjusted.P.value&lt;0.05,c(&quot;Term&quot;, &quot;Overlap&quot;, &quot;Adjusted.P.value&quot;, &quot;Genes&quot;)]

DT::datatable(df,caption = htmltools::tags$caption( style = &#39;caption-side: topleft; text-align = left; color:black;&#39;,&#39;Enriched pathways from GO_Biological_Process_2021&#39;),options = list(pageLength = 5) )</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-57a5e797e447beab7be8" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-57a5e797e447beab7be8">{"x":{"filter":"none","vertical":false,"caption":"<caption style=\"caption-side: topleft; text-align = left; color:black;\">Enriched pathways from GO_Biological_Process_2021<\/caption>","data":[[],[],[],[],[]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":3},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Term","targets":1},{"name":"Overlap","targets":2},{"name":"Adjusted.P.value","targets":3},{"name":"Genes","targets":4}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>print(&quot;GO_Cellular_Component_2021&quot;)</code></pre>
<pre><code>[1] &quot;GO_Cellular_Component_2021&quot;</code></pre>
<pre class="r"><code>db &lt;- &quot;GO_Cellular_Component_2021&quot;
df &lt;- GO_enrichment[[db]]
print(plotEnrich(GO_enrichment[[db]]))</code></pre>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/GO_Cellular_Component_2021-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-GO_Cellular_Component_2021-1">
Past versions of GO_Cellular_Component_2021-1.png
</button>
</p>
<div id="fig-GO_Cellular_Component_2021-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/GO_Cellular_Component_2021-1.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/b53f41c3468e7e78241db3f085a7fc17baa86757/docs/figure/cTWAS_LD_european_combined_output.Rmd/GO_Cellular_Component_2021-1.png" target="_blank">b53f41c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>df &lt;- df[df$Adjusted.P.value&lt;0.05,c(&quot;Term&quot;, &quot;Overlap&quot;, &quot;Adjusted.P.value&quot;, &quot;Genes&quot;)]

DT::datatable(df,caption = htmltools::tags$caption( style = &#39;caption-side: topleft; text-align = left; color:black;&#39;,&#39;Enriched pathways from GO_Cellular_Component_2021&#39;),options = list(pageLength = 5) )</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-0a34d9ca313023088b60" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-0a34d9ca313023088b60">{"x":{"filter":"none","vertical":false,"caption":"<caption style=\"caption-side: topleft; text-align = left; color:black;\">Enriched pathways from GO_Cellular_Component_2021<\/caption>","data":[[],[],[],[],[]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":3},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Term","targets":1},{"name":"Overlap","targets":2},{"name":"Adjusted.P.value","targets":3},{"name":"Genes","targets":4}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>print(&quot;GO_Molecular_Function_2021&quot;)</code></pre>
<pre><code>[1] &quot;GO_Molecular_Function_2021&quot;</code></pre>
<pre class="r"><code>db &lt;- &quot;GO_Molecular_Function_2021&quot;
df &lt;- GO_enrichment[[db]]
print(plotEnrich(GO_enrichment[[db]]))</code></pre>
<pre><code>Warning in plotEnrich(GO_enrichment[[db]]): There are duplicated trimmed names
in the plot, consider increasing the &#39;numChar&#39; setting.</code></pre>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/GO_Molecular_Function_2021-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-GO_Molecular_Function_2021-1">
Past versions of GO_Molecular_Function_2021-1.png
</button>
</p>
<div id="fig-GO_Molecular_Function_2021-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/GO_Molecular_Function_2021-1.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/b53f41c3468e7e78241db3f085a7fc17baa86757/docs/figure/cTWAS_LD_european_combined_output.Rmd/GO_Molecular_Function_2021-1.png" target="_blank">b53f41c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>df &lt;- df[df$Adjusted.P.value&lt;0.05,c(&quot;Term&quot;, &quot;Overlap&quot;, &quot;Adjusted.P.value&quot;, &quot;Genes&quot;)]

DT::datatable(df,caption = htmltools::tags$caption( style = &#39;caption-side: topleft; text-align = left; color:black;&#39;,&#39;Enriched pathways from GO_Molecular_Function_2021&#39;),options = list(pageLength = 5) )</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-38f6575cbe0384702c9e" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-38f6575cbe0384702c9e">{"x":{"filter":"none","vertical":false,"caption":"<caption style=\"caption-side: topleft; text-align = left; color:black;\">Enriched pathways from GO_Molecular_Function_2021<\/caption>","data":[[],[],[],[],[]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":3},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Term","targets":1},{"name":"Overlap","targets":2},{"name":"Adjusted.P.value","targets":3},{"name":"Genes","targets":4}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="locus-plots-from-highest-pip-to-lowest-0.8-cutoff"
class="section level2">
<h2>Locus plots (from highest PIP to lowest 0.8 cutoff)</h2>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;2_240620667_241210506&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-21 13:18:24.108784 INFO::Limit to protein coding genes
2024-10-21 13:18:24.112868 INFO::focal id: intron_2_241195845_241199316|prostate_splicing
2024-10-21 13:18:24.116613 INFO::focal molecular trait: ANO7 prostate mashr_splicing
2024-10-21 13:18:24.116872 INFO::Range of locus: chr2:240618245-241608572</code></pre>
<pre><code>chromosome 2, position 240618245 to 241608572</code></pre>
<pre><code>2181 SNPs/datapoints</code></pre>
<pre><code>2024-10-21 13:18:24.362069 INFO::focal molecular trait QTL positions: 241195850,241202304
2024-10-21 13:18:24.376352 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 224 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/locus%20plot-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-1">
Past versions of "locus plot-1.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-1.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/b6b2e9d193f8b35da919ac778a1f1e88634beb74/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-1.png" target="_blank">b6b2e9d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;2_241210506_242147200&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-21 13:18:25.088354 INFO::Limit to protein coding genes
2024-10-21 13:18:25.089054 INFO::focal id: FARP2|prostate_RefZ_expression
2024-10-21 13:18:25.089306 INFO::focal molecular trait: FARP2 prostate RefZ_expression
2024-10-21 13:18:25.089563 INFO::Range of locus: chr2:241206843-242115497</code></pre>
<pre><code>chromosome 2, position 241206843 to 242115497</code></pre>
<pre><code>2572 SNPs/datapoints</code></pre>
<pre><code>2024-10-21 13:18:25.213114 INFO::focal molecular trait QTL positions: 241222776,241234501,241245223,241345230,241375608,241422182,241493933,241497112,241503095,241526862,241533917,241664954
2024-10-21 13:18:25.226104 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 452 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/locus%20plot-2.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-2">
Past versions of "locus plot-2.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-2.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/b6b2e9d193f8b35da919ac778a1f1e88634beb74/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-2.png" target="_blank">b6b2e9d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;1_153208353_154797927&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-21 13:18:26.038685 INFO::Limit to protein coding genes
2024-10-21 13:18:26.039348 INFO::focal id: ENSG00000143545.8|lymphocytes_expression
2024-10-21 13:18:26.039606 INFO::focal molecular trait: RAB13 lymphocytes mashr_expression
2024-10-21 13:18:26.03987 INFO::Range of locus: chr1:153208786-154797763</code></pre>
<pre><code>chromosome 1, position 153208786 to 154797763</code></pre>
<pre><code>3665 SNPs/datapoints</code></pre>
<pre><code>2024-10-21 13:18:26.165745 INFO::focal molecular trait QTL positions: 153985724,153989308
2024-10-21 13:18:26.179099 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 455 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/locus%20plot-3.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-3">
Past versions of "locus plot-3.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-3.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/b6b2e9d193f8b35da919ac778a1f1e88634beb74/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-3.png" target="_blank">b6b2e9d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;2_236540389_237852338&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-21 13:18:27.143328 INFO::Limit to protein coding genes
2024-10-21 13:18:27.143913 INFO::focal id: ENSG00000115648.13|lymphocytes_expression
2024-10-21 13:18:27.144169 INFO::focal molecular trait: MLPH lymphocytes mashr_expression
2024-10-21 13:18:27.144437 INFO::Range of locus: chr2:236541810-237851575</code></pre>
<pre><code>chromosome 2, position 236541810 to 237851575</code></pre>
<pre><code>4218 SNPs/datapoints</code></pre>
<pre><code>2024-10-21 13:18:27.211308 INFO::focal molecular trait QTL positions: 237484645,237484791
2024-10-21 13:18:27.224356 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 136 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/locus%20plot-4.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-4">
Past versions of "locus plot-4.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-4.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/b6b2e9d193f8b35da919ac778a1f1e88634beb74/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-4.png" target="_blank">b6b2e9d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;2_19492643_20850730&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-21 13:18:27.845781 INFO::Limit to protein coding genes
2024-10-21 13:18:27.846389 INFO::focal id: GDF7|prostate_RefZ_expression
2024-10-21 13:18:27.846641 INFO::focal molecular trait: GDF7 prostate RefZ_expression
2024-10-21 13:18:27.846915 INFO::Range of locus: chr2:19352676-20849469</code></pre>
<pre><code>chromosome 2, position 19352676 to 20849469</code></pre>
<pre><code>4166 SNPs/datapoints</code></pre>
<pre><code>2024-10-21 13:18:27.913482 INFO::focal molecular trait QTL positions: 20241617,20677676,20678345,20678393,20681073
2024-10-21 13:18:27.926371 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 126 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<pre><code>Warning: ggrepel: 6 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/locus%20plot-5.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-5">
Past versions of "locus plot-5.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-5.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/b6b2e9d193f8b35da919ac778a1f1e88634beb74/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-5.png" target="_blank">b6b2e9d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;1_203365606_204711940&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-21 13:18:28.514072 INFO::Limit to protein coding genes
2024-10-21 13:18:28.51465 INFO::focal id: ENSG00000133056.13|lymphocytes_expression
2024-10-21 13:18:28.514898 INFO::focal molecular trait: PIK3C2B lymphocytes mashr_expression
2024-10-21 13:18:28.515176 INFO::Range of locus: chr1:203365611-204711982</code></pre>
<pre><code>chromosome 1, position 203365611 to 204711982</code></pre>
<pre><code>3619 SNPs/datapoints</code></pre>
<pre><code>2024-10-21 13:18:28.588323 INFO::focal molecular trait QTL positions: 204495053
2024-10-21 13:18:28.601863 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 149 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/locus%20plot-6.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-6">
Past versions of "locus plot-6.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-6.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/b6b2e9d193f8b35da919ac778a1f1e88634beb74/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-6.png" target="_blank">b6b2e9d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;2_70528066_72947719&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-21 13:18:29.254166 INFO::Limit to protein coding genes
2024-10-21 13:18:29.254824 INFO::focal id: TGFA|prostate_RefZ_expression
2024-10-21 13:18:29.255104 INFO::focal molecular trait: TGFA prostate RefZ_expression
2024-10-21 13:18:29.255397 INFO::Range of locus: chr2:70500636-72945943</code></pre>
<pre><code>chromosome 2, position 70500636 to 72945943</code></pre>
<pre><code>6308 SNPs/datapoints</code></pre>
<pre><code>2024-10-21 13:18:29.330899 INFO::focal molecular trait QTL positions: 70532295,70534139,70535369,70536750,70537853
2024-10-21 13:18:29.344433 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 183 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/locus%20plot-7.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-7">
Past versions of "locus plot-7.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-7.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/b6b2e9d193f8b35da919ac778a1f1e88634beb74/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-7.png" target="_blank">b6b2e9d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;1_87662948_89600744&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-21 13:18:30.066679 INFO::Limit to protein coding genes
2024-10-21 13:18:30.067308 INFO::focal id: LMO4|prostate_RefZ_expression
2024-10-21 13:18:30.067566 INFO::focal molecular trait: LMO4 prostate RefZ_expression
2024-10-21 13:18:30.067839 INFO::Range of locus: chr1:87662880-89600032</code></pre>
<pre><code>chromosome 1, position 87662880 to 89600032</code></pre>
<pre><code>4704 SNPs/datapoints</code></pre>
<pre><code>2024-10-21 13:18:30.132283 INFO::focal molecular trait QTL positions: 87740404,87758547
2024-10-21 13:18:30.14587 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 254 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/locus%20plot-8.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-8">
Past versions of "locus plot-8.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-8.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/b6b2e9d193f8b35da919ac778a1f1e88634beb74/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-8.png" target="_blank">b6b2e9d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;1_44503511_46433829&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-21 13:18:30.798341 INFO::Limit to protein coding genes
2024-10-21 13:18:30.799066 INFO::focal id: GPBP1L1|prostate_RefZ_expression
2024-10-21 13:18:30.79935 INFO::focal molecular trait: GPBP1L1 prostate RefZ_expression
2024-10-21 13:18:30.799631 INFO::Range of locus: chr1:44503818-46433232</code></pre>
<pre><code>chromosome 1, position 44503818 to 46433232</code></pre>
<pre><code>4284 SNPs/datapoints</code></pre>
<pre><code>2024-10-21 13:18:30.89065 INFO::focal molecular trait QTL positions: 45910127,46029352
2024-10-21 13:18:30.937735 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 681 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<pre><code>Warning: ggrepel: 7 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/locus%20plot-9.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-9">
Past versions of "locus plot-9.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-9.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/b6b2e9d193f8b35da919ac778a1f1e88634beb74/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-9.png" target="_blank">b6b2e9d</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;1_204711940_206268085&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-21 13:18:32.04805 INFO::Limit to protein coding genes
2024-10-21 13:18:32.048658 INFO::focal id: NUCKS1|prostate_RefZ_expression
2024-10-21 13:18:32.04892 INFO::focal molecular trait: NUCKS1 prostate RefZ_expression
2024-10-21 13:18:32.049188 INFO::Range of locus: chr1:204712783-206237183</code></pre>
<pre><code>chromosome 1, position 204712783 to 206237183</code></pre>
<pre><code>3840 SNPs/datapoints</code></pre>
<pre><code>2024-10-21 13:18:32.120946 INFO::focal molecular trait QTL positions: 205702516,205720679,205766484,205766763
2024-10-21 13:18:32.133844 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 164 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/locus%20plot-10.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-10">
Past versions of "locus plot-10.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-10" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-10.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;2_205870980_207780675&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-21 13:18:32.876702 INFO::Limit to protein coding genes
2024-10-21 13:18:32.87731 INFO::focal id: intron_2_207165706_207166757|lymphocytes_splicing
2024-10-21 13:18:32.877579 INFO::focal molecular trait: MIR7845 lymphocytes mashr_splicing
2024-10-21 13:18:32.877845 INFO::Range of locus: chr2:205871698-207779797</code></pre>
<pre><code>chromosome 2, position 205871698 to 207779797</code></pre>
<pre><code>4892 SNPs/datapoints</code></pre>
<pre><code>2024-10-21 13:18:32.973502 INFO::focal molecular trait QTL positions: 207166747
2024-10-21 13:18:32.986643 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 212 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/locus%20plot-11.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-11">
Past versions of "locus plot-11.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-11" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-11.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;2_213149558_214709071&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-21 13:18:33.679327 INFO::Limit to protein coding genes
2024-10-21 13:18:33.679931 INFO::focal id: ENSG00000030419.16|lymphocytes_expression
2024-10-21 13:18:33.680195 INFO::focal molecular trait: IKZF2 lymphocytes mashr_expression
2024-10-21 13:18:33.680473 INFO::Range of locus: chr2:213075959-214707887</code></pre>
<pre><code>chromosome 2, position 213075959 to 214707887</code></pre>
<pre><code>4145 SNPs/datapoints</code></pre>
<pre><code>2024-10-21 13:18:33.737617 INFO::focal molecular trait QTL positions: 213153600
2024-10-21 13:18:33.750503 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 14 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/locus%20plot-12.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-12">
Past versions of "locus plot-12.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-12" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-12.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;2_241210506_242147200&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-21 13:18:34.257033 INFO::Limit to protein coding genes
2024-10-21 13:18:34.257638 INFO::focal id: FARP2|prostate_RefZ_expression
2024-10-21 13:18:34.257871 INFO::focal molecular trait: FARP2 prostate RefZ_expression
2024-10-21 13:18:34.258123 INFO::Range of locus: chr2:241206843-242115497</code></pre>
<pre><code>chromosome 2, position 241206843 to 242115497</code></pre>
<pre><code>2572 SNPs/datapoints</code></pre>
<pre><code>2024-10-21 13:18:34.332273 INFO::focal molecular trait QTL positions: 241222776,241234501,241245223,241345230,241375608,241422182,241493933,241497112,241503095,241526862,241533917,241664954
2024-10-21 13:18:34.345877 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 452 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/locus%20plot-13.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-13">
Past versions of "locus plot-13.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-13" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-13.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;1_14565015_16571235&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-21 13:18:35.230131 INFO::Limit to protein coding genes
2024-10-21 13:18:35.230812 INFO::focal id: C1orf64|prostate_RefZ_expression
2024-10-21 13:18:35.231118 INFO::focal molecular trait: C1orf64 prostate RefZ_expression
2024-10-21 13:18:35.231422 INFO::Range of locus: chr1:14565291-16996858</code></pre>
<pre><code>chromosome 1, position 14565291 to 16996858</code></pre>
<pre><code>6337 SNPs/datapoints</code></pre>
<pre><code>2024-10-21 13:18:35.320276 INFO::focal molecular trait QTL positions: 15506412,15945755,16000927,16001958,16018898,16019124,16024780,16042283
2024-10-21 13:18:35.333486 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 368 unlabeled data points (too many overlaps). Consider increasing max.overlaps
ggrepel: 7 unlabeled data points (too many overlaps). Consider increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/locus%20plot-14.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-14">
Past versions of "locus plot-14.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-14" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-14.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;1_206268085_208237019&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-21 13:18:36.27524 INFO::Limit to protein coding genes
2024-10-21 13:18:36.275899 INFO::focal id: intron_1_206604729_206605400|prostate_splicing
2024-10-21 13:18:36.276159 INFO::focal molecular trait: EIF2D prostate mashr_splicing
2024-10-21 13:18:36.276432 INFO::Range of locus: chr1:206315136-208236915</code></pre>
<pre><code>chromosome 1, position 206315136 to 208236915</code></pre>
<pre><code>4446 SNPs/datapoints</code></pre>
<pre><code>2024-10-21 13:18:36.346239 INFO::focal molecular trait QTL positions: 206604463,206604711
2024-10-21 13:18:36.359504 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 342 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<pre><code>Warning: ggrepel: 28 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_combined_output.Rmd/locus%20plot-15.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-locus-plot-15">
Past versions of "locus plot-15.png"
</button>
</p>
<div id="fig-no-spaces-locus-plot-15" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/6cc8b8cd13debb32ccae908ef010901528d52252/docs/figure/cTWAS_LD_european_combined_output.Rmd/locus plot-15.png" target="_blank">6cc8b8c</a>
</td>
<td>
yy896
</td>
<td>
2024-10-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.1 (2024-06-14)
Platform: aarch64-apple-darwin20
Running under: macOS 15.0.1

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: America/New_York
tzcode source: internal

attached base packages:
[1] stats4    stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] enrichR_3.2               data.table_1.16.2        
 [3] lubridate_1.9.3           forcats_1.0.0            
 [5] stringr_1.5.1             dplyr_1.1.4              
 [7] purrr_1.0.2               tidyr_1.3.1              
 [9] tibble_3.2.1              tidyverse_2.0.0          
[11] ggrepel_0.9.6             ggplot2_3.5.1            
[13] EnsDb.Hsapiens.v86_2.99.0 ensembldb_2.28.1         
[15] AnnotationFilter_1.28.0   GenomicFeatures_1.56.0   
[17] AnnotationDbi_1.66.0      Biobase_2.64.0           
[19] GenomicRanges_1.56.2      GenomeInfoDb_1.40.1      
[21] IRanges_2.38.1            S4Vectors_0.42.1         
[23] BiocGenerics_0.50.0       ctwas_0.4.16             
[25] readr_2.1.5              

loaded via a namespace (and not attached):
  [1] RColorBrewer_1.1-3          rstudioapi_0.17.0          
  [3] jsonlite_1.8.9              magrittr_2.0.3             
  [5] farver_2.1.2                rmarkdown_2.28             
  [7] fs_1.6.4                    BiocIO_1.14.0              
  [9] zlibbioc_1.50.0             vctrs_0.6.5                
 [11] memoise_2.0.1               Rsamtools_2.20.0           
 [13] RCurl_1.98-1.16             mixsqp_0.3-54              
 [15] htmltools_0.5.8.1           S4Arrays_1.4.1             
 [17] curl_5.2.3                  SparseArray_1.4.8          
 [19] sass_0.4.9                  bslib_0.8.0                
 [21] htmlwidgets_1.6.4           plotly_4.10.4              
 [23] zoo_1.8-12                  cachem_1.1.0               
 [25] gggrid_0.2-0                GenomicAlignments_1.40.0   
 [27] whisker_0.4.1               lifecycle_1.0.4            
 [29] pkgconfig_2.0.3             Matrix_1.7-0               
 [31] R6_2.5.1                    fastmap_1.2.0              
 [33] GenomeInfoDbData_1.2.12     MatrixGenerics_1.16.0      
 [35] digest_0.6.37               colorspace_2.1-1           
 [37] rprojroot_2.0.4             irlba_2.3.5.1              
 [39] crosstalk_1.2.1             RSQLite_2.3.7              
 [41] WriteXLS_6.7.0              labeling_0.4.3             
 [43] pgenlibr_0.3.7              fansi_1.0.6                
 [45] timechange_0.3.0            httr_1.4.7                 
 [47] abind_1.4-8                 compiler_4.4.1             
 [49] bit64_4.5.2                 withr_3.0.1                
 [51] BiocParallel_1.38.0         DBI_1.2.3                  
 [53] highr_0.11                  DelayedArray_0.30.1        
 [55] rjson_0.2.23                tools_4.4.1                
 [57] httpuv_1.6.15               glue_1.8.0                 
 [59] restfulr_0.0.15             promises_1.3.0             
 [61] grid_4.4.1                  generics_0.1.3             
 [63] LDlinkR_1.4.0               gtable_0.3.5               
 [65] tzdb_0.4.0                  locuszoomr_0.3.5           
 [67] hms_1.1.3                   utf8_1.2.4                 
 [69] XVector_0.44.0              pillar_1.9.0               
 [71] later_1.3.2                 logging_0.10-108           
 [73] lattice_0.22-6              rtracklayer_1.64.0         
 [75] bit_4.5.0                   tidyselect_1.2.1           
 [77] Biostrings_2.72.1           knitr_1.48                 
 [79] git2r_0.33.0.9000           ProtGenerics_1.36.0        
 [81] SummarizedExperiment_1.34.0 xfun_0.48                  
 [83] matrixStats_1.4.1           DT_0.33                    
 [85] stringi_1.8.4               UCSC.utils_1.0.0           
 [87] workflowr_1.7.1             lazyeval_0.2.2             
 [89] yaml_2.3.10                 evaluate_1.0.1             
 [91] codetools_0.2-20            cli_3.6.3                  
 [93] munsell_0.5.1               jquerylib_0.1.4            
 [95] Rcpp_1.0.13                 png_0.1-8                  
 [97] XML_3.99-0.17               parallel_4.4.1             
 [99] blob_1.2.4                  bitops_1.0-9               
[101] viridisLite_0.4.2           scales_1.3.0               
[103] crayon_1.5.3                rlang_1.1.4                
[105] cowplot_1.1.3               KEGGREST_1.44.1            </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
