<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Nina Yifan Yang" />

<meta name="date" content="2024-10-04" />

<title>Prostate Cancer ~ Predictdb eQTL&amp;sQTL from Prostate&amp;Testis&amp;Liver</title>

<script src="site_libs/header-attrs-2.28/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.33/datatables.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Multigroup_cTWAS</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/yy896/Multigroup_cTWAS">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">Prostate Cancer ~ Predictdb eQTL&amp;sQTL
from Prostate&amp;Testis&amp;Liver</h1>
<h4 class="author">Nina Yifan Yang</h4>
<h4 class="date">2024-10-04</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-10-04
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Multigroup_cTWAS/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20241004code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20241004)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20241004code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20241004)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomyy896MultigroupcTWAStreea715301abe738a788dc41bff144e60cdf3e9d77atargetblanka715301a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/yy896/Multigroup_cTWAS/tree/a715301abe738a788dc41bff144e60cdf3e9d77a" target="_blank">a715301</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomyy896MultigroupcTWAStreea715301abe738a788dc41bff144e60cdf3e9d77atargetblanka715301a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/yy896/Multigroup_cTWAS/tree/a715301abe738a788dc41bff144e60cdf3e9d77a" target="_blank">a715301</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory

Untracked files:
    Untracked:  analysis/cTWAS_noLD_european_GENE_SUMMARY.Rmd
    Untracked:  analysis/cTWAS_noLD_european_intestinewb_output.Rmd
    Untracked:  analysis/cTWAS_noLD_european_liver_output.Rmd
    Untracked:  analysis/cTWAS_noLD_european_lung_output.Rmd
    Untracked:  analysis/cTWAS_noLD_european_lungliver_output.Rmd
    Untracked:  analysis/cTWAS_noLD_european_lungliverlymphocytes_output.Rmd
    Untracked:  analysis/cTWAS_noLD_european_lymphocytes_output.Rmd
    Untracked:  analysis/cTWAS_noLD_european_prostatetestis_output.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown
(<code>analysis/cTWAS_noLD_european_prostatetestisliver_output.Rmd</code>)
and HTML
(<code>docs/cTWAS_noLD_european_prostatetestisliver_output.html</code>)
files. If you’ve configured a remote Git repository (see
<code>?wflow_git_remote</code>), click on the hyperlinks in the table
below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/a715301abe738a788dc41bff144e60cdf3e9d77a/analysis/cTWAS_noLD_european_prostatetestisliver_output.Rmd" target="_blank">a715301</a>
</td>
<td>
yy896
</td>
<td>
2024-10-04
</td>
<td>
Add my first analysis
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p>We present a sample cTWAS report based on real data analysis. The
analyzed trait is prostate cancer, the prediction models are Prostate
&amp; Testis &amp; Liver gene expression and splicing models trained on
GTEx v8 in the PredictDB format.</p>
<div id="analysis-settings" class="section level2">
<h2>Analysis Settings</h2>
<div id="input-data" class="section level3">
<h3>Input Data</h3>
<ul>
<li>GWAS Z-scores</li>
</ul>
<p>The summary statistics for prostate cancer are downloaded from <a
href="https://www.ebi.ac.uk/gwas/downloads/summary-statistics"
class="uri">https://www.ebi.ac.uk/gwas/downloads/summary-statistics</a>,
using dataset ID: GCST90274714.</p>
<p>The sample size is</p>
<pre><code>[1] &quot;gwas_n = 726828&quot;</code></pre>
<ul>
<li>Prediction models</li>
</ul>
<p>The prediction models used in this analysis are prostate &amp; testis
&amp; liver gene expression and splicing models, trained on GTEx v8 in
the PredictDB format. These models can be downloaded from <a
href="https://predictdb.org/post/2021/07/21/gtex-v8-models-on-eqtl-and-sqtl/"
class="uri">https://predictdb.org/post/2021/07/21/gtex-v8-models-on-eqtl-and-sqtl/</a></p>
<ul>
<li>Reference data</li>
</ul>
<p>We did not include LD reference in the analysis.</p>
</div>
<div id="data-processing-and-harmonization" class="section level3">
<h3>Data processing and harmonization</h3>
<p>We map the reference SNPs to regions following the instructions from
the cTWAS tutorial.</p>
<p>When processing z-scores, we exclude multi-allelic and
strand-ambiguous variants by setting
<code>drop_multiallelic = TRUE</code> and
<code>drop_strand_ambig = TRUE</code>.</p>
<p>The process can be divided into steps below, users can expand the
code snippets below to view the exact code used.</p>
<ul>
<li>Input and output settings</li>
</ul>
<pre class="r"><code>weight_prostate_expression_file &lt;- system.file(&quot;extdata/project_data&quot;, 
                                               &quot;mashr_Prostate_e.db&quot;, package = &quot;ctwas&quot;)
weight_prostate_splicing_file &lt;- system.file(&quot;extdata/project_data&quot;, 
                                             &quot;mashr_Prostate_s.db&quot;, package = &quot;ctwas&quot;)
weight_testis_expression_file &lt;- system.file(&quot;extdata/project_data&quot;, 
                                             &quot;mashr_Testis_e.db&quot;, package = &quot;ctwas&quot;)
weight_testis_splicing_file &lt;- system.file(&quot;extdata/project_data&quot;, 
                                           &quot;mashr_Testis_s.db&quot;, package = &quot;ctwas&quot;)
weight_liver_expression_file &lt;- system.file(&quot;extdata/project_data&quot;, 
                                            &quot;mashr_Liver_e.db&quot;, package = &quot;ctwas&quot;)
weight_liver_splicing_file &lt;- system.file(&quot;extdata/project_data&quot;, 
                                          &quot;mashr_Liver_s.db&quot;, package = &quot;ctwas&quot;)
z_snp &lt;- vroom::vroom(&quot;~/Desktop/Prostate Cancer Project/GCST90274714.mod.tsv.gz&quot;, col_names = TRUE, show_col_types = FALSE)
z_snp$Z &lt;- z_snp$beta / z_snp$standard_error

z_snp &lt;- z_snp[, c(&quot;rsid&quot;, &quot;other_allele&quot;, &quot;effect_allele&quot;, &quot;Z&quot;)]
colnames(z_snp) &lt;- c(&quot;id&quot;, &quot;A1&quot;, &quot;A2&quot;, &quot;z&quot;)

region_file &lt;- system.file(&quot;extdata/ldetect&quot;, &quot;EUR.b38.ldetect.regions.RDS&quot;, package = &quot;ctwas&quot;)
region_info &lt;- readRDS(region_file)

## other parameters
ncore &lt;- 6</code></pre>
<ul>
<li>Preprocessing GWAS</li>
</ul>
<pre class="r"><code>### Preprocess SNP_map
ref_snp_info_file &lt;- system.file(&quot;extdata/project_data&quot;, &quot;ukb_b38_0.1_var_info.Rvar.gz&quot;, package = &quot;ctwas&quot;)

ref_snp_info &lt;- data.table::fread(ref_snp_info_file, sep = &quot;\t&quot;)
class(ref_snp_info) &lt;- &quot;data.frame&quot;

res &lt;- create_snp_map(region_info, ref_snp_info)
region_info &lt;- res$region_info
snp_map &lt;- res$snp_map

### Preprocess GWAS z-scores
z_snp &lt;- preprocess_z_snp(z_snp, snp_map, 
                          drop_multiallelic = TRUE, 
                          drop_strand_ambig = TRUE)</code></pre>
<ul>
<li>Preprocessing weights</li>
</ul>
<pre class="r"><code>weights_prostate_expression &lt;- preprocess_weights(weight_prostate_expression_file,
                                                  region_info,
                                                  gwas_snp_ids = z_snp$id,
                                                  snp_map = snp_map,
                                                  type = &quot;expression&quot;,
                                                  context = &quot;prostate&quot;,
                                                  weight_name = &quot;prostate_expression&quot;,
                                                  weight_format = &quot;PredictDB&quot;,
                                                  drop_strand_ambig = TRUE,
                                                  scale_predictdb_weights = TRUE,
                                                  load_predictdb_LD = TRUE,
                                                  filter_protein_coding_genes = TRUE,
                                                  ncore = 6)
weights_prostate_splicing &lt;- preprocess_weights(weight_prostate_splicing_file,
                                                region_info,
                                                gwas_snp_ids = z_snp$id,
                                                snp_map = snp_map,
                                                type = &quot;splicing&quot;,
                                                context = &quot;prostate&quot;,
                                                weight_name = &quot;prostate_splicing&quot;,
                                                weight_format = &quot;PredictDB&quot;,
                                                drop_strand_ambig = TRUE,
                                                scale_predictdb_weights = TRUE,
                                                load_predictdb_LD = TRUE,
                                                filter_protein_coding_genes = TRUE,
                                                ncore = 6)

weights_testis_expression &lt;- preprocess_weights(weight_testis_expression_file,
                                                region_info,
                                                gwas_snp_ids = z_snp$id,
                                                snp_map = snp_map,
                                                type = &quot;expression&quot;,
                                                context = &quot;testis&quot;,
                                                weight_name = &quot;testis_expression&quot;,
                                                weight_format = &quot;PredictDB&quot;,
                                                drop_strand_ambig = TRUE,
                                                scale_predictdb_weights = TRUE,
                                                load_predictdb_LD = TRUE,
                                                filter_protein_coding_genes = TRUE,
                                                ncore = 6)
weights_testis_splicing &lt;- preprocess_weights(weight_testis_splicing_file,
                                              region_info,
                                              gwas_snp_ids = z_snp$id,
                                              snp_map = snp_map,
                                              type = &quot;splicing&quot;,
                                              context = &quot;testis&quot;,
                                              weight_name = &quot;testis_splicing&quot;,
                                              weight_format = &quot;PredictDB&quot;,
                                              drop_strand_ambig = TRUE,
                                              scale_predictdb_weights = TRUE,
                                              load_predictdb_LD = TRUE,
                                              filter_protein_coding_genes = TRUE,
                                              ncore = 6)

weights_liver_expression &lt;- preprocess_weights(weight_liver_expression_file,
                                               region_info,
                                               gwas_snp_ids = z_snp$id,
                                               snp_map = snp_map,
                                               type = &quot;expression&quot;,
                                               context = &quot;liver&quot;,
                                               weight_name = &quot;liver_expression&quot;,
                                               weight_format = &quot;PredictDB&quot;,
                                               drop_strand_ambig = TRUE,
                                               scale_predictdb_weights = TRUE,
                                               load_predictdb_LD = TRUE,
                                               filter_protein_coding_genes = TRUE,
                                               ncore = 6)
weights_liver_splicing &lt;- preprocess_weights(weight_liver_splicing_file,
                                             region_info,
                                             gwas_snp_ids = z_snp$id,
                                             snp_map = snp_map,
                                             type = &quot;splicing&quot;,
                                             context = &quot;liver&quot;,
                                             weight_name = &quot;liver_splicing&quot;,
                                             weight_format = &quot;PredictDB&quot;,
                                             drop_strand_ambig = TRUE,
                                             scale_predictdb_weights = TRUE,
                                             load_predictdb_LD = TRUE,
                                             filter_protein_coding_genes = TRUE,
                                             ncore = 6)

weights &lt;- c(
  weights_prostate_expression,
  weights_prostate_splicing,
  weights_testis_expression,
  weights_testis_splicing,
  weights_liver_expression,
  weights_liver_splicing)</code></pre>
</div>
</div>
<div id="running-ctwas-analysis" class="section level2">
<h2>Running cTWAS analysis</h2>
<p>We use the ctwas main function <code>ctwas_sumstats_noLD</code> to
run the cTWAS analysis without LD. For more details on this function,
refer to the cTWAS tutorial: <a
href="https://xinhe-lab.github.io/multigroup_ctwas/articles/minimal_tutorial.html"
class="uri">https://xinhe-lab.github.io/multigroup_ctwas/articles/minimal_tutorial.html</a></p>
<p>All arguments are set to their default values.</p>
<p>Users can expand the code snippets below to view the exact code
used.</p>
<pre class="r"><code>ctwas_res &lt;- ctwas_sumstats_noLD(z_snp,
                                 weights,
                                 region_info,
                                 snp_map,
                                 thin = 0.1, 
                                 ncore = 6)</code></pre>
</div>
<div id="parameter-estimation" class="section level2">
<h2>Parameter estimation</h2>
<p><code>ctwas_res</code> is the object contains the outputs of
cTWAS</p>
<p>We extract the estimated parameters by
<code>param &lt;- ctwas_res$param</code></p>
<p>We make plots using the function
<code>make_convergence_plots(param, gwas_n)</code> to see how estimated
parameters converge during the execution of the program:</p>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/unnamed-chunk-7-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>These plots show the estimated prior inclusion probability, prior
effect size variance, enrichment and proportion of variance explained
(PVE) over the iterations of parameter estimation. The enrichment is
defined as the ratio of the prior inclusion probability of molecular
traits over the prior inclusion probability of variants. We generally
expect molecular traits to have higher prior inclusion probability than
variants. Enrichment values typically range from 20 - 100 for expression
traits.</p>
<p>Then, we use <code>summarize_param(param, gwas_n)</code> to obtain
estimated parameters (from the last iteration) and to compute the PVE by
variants and molecular traits.</p>
<pre><code>[1] &quot;The number of genes/introns/SNPs used in the analysis is:&quot;</code></pre>
<pre><code>prostate|expression   prostate|splicing   testis|expression     testis|splicing 
               9693               29875               11361               50036 
   liver|expression      liver|splicing                 SNP 
               8721               19025             7146350 </code></pre>
<p><code>ctwas_parameters$attributable_pve</code> contains the
proportion of heritability mediated by molecular traits and variants, we
visualize it using pie chart.</p>
<pre class="r"><code>ctwas_parameters &lt;- summarize_param(param, gwas_n)
data &lt;- data.frame(
  category = names(ctwas_parameters$attributable_pve),
  percentage = ctwas_parameters$attributable_pve
)
# Calculate percentage labels for the chart
data$percentage_label &lt;- paste0(round(data$percentage * 100, 1), &quot;%&quot;)

ggplot(data, aes(x = &quot;&quot;, y = percentage, fill = category)) +
  geom_bar(stat = &quot;identity&quot;, width = 1) +
  coord_polar(&quot;y&quot;, start = 0) +
  theme_void() +  # Remove background and axes
  geom_text(aes(label = percentage_label), 
            position = position_stack(vjust = 0.5)) +
  scale_fill_brewer(palette = &quot;Pastel1&quot;) +
  labs(fill = &quot;Category&quot;) +
  ggtitle(&quot;Percent of heritability&quot;)</code></pre>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/piechart-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="diagnosis-plots" class="section level2">
<h2>Diagnosis plots</h2>
<p>For all genes analyzed, we compare the z-scores and fine-mapping
PIPs”</p>
<pre class="r"><code>ggplot(data = finemap_res[finemap_res$type!=&quot;SNP&quot;,], aes(x = abs(z), y = susie_pip)) +
  geom_point() +
  labs(x = &quot;abs(z-scores)&quot;, y = &quot;PIPs&quot;) +
  theme_minimal()</code></pre>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/unnamed-chunk-9-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="fine-mapping-results" class="section level2">
<h2>Fine-mapping results</h2>
<p>We process the fine-mapping results here.</p>
<p>We first add gene annotations to cTWAS results.</p>
<p><code>mapping_table</code> is the combination of both expression and
splicing data.</p>
<pre class="r"><code>finemap_res &lt;- anno_finemap_res(finemap_res,
                                snp_map = snp_map,
                                mapping_table = mapping_table,
                                add_gene_annot = TRUE,
                                map_by = &quot;molecular_id&quot;,
                                drop_unmapped = TRUE,
                                add_position = TRUE,
                                use_gene_pos = &quot;mid&quot;)</code></pre>
<p>The annotated fine-mapping results, ones within credible sets are
shown</p>
<div class="datatables html-widget html-fill-item" id="htmlwidget-232a8e63d6aacd296363" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-232a8e63d6aacd296363">{"x":{"filter":"none","vertical":false,"data":[["3792","15557","17023","25238","50737","57121","58288","59976","61878","79911","80619","83556","91831","98913","101149","102273","105255","108799"],["ENSG00000163874.10|prostate_expression","ENSG00000118263.14|liver_expression","intron_2_241195845_241199316|prostate_splicing","ENSG00000121897.14|prostate_expression","ENSG00000138111.14|prostate_expression","ENSG00000137673.8|liver_expression","ENSG00000165495.15|prostate_expression","ENSG00000121380.12|prostate_expression","ENSG00000120832.9|testis_expression","ENSG00000153786.12|prostate_expression","ENSG00000179409.10|prostate_expression","ENSG00000141052.17|liver_expression","ENSG00000141934.9|prostate_expression","ENSG00000167641.10|testis_expression","ENSG00000105327.17|prostate_expression","ENSG00000170892.10|testis_expression","ENSG00000126001.15|testis_expression","ENSG00000184058.12|prostate_expression"],["expression","expression","splicing","expression","expression","expression","expression","expression","expression","expression","expression","expression","expression","expression","expression","expression","expression","expression"],["prostate","liver","prostate","prostate","prostate","liver","prostate","prostate","testis","prostate","prostate","liver","prostate","testis","prostate","testis","testis","prostate"],["prostate|expression","liver|expression","prostate|splicing","prostate|expression","prostate|expression","liver|expression","prostate|expression","prostate|expression","testis|expression","prostate|expression","prostate|expression","liver|expression","prostate|expression","testis|expression","prostate|expression","testis|expression","testis|expression","prostate|expression"],["1_37083582_38266175","2_205870980_207780675","2_240620667_241210506","4_38905401_40200989","10_101189482_102620653","11_101460390_104088908","11_124625632_126441425","12_11601912_12580594","12_106564970_108632125","16_84412277_85115079","17_150263_1269105","17_12631452_13277055","19_80840_610729","19_37036131_39679979","19_46646825_48778970","19_54099074_55216712","20_34225635_36332043","22_17813322_19924835"],[-4.57196354978897,6.19298245614035,13.791151879786,-4.64,-10.02440158725867,-12.29279413978504,9.978798586572438,4.625901499526558,-4.892857142857143,6.991183970613101,14.40350877192982,5.211111111111111,-4.397861388346826,-17.78181818181818,5.136986301369863,-5.317204335134762,23.11759293172035,9],[0.8232200579391992,0.9304667401752068,0.9999999998896247,0.8068983164837592,0.8357830237945643,0.9963681383823847,0.9898709014182919,0.8164748611267992,0.8293440488682946,0.9913395235775887,0.9890273216628932,0.8279601532949281,0.8420513161589621,0.8636257351919318,0.9364867399750951,0.9925336374082983,1,0.8857755181686121],[19.91491163672772,35.74520686691225,162.4157942987603,20.48348032057104,92.11280676047758,138.0375686699603,91.28528012806778,20.36497177155788,22.67017117998169,45.29192944506745,189.1550822015219,25.58729510314031,18.49821387319035,287.7935747511692,24.89144903381635,26.60058943718235,485.7655064936923,74.43329261066856],[1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1],["ENSG00000163874.10","ENSG00000118263.14","intron_2_241195845_241199316","ENSG00000121897.14","ENSG00000138111.14","ENSG00000137673.8","ENSG00000165495.15","ENSG00000121380.12","ENSG00000120832.9","ENSG00000153786.12","ENSG00000179409.10","ENSG00000141052.17","ENSG00000141934.9","ENSG00000167641.10","ENSG00000105327.17","ENSG00000170892.10","ENSG00000126001.15","ENSG00000184058.12"],[4.831750025929168e-06,5.903630251842041e-10,2.881223371161483e-43,3.484091780689317e-06,1.190787383032674e-23,9.90231199425496e-35,1.8873814535796e-23,3.72972533281776e-06,9.938254618921576e-07,2.72576046411427e-12,4.918188507245037e-47,1.8771298072025e-07,1.09322757760192e-05,9.775769643266285e-71,2.791792973768315e-07,1.0537371290076e-07,3.08096146668202e-118,2.257176811907682e-19],["ZC3H12A","KLF7","ANO7","LIAS","MFSD13A","MMP7","PKNOX2","BCL2L14","MTERF2","ZDHHC7","GEMIN4","MYOCD","PLPP2","PPP1R14A","BBC3","TSEN34","CEP250","TBX1"],["protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding"],["1","2","2","4","10","11","11","12","12","16","17","17","19","19","19","19","20","22"],["37474552","207074137","241195845","39459000","102461395","102520508","125164687","12049844","106977291","84974181","744414","12665890","281040","38251237","47220822","54189938","35455164","19756703"],["37484379","207167267","241199316","39477653","102477045","102530753","125433389","12211084","106987166","85011535","753999","12768949","291504","38256591","47232766","54194536","35519280","19783593"],[37479466,207120702,241197580,39468326,102469220,102525630,125299038,12130464,106982228,84992858,749206,12717420,286272,38253914,47226794,54192237,35487222,19770148]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>id<\/th>\n      <th>type<\/th>\n      <th>context<\/th>\n      <th>group<\/th>\n      <th>region_id<\/th>\n      <th>z<\/th>\n      <th>susie_pip<\/th>\n      <th>mu2<\/th>\n      <th>cs_index<\/th>\n      <th>molecular_id<\/th>\n      <th>pval<\/th>\n      <th>gene_name<\/th>\n      <th>gene_type<\/th>\n      <th>chrom<\/th>\n      <th>start<\/th>\n      <th>end<\/th>\n      <th>pos<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[6,7,8,9,11,17]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"id","targets":1},{"name":"type","targets":2},{"name":"context","targets":3},{"name":"group","targets":4},{"name":"region_id","targets":5},{"name":"z","targets":6},{"name":"susie_pip","targets":7},{"name":"mu2","targets":8},{"name":"cs_index","targets":9},{"name":"molecular_id","targets":10},{"name":"pval","targets":11},{"name":"gene_name","targets":12},{"name":"gene_type","targets":13},{"name":"chrom","targets":14},{"name":"start","targets":15},{"name":"end","targets":16},{"name":"pos","targets":17}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<p>Next, we compute gene PIPs across different types of molecular
traits.</p>
<div id="combined-pips" class="section level3">
<h3>Combined PIPs</h3>
<pre class="r"><code>library(dplyr)
combined_pip_by_group &lt;- combine_gene_pips(finemap_res, 
                                             group_by = &quot;gene_name&quot;,
                                             by = &quot;group&quot;,
                                             method = &quot;combine_cs&quot;,
                                             filter_cs = TRUE)</code></pre>
<pre><code>2024-10-04 17:51:09.051538 INFO::Limit gene results to credible sets</code></pre>
<pre class="r"><code>combined_pip_table &lt;- subset(combined_pip_by_group, combined_pip &gt; 0.8)
DT::datatable(combined_pip_table)</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-2112c1f1dbc2c6427f4c" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-2112c1f1dbc2c6427f4c">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36"],["CEP250","ANO7","PCAT19","MMP7","TSEN34","ZDHHC7","TBX1","PKNOX2","GEMIN4","MSMB","ARNT2","ADGRG1","CCDC88C","TOR1A","BBC3","TACC1","KLF7","MYO5C","CHMP4C","MAPKBP1","RPS6KL1","WTAP","ZC3H12A","MTERF2","ZBTB38","PPP1R14A","GRHL1","FAM118A","PLPP2","MFSD13A","MYOCD","BCL2L14","EIF2D","FBXO38","LIAS","MAST2"],[null,null,null,null,null,0.9913395235775887,0.8857755181686121,0.9898709014182919,0.9890273216628932,0.5418888103196474,0.6000798536883102,null,null,null,0.9364867399750951,null,null,0.6734946421221127,0.5568188919743071,null,0.4045565259791998,null,0.8232200579391992,null,null,null,0.7411201938900475,0.0289700655285251,0.8420513161589621,0.8357830237945643,null,0.8164748611267992,null,0.4143194152970309,0.8068983164837592,null],[null,0.9999999998896247,null,null,null,null,null,null,null,null,null,0.4112503970681436,0.642193965017014,0.4662347909924536,null,0.5415780161701854,null,null,null,0.5476068663426151,0.1554827873049704,0.8256431107772111,0.02402199042717124,null,0.07165198726107291,null,0.1179209567586265,0.4360345634679296,null,null,null,null,0.6396196355425303,0.2463186953246922,null,0.04238496286977045],[1,null,null,null,0.9925336374082983,null,null,null,null,0.4463044401423155,0.3863094031932459,null,null,0.302919404222582,null,null,null,null,0.03560764086189538,null,0.260438655202937,null,0.02816090356852563,0.8293440488682946,null,0.8636257351919318,null,0.01974369678980792,null,null,null,null,null,0.07605366853307682,null,0.7318371938318261],[null,null,0.9999999986374837,null,null,null,0.1047695451473374,null,null,null,null,0.5615658038206144,null,0.173536778222639,null,0.392104836632716,null,null,null,0.3738808427514093,0.03561390678646226,0.06112551425653912,null,0.04919172418075179,0.7935716638362896,null,null,0.3433480517190179,null,null,null,null,0.1707804329340442,0.07157770800615471,null,0.002624727675970551],[null,null,null,0.9963681383823847,null,null,null,null,null,null,null,null,null,null,null,null,0.9304667401752068,0.2529199882861324,0.2526334553281953,null,0.03092086729312715,null,null,null,null,null,null,null,null,null,0.8279601532949281,null,null,null,null,0.02358790926095899],[null,null,null,null,null,null,null,null,null,null,null,null,0.3257539420241273,null,null,null,null,null,0.07719394473339469,null,null,null,0.007756926125579544,null,null,null,null,0.01764185739738433,null,null,null,null,null,null,null,0.001241368121702946],[1,0.9999999998896247,0.9999999986374837,0.9963681383823847,0.9925336374082983,0.9913395235775887,0.9905450633159495,0.9898709014182919,0.9890273216628932,0.9881932504619629,0.9863892568815561,0.972816200888758,0.9679479070411412,0.9426909734376745,0.9364867399750951,0.9336828528029014,0.9304667401752068,0.9264146304082451,0.9222539328977924,0.9214877090940244,0.8870127425666965,0.8867686250337502,0.8831598780604755,0.8785357730490464,0.8652236510973624,0.8636257351919318,0.8590411506486739,0.845738234902665,0.8420513161589621,0.8357830237945643,0.8279601532949281,0.8164748611267992,0.8104000684765746,0.8082694871609546,0.8068983164837592,0.8016761617602292]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>gene_name<\/th>\n      <th>prostate|expression_pip<\/th>\n      <th>prostate|splicing_pip<\/th>\n      <th>testis|expression_pip<\/th>\n      <th>testis|splicing_pip<\/th>\n      <th>liver|expression_pip<\/th>\n      <th>liver|splicing_pip<\/th>\n      <th>combined_pip<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,3,4,5,6,7,8]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"gene_name","targets":1},{"name":"prostate|expression_pip","targets":2},{"name":"prostate|splicing_pip","targets":3},{"name":"testis|expression_pip","targets":4},{"name":"testis|splicing_pip","targets":5},{"name":"liver|expression_pip","targets":6},{"name":"liver|splicing_pip","targets":7},{"name":"combined_pip","targets":8}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div id="locus-plots-from-highest-pip-to-lowest-0.8-cutoff"
class="section level2">
<h2>Locus plots (from highest PIP to lowest 0.8 cutoff)</h2>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;20_34225635_36332043&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:51:09.662273 INFO::Limit to protein coding genes
2024-10-04 17:51:09.666311 INFO::focal id: ENSG00000126001.15|testis_expression
2024-10-04 17:51:09.666583 INFO::focal molecular trait: CEP250 testis expression
2024-10-04 17:51:09.666857 INFO::Range of locus: chr20:34225783-36417586</code></pre>
<pre><code>chromosome 20, position 34225783 to 36417586</code></pre>
<pre><code>4269 SNPs/datapoints</code></pre>
<pre><code>2024-10-04 17:51:09.764679 INFO::focal molecular trait QTL positions: 35445740,35454222,35456125
2024-10-04 17:51:09.77921 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 831 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/locus%20plot-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;2_240620667_241210506&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:51:11.156742 INFO::Limit to protein coding genes
2024-10-04 17:51:11.157369 INFO::focal id: intron_2_241195845_241199316|prostate_splicing
2024-10-04 17:51:11.157624 INFO::focal molecular trait: ANO7 prostate splicing
2024-10-04 17:51:11.157909 INFO::Range of locus: chr2:240621191-241608572</code></pre>
<pre><code>chromosome 2, position 240621191 to 241608572</code></pre>
<pre><code>2142 SNPs/datapoints</code></pre>
<pre><code>2024-10-04 17:51:11.429069 INFO::focal molecular trait QTL positions: 241195850,241202304
2024-10-04 17:51:11.442227 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 184 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/locus%20plot-2.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;11_101460390_104088908&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:51:12.214584 INFO::Limit to protein coding genes
2024-10-04 17:51:12.215313 INFO::focal id: ENSG00000137673.8|liver_expression
2024-10-04 17:51:12.215725 INFO::focal molecular trait: MMP7 liver expression
2024-10-04 17:51:12.216044 INFO::Range of locus: chr11:101460301-104087928</code></pre>
<pre><code>chromosome 11, position 101460301 to 104087928</code></pre>
<pre><code>8445 SNPs/datapoints</code></pre>
<pre><code>2024-10-04 17:51:12.269546 INFO::focal molecular trait QTL positions: 102530902,102530930
2024-10-04 17:51:12.282337 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 94 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/locus%20plot-3.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;19_54099074_55216712&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:51:12.993591 INFO::Limit to protein coding genes
2024-10-04 17:51:12.994208 INFO::focal id: ENSG00000170892.10|testis_expression
2024-10-04 17:51:12.994459 INFO::focal molecular trait: TSEN34 testis expression
2024-10-04 17:51:12.994728 INFO::Range of locus: chr19:54098580-55216001</code></pre>
<pre><code>chromosome 19, position 54098580 to 55216001</code></pre>
<pre><code>4117 SNPs/datapoints</code></pre>
<pre><code>2024-10-04 17:51:13.055069 INFO::focal molecular trait QTL positions: 54189591,54190017
2024-10-04 17:51:13.06788 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 367 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/locus%20plot-4.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;16_84412277_85115079&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:51:13.966343 INFO::Limit to protein coding genes
2024-10-04 17:51:13.966871 INFO::focal id: ENSG00000153786.12|prostate_expression
2024-10-04 17:51:13.967134 INFO::focal molecular trait: ZDHHC7 prostate expression
2024-10-04 17:51:13.967416 INFO::Range of locus: chr16:84412178-85114834</code></pre>
<pre><code>chromosome 16, position 84412178 to 85114834</code></pre>
<pre><code>3434 SNPs/datapoints</code></pre>
<pre><code>2024-10-04 17:51:14.173611 INFO::focal molecular trait QTL positions: 85011497,85011520,85018213
2024-10-04 17:51:14.186885 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 135 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/locus%20plot-5.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;11_124625632_126441425&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:51:14.883641 INFO::Limit to protein coding genes
2024-10-04 17:51:14.884283 INFO::focal id: ENSG00000165495.15|prostate_expression
2024-10-04 17:51:14.884542 INFO::focal molecular trait: PKNOX2 prostate expression
2024-10-04 17:51:14.884813 INFO::Range of locus: chr11:124627414-126441118</code></pre>
<pre><code>chromosome 11, position 124627414 to 126441118</code></pre>
<pre><code>5454 SNPs/datapoints</code></pre>
<pre><code>2024-10-04 17:51:14.940327 INFO::focal molecular trait QTL positions: 125184897
2024-10-04 17:51:14.953132 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 469 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/locus%20plot-6.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;17_150263_1269105&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:51:15.876844 INFO::Limit to protein coding genes
2024-10-04 17:51:15.87743 INFO::focal id: ENSG00000179409.10|prostate_expression
2024-10-04 17:51:15.877692 INFO::focal molecular trait: GEMIN4 prostate expression
2024-10-04 17:51:15.87798 INFO::Range of locus: chr17:150409-1727661</code></pre>
<pre><code>chromosome 17, position 150409 to 1727661</code></pre>
<pre><code>4287 SNPs/datapoints</code></pre>
<pre><code>2024-10-04 17:51:15.933465 INFO::focal molecular trait QTL positions: 715725
2024-10-04 17:51:15.946788 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 176 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/locus%20plot-7.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;19_46646825_48778970&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:51:16.919382 INFO::Limit to protein coding genes
2024-10-04 17:51:16.920083 INFO::focal id: ENSG00000105327.17|prostate_expression
2024-10-04 17:51:16.920353 INFO::focal molecular trait: BBC3 prostate expression
2024-10-04 17:51:16.920646 INFO::Range of locus: chr19:46646725-48778681</code></pre>
<pre><code>chromosome 19, position 46646725 to 48778681</code></pre>
<pre><code>6505 SNPs/datapoints</code></pre>
<pre><code>2024-10-04 17:51:16.981147 INFO::focal molecular trait QTL positions: 47232618
2024-10-04 17:51:16.99398 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 659 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/locus%20plot-8.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;2_205870980_207780675&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:51:18.454419 INFO::Limit to protein coding genes
2024-10-04 17:51:18.45495 INFO::focal id: ENSG00000118263.14|liver_expression
2024-10-04 17:51:18.455214 INFO::focal molecular trait: KLF7 liver expression
2024-10-04 17:51:18.45548 INFO::Range of locus: chr2:205871698-207779797</code></pre>
<pre><code>chromosome 2, position 205871698 to 207779797</code></pre>
<pre><code>4862 SNPs/datapoints</code></pre>
<pre><code>2024-10-04 17:51:18.509172 INFO::focal molecular trait QTL positions: 207166747
2024-10-04 17:51:18.521992 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 179 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/locus%20plot-9.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;22_17813322_19924835&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:51:19.299058 INFO::Limit to protein coding genes
2024-10-04 17:51:19.299689 INFO::focal id: ENSG00000184058.12|prostate_expression
2024-10-04 17:51:19.299985 INFO::focal molecular trait: TBX1 prostate expression
2024-10-04 17:51:19.300271 INFO::Range of locus: chr22:17813372-19925414</code></pre>
<pre><code>chromosome 22, position 17813372 to 19925414</code></pre>
<pre><code>4701 SNPs/datapoints</code></pre>
<pre><code>2024-10-04 17:51:19.353407 INFO::focal molecular trait QTL positions: 19762002
2024-10-04 17:51:19.36668 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 224 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/locus%20plot-10.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;19_37036131_39679979&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:51:20.481344 INFO::Limit to protein coding genes
2024-10-04 17:51:20.482034 INFO::focal id: ENSG00000167641.10|testis_expression
2024-10-04 17:51:20.482316 INFO::focal molecular trait: PPP1R14A testis expression
2024-10-04 17:51:20.482614 INFO::Range of locus: chr19:37036031-39679563</code></pre>
<pre><code>chromosome 19, position 37036031 to 39679563</code></pre>
<pre><code>7148 SNPs/datapoints</code></pre>
<pre><code>2024-10-04 17:51:20.551479 INFO::focal molecular trait QTL positions: 38256477
2024-10-04 17:51:20.564499 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 705 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/locus%20plot-11.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;19_80840_610729&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:51:21.92139 INFO::Limit to protein coding genes
2024-10-04 17:51:21.921966 INFO::focal id: ENSG00000141934.9|prostate_expression
2024-10-04 17:51:21.922184 INFO::focal molecular trait: PLPP2 prostate expression
2024-10-04 17:51:21.922438 INFO::Range of locus: chr19:206677-649427</code></pre>
<pre><code>chromosome 19, position 206677 to 649427</code></pre>
<pre><code>1837 SNPs/datapoints</code></pre>
<pre><code>2024-10-04 17:51:21.976064 INFO::focal molecular trait QTL positions: 290947,292082
2024-10-04 17:51:21.989691 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 146 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/locus%20plot-12.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;10_101189482_102620653&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:51:22.71802 INFO::Limit to protein coding genes
2024-10-04 17:51:22.718612 INFO::focal id: ENSG00000138111.14|prostate_expression
2024-10-04 17:51:22.718856 INFO::focal molecular trait: MFSD13A prostate expression
2024-10-04 17:51:22.719118 INFO::Range of locus: chr10:101190250-102621720</code></pre>
<pre><code>chromosome 10, position 101190250 to 102621720</code></pre>
<pre><code>2664 SNPs/datapoints</code></pre>
<pre><code>2024-10-04 17:51:22.769337 INFO::focal molecular trait QTL positions: 102462097,102463206,102466080
2024-10-04 17:51:22.782552 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 340 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/locus%20plot-13.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;12_106564970_108632125&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:51:23.792768 INFO::Limit to protein coding genes
2024-10-04 17:51:23.793388 INFO::focal id: ENSG00000120832.9|testis_expression
2024-10-04 17:51:23.793656 INFO::focal molecular trait: MTERF2 testis expression
2024-10-04 17:51:23.79393 INFO::Range of locus: chr12:106564986-108631756</code></pre>
<pre><code>chromosome 12, position 106564986 to 108631756</code></pre>
<pre><code>5483 SNPs/datapoints</code></pre>
<pre><code>2024-10-04 17:51:23.845285 INFO::focal molecular trait QTL positions: 106988381
2024-10-04 17:51:23.858341 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 186 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<pre><code>Warning: ggrepel: 11 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/locus%20plot-14.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;17_12631452_13277055&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:51:24.688589 INFO::Limit to protein coding genes
2024-10-04 17:51:24.689133 INFO::focal id: ENSG00000141052.17|liver_expression
2024-10-04 17:51:24.689364 INFO::focal molecular trait: MYOCD liver expression
2024-10-04 17:51:24.689626 INFO::Range of locus: chr17:12631475-13548774</code></pre>
<pre><code>chromosome 17, position 12631475 to 13548774</code></pre>
<pre><code>2239 SNPs/datapoints</code></pre>
<pre><code>2024-10-04 17:51:24.741933 INFO::focal molecular trait QTL positions: 12666125
2024-10-04 17:51:24.755362 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 28 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/locus%20plot-15.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;1_37083582_38266175&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:51:25.351715 INFO::Limit to protein coding genes
2024-10-04 17:51:25.352293 INFO::focal id: ENSG00000163874.10|prostate_expression
2024-10-04 17:51:25.352537 INFO::focal molecular trait: ZC3H12A prostate expression
2024-10-04 17:51:25.352803 INFO::Range of locus: chr1:37083938-38266050</code></pre>
<pre><code>chromosome 1, position 37083938 to 38266050</code></pre>
<pre><code>3030 SNPs/datapoints</code></pre>
<pre><code>2024-10-04 17:51:25.405872 INFO::focal molecular trait QTL positions: 37475037,37499397
2024-10-04 17:51:25.41947 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 190 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<pre><code>Warning: ggrepel: 42 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/locus%20plot-16.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;12_11601912_12580594&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:51:26.450338 INFO::Limit to protein coding genes
2024-10-04 17:51:26.450891 INFO::focal id: ENSG00000121380.12|prostate_expression
2024-10-04 17:51:26.451124 INFO::focal molecular trait: BCL2L14 prostate expression
2024-10-04 17:51:26.451384 INFO::Range of locus: chr12:11034257-12580271</code></pre>
<pre><code>chromosome 12, position 11034257 to 12580271</code></pre>
<pre><code>2904 SNPs/datapoints</code></pre>
<pre><code>2024-10-04 17:51:26.498384 INFO::focal molecular trait QTL positions: 12052195,12052348
2024-10-04 17:51:26.511211 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 69 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/locus%20plot-17.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;4_38905401_40200989&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-10-04 17:51:27.169151 INFO::Limit to protein coding genes
2024-10-04 17:51:27.169728 INFO::focal id: ENSG00000121897.14|prostate_expression
2024-10-04 17:51:27.169971 INFO::focal molecular trait: LIAS prostate expression
2024-10-04 17:51:27.170236 INFO::Range of locus: chr4:38905595-40200074</code></pre>
<pre><code>chromosome 4, position 38905595 to 40200074</code></pre>
<pre><code>3776 SNPs/datapoints</code></pre>
<pre><code>2024-10-04 17:51:27.218658 INFO::focal molecular trait QTL positions: 39465159
2024-10-04 17:51:27.231754 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 226 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_noLD_european_prostatetestisliver_output.Rmd/locus%20plot-18.png" width="960" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.1 (2024-06-14)
Platform: aarch64-apple-darwin20
Running under: macOS 15.0.1

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: America/New_York
tzcode source: internal

attached base packages:
[1] stats4    stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] data.table_1.16.0         lubridate_1.9.3          
 [3] forcats_1.0.0             stringr_1.5.1            
 [5] dplyr_1.1.4               purrr_1.0.2              
 [7] tidyr_1.3.1               tibble_3.2.1             
 [9] tidyverse_2.0.0           ggrepel_0.9.6            
[11] ggplot2_3.5.1             EnsDb.Hsapiens.v86_2.99.0
[13] ensembldb_2.28.1          AnnotationFilter_1.28.0  
[15] GenomicFeatures_1.56.0    AnnotationDbi_1.66.0     
[17] Biobase_2.64.0            GenomicRanges_1.56.1     
[19] GenomeInfoDb_1.40.1       IRanges_2.38.1           
[21] S4Vectors_0.42.1          BiocGenerics_0.50.0      
[23] ctwas_0.4.14              readr_2.1.5              
[25] workflowr_1.7.1          

loaded via a namespace (and not attached):
  [1] RColorBrewer_1.1-3          rstudioapi_0.16.0          
  [3] jsonlite_1.8.9              magrittr_2.0.3             
  [5] farver_2.1.2                rmarkdown_2.28             
  [7] fs_1.6.4                    BiocIO_1.14.0              
  [9] zlibbioc_1.50.0             vctrs_0.6.5                
 [11] memoise_2.0.1               Rsamtools_2.20.0           
 [13] RCurl_1.98-1.16             mixsqp_0.3-54              
 [15] htmltools_0.5.8.1           S4Arrays_1.4.1             
 [17] curl_5.2.3                  SparseArray_1.4.8          
 [19] sass_0.4.9                  bslib_0.8.0                
 [21] htmlwidgets_1.6.4           plotly_4.10.4              
 [23] zoo_1.8-12                  cachem_1.1.0               
 [25] gggrid_0.2-0                GenomicAlignments_1.40.0   
 [27] whisker_0.4.1               lifecycle_1.0.4            
 [29] pkgconfig_2.0.3             Matrix_1.7-0               
 [31] R6_2.5.1                    fastmap_1.2.0              
 [33] GenomeInfoDbData_1.2.12     MatrixGenerics_1.16.0      
 [35] digest_0.6.37               colorspace_2.1-1           
 [37] ps_1.8.0                    rprojroot_2.0.4            
 [39] irlba_2.3.5.1               crosstalk_1.2.1            
 [41] RSQLite_2.3.7               labeling_0.4.3             
 [43] pgenlibr_0.3.7              timechange_0.3.0           
 [45] fansi_1.0.6                 httr_1.4.7                 
 [47] abind_1.4-8                 compiler_4.4.1             
 [49] bit64_4.0.5                 withr_3.0.1                
 [51] BiocParallel_1.38.0         DBI_1.2.3                  
 [53] highr_0.11                  DelayedArray_0.30.1        
 [55] rjson_0.2.23                tools_4.4.1                
 [57] httpuv_1.6.15               glue_1.7.0                 
 [59] restfulr_0.0.15             callr_3.7.6                
 [61] promises_1.3.0              grid_4.4.1                 
 [63] getPass_0.2-4               generics_0.1.3             
 [65] LDlinkR_1.4.0               gtable_0.3.5               
 [67] tzdb_0.4.0                  locuszoomr_0.3.5           
 [69] hms_1.1.3                   utf8_1.2.4                 
 [71] XVector_0.44.0              pillar_1.9.0               
 [73] later_1.3.2                 logging_0.10-108           
 [75] lattice_0.22-6              rtracklayer_1.64.0         
 [77] bit_4.5.0                   tidyselect_1.2.1           
 [79] Biostrings_2.72.1           knitr_1.48                 
 [81] git2r_0.33.0.9000           ProtGenerics_1.36.0        
 [83] SummarizedExperiment_1.34.0 xfun_0.47                  
 [85] matrixStats_1.4.1           DT_0.33                    
 [87] stringi_1.8.4               UCSC.utils_1.0.0           
 [89] lazyeval_0.2.2              yaml_2.3.10                
 [91] evaluate_1.0.0              codetools_0.2-20           
 [93] cli_3.6.3                   munsell_0.5.1              
 [95] processx_3.8.4              jquerylib_0.1.4            
 [97] Rcpp_1.0.13                 png_0.1-8                  
 [99] XML_3.99-0.17               parallel_4.4.1             
[101] blob_1.2.4                  bitops_1.0-8               
[103] viridisLite_0.4.2           scales_1.3.0               
[105] crayon_1.5.3                rlang_1.1.4                
[107] cowplot_1.1.3               KEGGREST_1.44.1            </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
