<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Nina Yifan Yang" />

<meta name="date" content="2024-11-09" />

<title>Prostate Cancer ~ Munro 7 colon transverse QTLs</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.33/datatables.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Multigroup_cTWAS</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/yy896/Multigroup_cTWAS">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">Prostate Cancer ~ Munro 7 colon transverse
QTLs</h1>
<h4 class="author">Nina Yifan Yang</h4>
<h4 class="date">2024-11-09</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-11-09
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Multigroup_cTWAS/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20241004code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20241004)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20241004code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20241004)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomyy896MultigroupcTWAStree04dcb6e173d4f949e04c6c06ca6358b7a76f7e22targetblank04dcb6ea">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/yy896/Multigroup_cTWAS/tree/04dcb6e173d4f949e04c6c06ca6358b7a76f7e22" target="_blank">04dcb6e</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomyy896MultigroupcTWAStree04dcb6e173d4f949e04c6c06ca6358b7a76f7e22targetblank04dcb6ea"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/yy896/Multigroup_cTWAS/tree/04dcb6e173d4f949e04c6c06ca6358b7a76f7e22" target="_blank">04dcb6e</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    analysis/.DS_Store
    Ignored:    analysis/.Rhistory

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown
(<code>analysis/cTWAS_LD_european_munro_colon_transverse7_output.Rmd</code>)
and HTML
(<code>docs/cTWAS_LD_european_munro_colon_transverse7_output.html</code>)
files. If you’ve configured a remote Git repository (see
<code>?wflow_git_remote</code>), click on the hyperlinks in the table
below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/yy896/Multigroup_cTWAS/blob/04dcb6e173d4f949e04c6c06ca6358b7a76f7e22/analysis/cTWAS_LD_european_munro_colon_transverse7_output.Rmd" target="_blank">04dcb6e</a>
</td>
<td>
yy896
</td>
<td>
2024-11-09
</td>
<td>
UPDATED RESULTS
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p>Here presenting a cTWAS results report based on real data analysis.
The analyzed trait is prostate cancer, the prediction models are Munro
et.al multimodal 7 QTLs.</p>
<div id="analysis-settings" class="section level2">
<h2>Analysis Settings</h2>
<div id="input-data" class="section level3">
<h3>Input Data</h3>
<ul>
<li>GWAS Z-scores</li>
</ul>
<p>The summary statistics for prostate cancer are downloaded from <a
href="https://www.ebi.ac.uk/gwas/downloads/summary-statistics"
class="uri">https://www.ebi.ac.uk/gwas/downloads/summary-statistics</a>,
using dataset ID: GCST90274714.</p>
<p>The sample size is</p>
<pre><code>[1] &quot;gwas_n = 726828&quot;</code></pre>
<ul>
<li>Prediction models</li>
</ul>
<p>The prediction models used in this analysis are Munro et.al 7 QTLs.
Source data: <a
href="https://www.dropbox.com/scl/fo/c88y8wu8u6beaxar3bs7g/h?rlkey=d2atycui3uyq83igk4hipu8ln&amp;dl=0"
class="uri">https://www.dropbox.com/scl/fo/c88y8wu8u6beaxar3bs7g/h?rlkey=d2atycui3uyq83igk4hipu8ln&amp;dl=0</a>.</p>
<ul>
<li>Reference data</li>
</ul>
<p>We did include ukb38 whole genome LD reference in the analysis.</p>
</div>
<div id="data-processing-and-harmonization" class="section level3">
<h3>Data processing and harmonization</h3>
<p>We map the reference SNPs to regions following the instructions from
the cTWAS tutorial.</p>
<p>When processing z-scores, we exclude multi-allelic and
strand-ambiguous variants by setting
<code>drop_multiallelic = TRUE</code> and
<code>drop_strand_ambig = TRUE</code>.</p>
<p>The process can be divided into steps below, users can expand the
code snippets below to view the exact code used.</p>
<ul>
<li>Input and output settings</li>
</ul>
<pre class="r"><code>weight_files &lt;- c(&quot;/project2/xinhe/shared_data/multigroup_ctwas/weights/Munro_alt_polyA_PredictDB/apa_Colon_Transverse.db&quot;,
                  &quot;/project2/xinhe/shared_data/multigroup_ctwas/weights/Munro_alt_stability_PredictDB/Colon_Transverse.db&quot;,
                  &quot;/project2/xinhe/shared_data/multigroup_ctwas/weights/Munro_alt_tss_PredictDB/tss_Colon_Transverse.db&quot;,
                  &quot;/project2/xinhe/shared_data/multigroup_ctwas/weights/Munro_expression_PredictDB/expression_Colon_Transverse.db&quot;,
                  &quot;/project2/xinhe/shared_data/multigroup_ctwas/weights/Munro_isoform_PredictDB/isoform_Colon_Transverse.db&quot;,
                  &quot;/project2/xinhe/shared_data/multigroup_ctwas/weights/Munro_splicing_PredictDB/splicing_Colon_Transverse.db&quot;,
                  &quot;/project2/xinhe/shared_data/multigroup_ctwas/weights/Munro_stability_PredictDB/stability_Colon_Transverse.db&quot;)

z_snp &lt;- vroom::vroom(&quot;/project2/xinhe/yifany/GCST90274714.mod.tsv.gz&quot;, col_names = TRUE, show_col_types = FALSE)
z_snp$Z &lt;- z_snp$beta / z_snp$standard_error

# gwas_n &lt;- as.numeric(names(sort(table(z_snp$base_pair_location), decreasing = TRUE)[1]))
# cat(&quot;gwas_n =&quot;, gwas_n, &quot;\n&quot;)
gwas_n &lt;- 726828

z_snp &lt;- z_snp[, c(&quot;rsid&quot;, &quot;other_allele&quot;, &quot;effect_allele&quot;, &quot;Z&quot;)]
colnames(z_snp) &lt;- c(&quot;id&quot;, &quot;A1&quot;, &quot;A2&quot;, &quot;z&quot;)

region_file &lt;- system.file(&quot;extdata/ldetect&quot;, &quot;EUR.b38.ldetect.regions.RDS&quot;, package = &quot;ctwas&quot;)
region_info &lt;- readRDS(region_file)

# output dir
outputdir &lt;- &quot;/project2/xinhe/yifany/multigroup_cTWAS&quot;
dir.create(outputdir, showWarnings=F, recursive=T)

# Number of cores
ncore &lt;- 5</code></pre>
<ul>
<li>Preprocessing GWAS</li>
</ul>
<pre class="r"><code>### Preprocess SNP_map and LD_map
genome_version &lt;- &quot;b38&quot;
region_file &lt;- system.file(&quot;extdata/ldetect&quot;, paste0(&quot;EUR.&quot;, genome_version, &quot;.ldetect.regions.RDS&quot;), package = &quot;ctwas&quot;)
region_info &lt;- readRDS(region_file)

LD_dir &lt;- &quot;/project2/mstephens/wcrouse/UKB_LDR_0.1&quot;

LD_filestem &lt;- sprintf(&quot;ukb_%s_0.1_chr%s.R_snp.%s_%s&quot;, genome_version, region_info$chrom, region_info$start, region_info$stop)

region_metatable &lt;- region_info
region_metatable$LD_file &lt;- file.path(LD_dir, paste0(LD_filestem, &quot;.RDS&quot;))
region_metatable$SNP_file &lt;- file.path(LD_dir, paste0(LD_filestem, &quot;.Rvar&quot;))

res &lt;- create_snp_LD_map(region_metatable)
region_info &lt;- res$region_info
snp_map &lt;- res$snp_map
LD_map &lt;- res$LD_map

z_snp &lt;- preprocess_z_snp(z_snp, snp_map, 
                          drop_multiallelic = TRUE, 
                          drop_strand_ambig = TRUE)</code></pre>
<ul>
<li>Preprocessing weights</li>
</ul>
<pre class="r"><code>weights_colon_transverse_apaQTL &lt;- preprocess_weights(weight_files[1],
                                                  region_info,
                                                  gwas_snp_ids = z_snp$id,
                                                  snp_map = snp_map,
                                                  LD_map = LD_map,
                                                  type = &quot;apaQTL&quot;,
                                                  context = &quot;colon_transverse&quot;,
                                                  weight_format = &quot;PredictDB&quot;,
                                                  drop_strand_ambig = TRUE,
                                                  scale_predictdb_weights = TRUE,
                                                  load_predictdb_LD = FALSE,
                                                  filter_protein_coding_genes = TRUE,
                                                  ncore = ncore)

weights_colon_transverse_astQTL &lt;- preprocess_weights(weight_files[2],
                                             region_info,
                                             gwas_snp_ids = z_snp$id,
                                             snp_map = snp_map,
                                             LD_map = LD_map,
                                             type = &quot;astQTL&quot;,
                                             context = &quot;colon_transverse&quot;,
                                             weight_format = &quot;PredictDB&quot;,
                                             drop_strand_ambig = TRUE,
                                             scale_predictdb_weights = TRUE,
                                             load_predictdb_LD = FALSE,
                                             filter_protein_coding_genes = TRUE,
                                             ncore = ncore)

weights_colon_transverse_tssQTL &lt;- preprocess_weights(weight_files[3],
                                              region_info,
                                              gwas_snp_ids = z_snp$id,
                                              snp_map = snp_map,
                                              LD_map = LD_map,
                                              type = &quot;tssQTL&quot;,
                                              context = &quot;colon_transverse&quot;,
                                              weight_format = &quot;PredictDB&quot;,
                                              drop_strand_ambig = TRUE,
                                              scale_predictdb_weights = TRUE,
                                              load_predictdb_LD = FALSE,
                                              filter_protein_coding_genes = TRUE,
                                              ncore = ncore)

weights_colon_transverse_eQTL &lt;- preprocess_weights(weight_files[4],
                                            region_info,
                                            gwas_snp_ids = z_snp$id,
                                            snp_map = snp_map,
                                            LD_map = LD_map,
                                            type = &quot;munro_eQTL&quot;,
                                            context = &quot;colon_transverse&quot;,
                                            weight_format = &quot;PredictDB&quot;,
                                            drop_strand_ambig = TRUE,
                                            scale_predictdb_weights = TRUE,
                                            load_predictdb_LD = FALSE,
                                            filter_protein_coding_genes = TRUE,
                                            ncore = ncore)

weights_colon_transverse_isoQTL &lt;- preprocess_weights(weight_files[5],
                                            region_info,
                                            gwas_snp_ids = z_snp$id,
                                            snp_map = snp_map,
                                            LD_map = LD_map,
                                            type = &quot;isoQTL&quot;,
                                            context = &quot;colon_transverse&quot;,
                                            weight_format = &quot;PredictDB&quot;,
                                            drop_strand_ambig = TRUE,
                                            scale_predictdb_weights = TRUE,
                                            load_predictdb_LD = FALSE,
                                            filter_protein_coding_genes = TRUE,
                                            ncore = ncore)

weights_colon_transverse_sQTL &lt;- preprocess_weights(weight_files[6],
                                             region_info,
                                             gwas_snp_ids = z_snp$id,
                                             snp_map = snp_map,
                                             LD_map = LD_map,
                                             type = &quot;munro_sQTL&quot;,
                                             context = &quot;colon_transverse&quot;,
                                             weight_format = &quot;PredictDB&quot;,
                                             drop_strand_ambig = TRUE,
                                             scale_predictdb_weights = TRUE,
                                             load_predictdb_LD = FALSE,
                                             filter_protein_coding_genes = TRUE,
                                             ncore = ncore)

weights_colon_transverse_stQTL &lt;- preprocess_weights(weight_files[7],
                                              region_info,
                                              gwas_snp_ids = z_snp$id,
                                              snp_map = snp_map,
                                              LD_map = LD_map,
                                              type = &quot;stQTL&quot;,
                                              context = &quot;colon_transverse&quot;,
                                              weight_format = &quot;PredictDB&quot;,
                                              drop_strand_ambig = TRUE,
                                              scale_predictdb_weights = TRUE,
                                              load_predictdb_LD = FALSE,
                                              filter_protein_coding_genes = TRUE,
                                              ncore = ncore)

weights &lt;- c(
  weights_colon_transverse_apaQTL,
  weights_colon_transverse_astQTL,
  weights_colon_transverse_tssQTL,
  weights_colon_transverse_eQTL,
  weights_colon_transverse_isoQTL,
  weights_colon_transverse_sQTL,
  weights_colon_transverse_stQTL)</code></pre>
</div>
</div>
<div id="running-ctwas-analysis" class="section level2">
<h2>Running cTWAS analysis</h2>
<p>We use the ctwas main function <code>ctwas_sumstats</code> to run the
cTWAS analysis with LD. For more details on this function, refer to the
cTWAS tutorial: <a
href="https://xinhe-lab.github.io/multigroup_ctwas/articles/running_ctwas_analysis.html"
class="uri">https://xinhe-lab.github.io/multigroup_ctwas/articles/running_ctwas_analysis.html</a></p>
<p>Users can expand the code snippets below to view the exact code
used.</p>
<pre class="r"><code>ctwas_res &lt;- ctwas_sumstats_noLD(z_snp,
                                 weights,
                                 region_info,
                                 snp_map,
                                 thin = 0.1, 
                                 ncore = 5)</code></pre>
</div>
<div id="parameter-estimation" class="section level2">
<h2>Parameter estimation</h2>
<p><code>ctwas_res</code> is the object contains the outputs of
cTWAS</p>
<p>We extract the estimated parameters by
<code>param &lt;- ctwas_res$param</code></p>
<p>We make plots using the function
<code>make_convergence_plots(param, gwas_n)</code> to see how estimated
parameters converge during the execution of the program:</p>
<p><img src="figure/cTWAS_LD_european_munro_colon_transverse7_output.Rmd/unnamed-chunk-7-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>These plots show the estimated prior inclusion probability, prior
effect size variance, enrichment and proportion of variance explained
(PVE) over the iterations of parameter estimation. The enrichment is
defined as the ratio of the prior inclusion probability of molecular
traits over the prior inclusion probability of variants. We generally
expect molecular traits to have higher prior inclusion probability than
variants. Enrichment values typically range from 20 - 100 for expression
traits.</p>
<p>Then, we use <code>summarize_param(param, gwas_n)</code> to obtain
estimated parameters (from the last iteration) and to compute the PVE by
variants and molecular traits.</p>
<pre><code>[1] &quot;The number of genes/introns/SNPs used in the analysis is:&quot;</code></pre>
<pre><code>    colon_transverse|astQTL colon_transverse|munro_eQTL 
                       6266                        4140 
    colon_transverse|isoQTL colon_transverse|munro_sQTL 
                       7410                        6175 
     colon_transverse|stQTL     colon_transverse|tssQTL 
                       2778                        3721 
    colon_transverse|apaQTL                         SNP 
                       5337                     5860240 </code></pre>
<p><code>ctwas_parameters$prop_heritability</code> contains the
proportion of heritability mediated by molecular traits and variants, we
visualize it using pie chart.</p>
<pre class="r"><code>ctwas_parameters &lt;- summarize_param(param, gwas_n)
data &lt;- data.frame(
  category = names(ctwas_parameters$prop_heritability),
  percentage = ctwas_parameters$prop_heritability
)
# Calculate percentage labels for the chart
data$percentage_label &lt;- paste0(round(data$percentage * 100), &quot;%&quot;)

ggplot(data, aes(x = &quot;&quot;, y = percentage, fill = category)) +
  geom_bar(stat = &quot;identity&quot;, width = 1) +
  coord_polar(&quot;y&quot;, start = 0) +
  theme_void() +  # Remove background and axes
  geom_text(aes(label = percentage_label), 
            position = position_stack(vjust = 0.5)) +
  scale_fill_brewer(palette = &quot;Pastel1&quot;) +
  labs(fill = &quot;Category&quot;) +
  ggtitle(&quot;Percent of heritability&quot;)</code></pre>
<p><img src="figure/cTWAS_LD_european_munro_colon_transverse7_output.Rmd/piechart-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="diagnosis-plots" class="section level2">
<h2>Diagnosis plots</h2>
<p>For all genes analyzed, we compare the z-scores and fine-mapping
PIPs”</p>
<pre class="r"><code>ggplot(data = finemap_res[finemap_res$type!=&quot;SNP&quot;,], aes(x = abs(z), y = susie_pip)) +
  geom_point() +
  labs(x = &quot;abs(z-scores)&quot;, y = &quot;PIPs&quot;) +
  theme_minimal()</code></pre>
<p><img src="figure/cTWAS_LD_european_munro_colon_transverse7_output.Rmd/unnamed-chunk-9-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="fine-mapping-results" class="section level2">
<h2>Fine-mapping results</h2>
<p>We process the fine-mapping results here.</p>
<p>We first add gene annotations to cTWAS results.</p>
<p><code>mapping_table</code> is a combination of Munro et.al 7
QTLs.</p>
<pre class="r"><code>finemap_res &lt;- anno_finemap_res(finemap_res,
                                snp_map = snp_map,
                                mapping_table = mapping_table,
                                add_gene_annot = TRUE,
                                map_by = &quot;molecular_id&quot;,
                                drop_unmapped = TRUE,
                                add_position = TRUE,
                                use_gene_pos = &quot;mid&quot;)</code></pre>
<p>The annotated fine-mapping results, ones within credible sets are
shown</p>
<pre class="r"><code>finemap_res_show &lt;- subset(finemap_res, group != &quot;SNP&quot; &amp; gene_type == &quot;protein_coding&quot; &amp; susie_pip &gt; 0.8 &amp; !is.na(cs))

DT::datatable(finemap_res_show,caption = htmltools::tags$caption( style = &#39;caption-side: topleft; text-align = left; color:black;&#39;,&#39;The annotated fine-mapping results, ones within credible sets are shown&#39;),options = list(pageLength = 5) )</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-232a8e63d6aacd296363" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-232a8e63d6aacd296363">{"x":{"filter":"none","vertical":false,"caption":"<caption style=\"caption-side: topleft; text-align = left; color:black;\">The annotated fine-mapping results, ones within credible sets are shown<\/caption>","data":[["935","1518","1822","2342","3827","5030","5660","8741","9471","10074","10707"],["ENSG00000116977:ENST00000489586|colon_transverse_isoQTL","ENSG00000145022|colon_transverse_munro_eQTL","ENSG00000168374|colon_transverse_munro_eQTL","ENSG00000188818:ENST00000503758|colon_transverse_isoQTL","ENSG00000241258|colon_transverse_munro_eQTL","ENSG00000138111:ENST00000450947|colon_transverse_isoQTL","ENSG00000170421.grp_1.upstream.ENST00000547031|colon_transverse_tssQTL","ENSG00000011143:ENST00000393119|colon_transverse_isoQTL","ENSG00000198453:ENST00000427117|colon_transverse_isoQTL","ENSG00000100997:chr20:25295030:25302219:clu_42973_-|colon_transverse_munro_sQTL","ENSG00000100354|colon_transverse_munro_eQTL"],["ENSG00000116977:ENST00000489586","ENSG00000145022","ENSG00000168374","ENSG00000188818:ENST00000503758","ENSG00000241258","ENSG00000138111:ENST00000450947","ENSG00000170421.grp_1.upstream.ENST00000547031","ENSG00000011143:ENST00000393119","ENSG00000198453:ENST00000427117","ENSG00000100997:chr20:25295030:25302219:clu_42973_-","ENSG00000100354"],["isoQTL","munro_eQTL","munro_eQTL","isoQTL","munro_eQTL","isoQTL","tssQTL","isoQTL","isoQTL","munro_sQTL","munro_eQTL"],["colon_transverse","colon_transverse","colon_transverse","colon_transverse","colon_transverse","colon_transverse","colon_transverse","colon_transverse","colon_transverse","colon_transverse","colon_transverse"],["colon_transverse|isoQTL","colon_transverse|munro_eQTL","colon_transverse|munro_eQTL","colon_transverse|isoQTL","colon_transverse|munro_eQTL","colon_transverse|isoQTL","colon_transverse|tssQTL","colon_transverse|isoQTL","colon_transverse|isoQTL","colon_transverse|munro_sQTL","colon_transverse|munro_eQTL"],["1_235656136_237392328","3_47685722_49279539","3_56399879_58171792","5_11940_982137","7_64180747_66224822","10_101189482_102620653","12_51382710_52645220","17_57280180_59410151","19_35978393_37036131","20_25363595_33027017","22_40149793_42294812"],[4.890906923824844,8.729817443944718,-4.830598086750761,4.797814198362136,4.545397522444451,-10.4909905800008,5.982127290189118,7.062348495886373,-6.213773622648656,6.886172151544451,-8.470209671539179],[0.8241820415316442,0.9999980044736907,0.8706004124005152,0.9740042574885465,0.9507574582689818,0.947713424278308,0.9999113636627363,0.9786849922921345,0.8936042242934754,0.842560943067354,0.9999999395342263],[21.77311145426621,64.61259848852875,20.41806026727815,20.98734116631151,18.18301815440295,96.81743204353263,35.82411592349772,44.38556824763651,34.57091843135249,47.16256268010397,60.88039979059176],["L1","L1","L1","L1","L1","L1","L1","L1","L1","L1","L1"],["LGALS8","TCTA","ARF4","ZDHHC11","CRCP","MFSD13A","KRT8","MKS1","ZNF568","ABHD12","TNRC6B"],["protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding"],["1","3","3","5","7","10","12","17","19","20","22"],["236518000","49412206","57571363","795606","66114604","102461395","52897187","58205437","36916329","25295030","40044817"],["236552981","49416475","57598220","850986","66154568","102477045","52949920","58219605","36997932","25302219","40335808"],[236535490,49414340,57584792,823296,66134586,102469220,52923554,58212521,36957130,25298624,40190312]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>id<\/th>\n      <th>molecular_id<\/th>\n      <th>type<\/th>\n      <th>context<\/th>\n      <th>group<\/th>\n      <th>region_id<\/th>\n      <th>z<\/th>\n      <th>susie_pip<\/th>\n      <th>mu2<\/th>\n      <th>cs<\/th>\n      <th>gene_name<\/th>\n      <th>gene_type<\/th>\n      <th>chrom<\/th>\n      <th>start<\/th>\n      <th>end<\/th>\n      <th>pos<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":[7,8,9,16]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"id","targets":1},{"name":"molecular_id","targets":2},{"name":"type","targets":3},{"name":"context","targets":4},{"name":"group","targets":5},{"name":"region_id","targets":6},{"name":"z","targets":7},{"name":"susie_pip","targets":8},{"name":"mu2","targets":9},{"name":"cs","targets":10},{"name":"gene_name","targets":11},{"name":"gene_type","targets":12},{"name":"chrom","targets":13},{"name":"start","targets":14},{"name":"end","targets":15},{"name":"pos","targets":16}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script>
<p>Next, we compute gene combined-PIPs across different types of
molecular traits, where only genes within credible sets are shown.</p>
<pre class="r"><code>susie_alpha_res &lt;- anno_susie_alpha_res(susie_alpha_res,
                                        mapping_table = mapping_table,
                                        map_by = &quot;molecular_id&quot;,
                                        drop_unmapped = TRUE)</code></pre>
<pre class="r"><code>combined_pip_by_group &lt;- combine_gene_pips(susie_alpha_res,
                                          group_by = &quot;gene_name&quot;,
                                          by = &quot;group&quot;,
                                          method = &quot;combine_cs&quot;,
                                          filter_cs = TRUE,
                                          include_cs_id = TRUE)
combined_pip_by_group$sQTL_pip_partition &lt;- sapply(combined_pip_by_group$gene_name, function(gene) {
  # Find rows in finemap_res_show matching the gene_name
  matching_rows &lt;- finemap_res_show %&gt;%
    dplyr::filter(gene_name == gene, type == &quot;sQTL&quot;)  # Match gene_name and filter by type == &quot;sQTL&quot;
  # If no matching rows, return NA
  if (nrow(matching_rows) == 0) {
    return(NA)
  }
  # Create the desired string format: molecular_id-round(susie_pip, digits = 4)
  paste(matching_rows$molecular_id, &quot;:PIP=&quot;, round(matching_rows$susie_pip, digits = 4), sep = &quot;&quot;, collapse = &quot;, &quot;)
})

combined_pip_table &lt;- subset(combined_pip_by_group, combined_pip &gt; 0.8)
DT::datatable(combined_pip_table, caption = htmltools::tags$caption(
                style = &#39;caption-side: topleft; text-align = left; color:black;&#39;,
                &#39;Gene PIPs, only genes within credible sets are shown&#39;),
              options = list(pageLength = 5))</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-2112c1f1dbc2c6427f4c" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-2112c1f1dbc2c6427f4c">{"x":{"filter":"none","vertical":false,"caption":"<caption style=\"caption-side: topleft; text-align = left; color:black;\">Gene PIPs, only genes within credible sets are shown<\/caption>","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23"],["TNRC6B","TCTA","KRT8","NICN1","MFSD13A","MKS1","VPS53","ZDHHC11","ZNF131","FGF10","CRCP","ZNF568","ARF4","UQCC1","CHMP4C","ABHD12","NEURL4","LGALS8","TOR1A","SLC39A13","KPNA4","TOMM7","RCCD1"],["22_40149793_42294812.L1","3_47685722_49279539.L1","12_51382710_52645220.L1","3_49279539_51797999.L1","10_101189482_102620653.L1","17_57280180_59410151.L1","17_150263_1269105.L1","5_11940_982137.L1","5_41888608_43983397.L1","5_43983397_50867564.L1","7_64180747_66224822.L1","19_35978393_37036131.L1","3_56399879_58171792.L1","20_34225635_36332043.L1","8_81044160_81781504.L1","20_24737088_25363595.L1,20_25363595_33027017.L1","17_5838014_7414079.L1","1_235656136_237392328.L1","9_129403191_130347830.L1","11_46984586_49844498.L1","3_159760101_161806716.L1","7_22468010_23431823.L1","15_89932319_91621162.L1"],[0.9999999395342263,0.9999980044736907,0.9999113636627363,0.9941624407713666,0.9821264447157606,0.9786849922921345,0.9764066864861842,0.9740042574885465,0.9704915988566034,0.9627660839590877,0.9507574582689818,0.9493862769378938,0.9463784370215995,0.9348828168590659,0.9309978975867832,0.9051493933252053,0.9049612287860094,0.8670660607681714,0.8552685608211281,0.8529091664793435,0.8445348454670933,0.8196162310425995,0.8151035162241227],[null,null,null,"3_49279539_51797999.L1",null,null,null,null,null,null,null,null,"3_56399879_58171792.L1","20_34225635_36332043.L1",null,"20_24737088_25363595.L1",null,"1_235656136_237392328.L1","9_129403191_130347830.L1",null,"3_159760101_161806716.L1",null,null],[null,null,null,0.2789816059815248,null,null,null,null,null,null,null,null,0.00741012552731124,0.4116050661530013,null,0.07185470138856176,null,0.04288401923652718,0.3577943138349811,null,0.004231407234732254,null,null],[null,null,null,"3_49279539_51797999.L1",null,null,null,null,"5_41888608_43983397.L1","5_43983397_50867564.L1",null,null,null,"20_34225635_36332043.L1","8_81044160_81781504.L1","20_24737088_25363595.L1","17_5838014_7414079.L1",null,"9_129403191_130347830.L1","11_46984586_49844498.L1","3_159760101_161806716.L1","7_22468010_23431823.L1",null],[null,null,null,0.4008759484322491,null,null,null,null,0.4791601186401423,0.4573364889175765,null,null,null,0.4789191433110112,0.04658209599812813,0.09540703671266637,0.2133666926045499,null,0.4576597165331917,0.4328878274074184,0.003895112714284332,0.1598153802435629,null],[null,null,"12_51382710_52645220.L1",null,null,null,"17_150263_1269105.L1",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"15_89932319_91621162.L1"],[null,null,0.9999113636627363,null,null,null,0.9764066864861842,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,0.3470457670031917],["22_40149793_42294812.L1","3_47685722_49279539.L1",null,"3_49279539_51797999.L1",null,null,null,null,null,null,"7_64180747_66224822.L1",null,"3_56399879_58171792.L1","20_34225635_36332043.L1","8_81044160_81781504.L1",null,null,null,"9_129403191_130347830.L1",null,null,"7_22468010_23431823.L1","15_89932319_91621162.L1"],[0.9999999395342263,0.9999980044736907,null,0.3143048863575929,null,null,null,null,null,null,0.9507574582689818,null,0.8706004124005152,0.02021696025144448,0.6958783795097749,null,null,null,0.03981453045295535,null,null,0.1224271996530968,0.2048701040280986],[null,null,null,null,"10_101189482_102620653.L1","17_57280180_59410151.L1",null,"5_11940_982137.L1",null,null,null,"19_35978393_37036131.L1","3_56399879_58171792.L1","20_34225635_36332043.L1",null,null,null,"1_235656136_237392328.L1",null,null,"3_159760101_161806716.L1",null,"15_89932319_91621162.L1"],[null,null,null,null,0.947713424278308,0.9786849922921345,null,0.9740042574885465,null,null,null,0.941698683593977,0.06836789909377305,0.024141647143609,null,null,null,0.8241820415316442,null,null,0.8364083255180768,null,0.2316613915878085],[null,null,null,null,"10_101189482_102620653.L1",null,null,null,null,null,null,"19_35978393_37036131.L1",null,null,null,"20_25363595_33027017.L1",null,null,null,null,null,null,"15_89932319_91621162.L1"],[null,null,null,null,0.03441302043745265,null,null,null,null,null,null,0.007687593343916777,null,null,null,0.842560943067354,null,null,null,null,null,null,0.03152625360502392],[null,null,null,null,null,null,null,null,"5_41888608_43983397.L1","5_43983397_50867564.L1",null,null,null,null,"8_81044160_81781504.L1","20_24737088_25363595.L1","17_5838014_7414079.L1",null,null,"11_46984586_49844498.L1",null,"7_22468010_23431823.L1",null],[null,null,null,null,null,null,null,null,0.491331480216461,0.5054295950415112,null,null,null,null,0.1885374220788802,0.2302790721478272,0.6915945361814594,null,null,0.420021339071925,null,0.5373736511459398,null],[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>gene_name<\/th>\n      <th>combined_cs_id<\/th>\n      <th>combined_pip<\/th>\n      <th>colon_transverse|apaQTL_cs_id<\/th>\n      <th>colon_transverse|apaQTL_pip<\/th>\n      <th>colon_transverse|astQTL_cs_id<\/th>\n      <th>colon_transverse|astQTL_pip<\/th>\n      <th>colon_transverse|tssQTL_cs_id<\/th>\n      <th>colon_transverse|tssQTL_pip<\/th>\n      <th>colon_transverse|munro_eQTL_cs_id<\/th>\n      <th>colon_transverse|munro_eQTL_pip<\/th>\n      <th>colon_transverse|isoQTL_cs_id<\/th>\n      <th>colon_transverse|isoQTL_pip<\/th>\n      <th>colon_transverse|munro_sQTL_cs_id<\/th>\n      <th>colon_transverse|munro_sQTL_pip<\/th>\n      <th>colon_transverse|stQTL_cs_id<\/th>\n      <th>colon_transverse|stQTL_pip<\/th>\n      <th>sQTL_pip_partition<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":[3,5,7,9,11,13,15,17]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"gene_name","targets":1},{"name":"combined_cs_id","targets":2},{"name":"combined_pip","targets":3},{"name":"colon_transverse|apaQTL_cs_id","targets":4},{"name":"colon_transverse|apaQTL_pip","targets":5},{"name":"colon_transverse|astQTL_cs_id","targets":6},{"name":"colon_transverse|astQTL_pip","targets":7},{"name":"colon_transverse|tssQTL_cs_id","targets":8},{"name":"colon_transverse|tssQTL_pip","targets":9},{"name":"colon_transverse|munro_eQTL_cs_id","targets":10},{"name":"colon_transverse|munro_eQTL_pip","targets":11},{"name":"colon_transverse|isoQTL_cs_id","targets":12},{"name":"colon_transverse|isoQTL_pip","targets":13},{"name":"colon_transverse|munro_sQTL_cs_id","targets":14},{"name":"colon_transverse|munro_sQTL_pip","targets":15},{"name":"colon_transverse|stQTL_cs_id","targets":16},{"name":"colon_transverse|stQTL_pip","targets":17},{"name":"sQTL_pip_partition","targets":18}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="ideogram" class="section level2">
<h2>Ideogram</h2>
<p><img src="figure/cTWAS_LD_european_munro_colon_transverse7_output.Rmd/Ideogram-1.png" width="3840" style="display: block; margin: auto;" /></p>
</div>
<div id="locus-plots-only-showing-top-10-here" class="section level2">
<h2>Locus plots (ONLY showing top 10 here)</h2>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;22_40149793_42294812&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-11-09 04:44:37.431462 INFO::Limit to protein coding genes
2024-11-09 04:44:37.435486 INFO::focal id: ENSG00000100354|colon_transverse_munro_eQTL
2024-11-09 04:44:37.439283 INFO::focal molecular trait: TNRC6B colon_transverse munro_eQTL
2024-11-09 04:44:37.439529 INFO::Range of locus: chr22:40149693-42294356</code></pre>
<pre><code>chromosome 22, position 40149693 to 42294356</code></pre>
<pre><code>3856 SNPs/datapoints</code></pre>
<pre><code>2024-11-09 04:44:37.537452 INFO::focal molecular trait QTL positions: 40156115,40254280
2024-11-09 04:44:37.552688 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 86 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_munro_colon_transverse7_output.Rmd/locus%20plot-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;3_47685722_49279539&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-11-09 04:44:38.220019 INFO::Limit to protein coding genes
2024-11-09 04:44:38.220575 INFO::focal id: ENSG00000145022|colon_transverse_munro_eQTL
2024-11-09 04:44:38.220809 INFO::focal molecular trait: TCTA colon_transverse munro_eQTL
2024-11-09 04:44:38.22103 INFO::Range of locus: chr3:47688075-49426236</code></pre>
<pre><code>chromosome 3, position 47688075 to 49426236</code></pre>
<pre><code>1827 SNPs/datapoints</code></pre>
<pre><code>2024-11-09 04:44:38.431232 INFO::focal molecular trait QTL positions: 49179039,49191513,49260772
2024-11-09 04:44:38.444575 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 107 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_munro_colon_transverse7_output.Rmd/locus%20plot-2.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;12_51382710_52645220&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-11-09 04:44:39.128314 INFO::Limit to protein coding genes
2024-11-09 04:44:39.128877 INFO::focal id: ENSG00000170421.grp_1.upstream.ENST00000547031|colon_transverse_tssQTL
2024-11-09 04:44:39.129124 INFO::focal molecular trait: KRT8 colon_transverse tssQTL
2024-11-09 04:44:39.129366 INFO::Range of locus: chr12:51371957-52923654</code></pre>
<pre><code>chromosome 12, position 51371957 to 52923654</code></pre>
<pre><code>3637 SNPs/datapoints</code></pre>
<pre><code>2024-11-09 04:44:39.200239 INFO::focal molecular trait QTL positions: 52534131,52539715
2024-11-09 04:44:39.213157 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 7 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_munro_colon_transverse7_output.Rmd/locus%20plot-3.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;3_49279539_51797999&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-11-09 04:44:39.799635 INFO::Limit to protein coding genes
2024-11-09 04:44:39.800203 INFO::focal id: ENSG00000145029.grp_1.downstream.ENST00000423832|colon_transverse_astQTL
2024-11-09 04:44:39.800442 INFO::focal molecular trait: NICN1 colon_transverse astQTL
2024-11-09 04:44:39.800678 INFO::Range of locus: chr3:48996076-51794719</code></pre>
<pre><code>chromosome 3, position 48996076 to 51794719</code></pre>
<pre><code>3519 SNPs/datapoints</code></pre>
<pre><code>2024-11-09 04:44:39.885092 INFO::focal molecular trait QTL positions: 49288745,49298312,49359318,49386805,49412205,49422917,49582060,49772408,49890070
2024-11-09 04:44:39.898126 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 110 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<pre><code>Warning: ggrepel: 2 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_munro_colon_transverse7_output.Rmd/locus%20plot-4.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;10_101189482_102620653&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-11-09 04:44:40.773504 INFO::Limit to protein coding genes
2024-11-09 04:44:40.77405 INFO::focal id: ENSG00000138111:ENST00000450947|colon_transverse_isoQTL
2024-11-09 04:44:40.774285 INFO::focal molecular trait: MFSD13A colon_transverse isoQTL
2024-11-09 04:44:40.774509 INFO::Range of locus: chr10:101190250-102619931</code></pre>
<pre><code>chromosome 10, position 101190250 to 102619931</code></pre>
<pre><code>2371 SNPs/datapoints</code></pre>
<pre><code>2024-11-09 04:44:40.823292 INFO::focal molecular trait QTL positions: 102476724,102476785,102476795,102479343,102504350,102582744,102591860
2024-11-09 04:44:40.836157 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 43 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<pre><code>Warning: ggrepel: 1 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_munro_colon_transverse7_output.Rmd/locus%20plot-5.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;17_57280180_59410151&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-11-09 04:44:41.544859 INFO::Limit to protein coding genes
2024-11-09 04:44:41.54546 INFO::focal id: ENSG00000011143:ENST00000393119|colon_transverse_isoQTL
2024-11-09 04:44:41.545712 INFO::focal molecular trait: MKS1 colon_transverse isoQTL
2024-11-09 04:44:41.545956 INFO::Range of locus: chr17:57280255-59409129</code></pre>
<pre><code>chromosome 17, position 57280255 to 59409129</code></pre>
<pre><code>4306 SNPs/datapoints</code></pre>
<pre><code>2024-11-09 04:44:41.614993 INFO::focal molecular trait QTL positions: 57817203,58283476,58343909
2024-11-09 04:44:41.62809 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 32 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_munro_colon_transverse7_output.Rmd/locus%20plot-6.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;17_150263_1269105&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-11-09 04:44:42.268746 INFO::Limit to protein coding genes
2024-11-09 04:44:42.269278 INFO::focal id: ENSG00000141252.grp_1.upstream.ENST00000681946|colon_transverse_tssQTL
2024-11-09 04:44:42.26953 INFO::focal molecular trait: VPS53 colon_transverse tssQTL
2024-11-09 04:44:42.269787 INFO::Range of locus: chr17:150409-1268542</code></pre>
<pre><code>chromosome 17, position 150409 to 1268542</code></pre>
<pre><code>4129 SNPs/datapoints</code></pre>
<pre><code>2024-11-09 04:44:42.320601 INFO::focal molecular trait QTL positions: 715725,715922,716602,781489,782400
2024-11-09 04:44:42.333774 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 18 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_munro_colon_transverse7_output.Rmd/locus%20plot-7.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;5_11940_982137&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-11-09 04:44:43.122659 INFO::Limit to protein coding genes
2024-11-09 04:44:43.12323 INFO::focal id: ENSG00000188818:ENST00000503758|colon_transverse_isoQTL
2024-11-09 04:44:43.123471 INFO::focal molecular trait: ZDHHC11 colon_transverse isoQTL
2024-11-09 04:44:43.123702 INFO::Range of locus: chr5:12125-981935</code></pre>
<pre><code>chromosome 5, position 12125 to 981935</code></pre>
<pre><code>2993 SNPs/datapoints</code></pre>
<pre><code>2024-11-09 04:44:43.190974 INFO::focal molecular trait QTL positions: 409645,480833,516693,539195,570433,659490,851467,919382
2024-11-09 04:44:43.205375 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 41 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_munro_colon_transverse7_output.Rmd/locus%20plot-8.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;5_41888608_43983397&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-11-09 04:44:43.924403 INFO::Limit to protein coding genes
2024-11-09 04:44:43.925022 INFO::focal id: ENSG00000172262|colon_transverse_stQTL
2024-11-09 04:44:43.925295 INFO::focal molecular trait: ZNF131 colon_transverse stQTL
2024-11-09 04:44:43.925551 INFO::Range of locus: chr5:41888508-43982603</code></pre>
<pre><code>chromosome 5, position 41888508 to 43982603</code></pre>
<pre><code>4874 SNPs/datapoints</code></pre>
<pre><code>2024-11-09 04:44:43.985087 INFO::focal molecular trait QTL positions: 43101962,43143697,43232209,43233163,43401878
2024-11-09 04:44:43.998499 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 14 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_munro_colon_transverse7_output.Rmd/locus%20plot-9.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>make_locusplot(finemap_res,
               region_id = &quot;5_43983397_50867564&quot;,
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = &quot;cs&quot;,
               color_pip_by = &quot;cs&quot;)</code></pre>
<pre><code>2024-11-09 04:44:44.595017 INFO::Limit to protein coding genes
2024-11-09 04:44:44.595651 INFO::focal id: ENSG00000070193|colon_transverse_stQTL
2024-11-09 04:44:44.595938 INFO::focal molecular trait: FGF10 colon_transverse stQTL
2024-11-09 04:44:44.59622 INFO::Range of locus: chr5:43983297-50867351</code></pre>
<pre><code>chromosome 5, position 43983297 to 50867351</code></pre>
<pre><code>6968 SNPs/datapoints</code></pre>
<pre><code>2024-11-09 04:44:44.64635 INFO::focal molecular trait QTL positions: 44318430,44328168,44331803,44365443,44376120,44576069
2024-11-09 04:44:44.659527 INFO::Limit PIPs to credible sets</code></pre>
<pre><code>Warning: ggrepel: 14 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<pre><code>Warning: ggrepel: 3 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/cTWAS_LD_european_munro_colon_transverse7_output.Rmd/locus%20plot-10.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="gsea-gene-ontology" class="section level2">
<h2>GSEA-Gene Ontology</h2>
<pre class="r"><code>dbs &lt;- c(&quot;GO_Biological_Process_2021&quot;, &quot;GO_Cellular_Component_2021&quot;, &quot;GO_Molecular_Function_2021&quot;)
genes &lt;- combined_pip_by_group$gene_name[combined_pip_by_group$combined_pip &gt;0.8]

#number of genes for gene set enrichment
sprintf(&quot;The number of genes used in enrichment analysis = %s&quot;, length(genes))</code></pre>
<pre><code>[1] &quot;The number of genes used in enrichment analysis = 23&quot;</code></pre>
<pre class="r"><code>GO_enrichment &lt;- enrichr(genes, dbs)</code></pre>
<pre><code>Uploading data to Enrichr... Done.
  Querying GO_Biological_Process_2021... Done.
  Querying GO_Cellular_Component_2021... Done.
  Querying GO_Molecular_Function_2021... Done.
Parsing results... Done.</code></pre>
<pre class="r"><code>print(&quot;GO_Biological_Process_2021&quot;)</code></pre>
<pre><code>[1] &quot;GO_Biological_Process_2021&quot;</code></pre>
<pre class="r"><code>db &lt;- &quot;GO_Biological_Process_2021&quot;
df &lt;- GO_enrichment[[db]]
print(plotEnrich(GO_enrichment[[db]]))</code></pre>
<p><img src="figure/cTWAS_LD_european_munro_colon_transverse7_output.Rmd/GO_Biological_Process_2021-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>df &lt;- df[df$Adjusted.P.value&lt;0.05,c(&quot;Term&quot;, &quot;Overlap&quot;, &quot;Adjusted.P.value&quot;, &quot;Genes&quot;)]

DT::datatable(df,caption = htmltools::tags$caption( style = &#39;caption-side: topleft; text-align = left; color:black;&#39;,&#39;Enriched pathways from GO_Biological_Process_2021&#39;),options = list(pageLength = 5) )</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-57a5e797e447beab7be8" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-57a5e797e447beab7be8">{"x":{"filter":"none","vertical":false,"caption":"<caption style=\"caption-side: topleft; text-align = left; color:black;\">Enriched pathways from GO_Biological_Process_2021<\/caption>","data":[[],[],[],[],[]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":3},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Term","targets":1},{"name":"Overlap","targets":2},{"name":"Adjusted.P.value","targets":3},{"name":"Genes","targets":4}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>print(&quot;GO_Cellular_Component_2021&quot;)</code></pre>
<pre><code>[1] &quot;GO_Cellular_Component_2021&quot;</code></pre>
<pre class="r"><code>db &lt;- &quot;GO_Cellular_Component_2021&quot;
df &lt;- GO_enrichment[[db]]
print(plotEnrich(GO_enrichment[[db]]))</code></pre>
<p><img src="figure/cTWAS_LD_european_munro_colon_transverse7_output.Rmd/GO_Cellular_Component_2021-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>df &lt;- df[df$Adjusted.P.value&lt;0.05,c(&quot;Term&quot;, &quot;Overlap&quot;, &quot;Adjusted.P.value&quot;, &quot;Genes&quot;)]

DT::datatable(df,caption = htmltools::tags$caption( style = &#39;caption-side: topleft; text-align = left; color:black;&#39;,&#39;Enriched pathways from GO_Cellular_Component_2021&#39;),options = list(pageLength = 5) )</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-0a34d9ca313023088b60" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-0a34d9ca313023088b60">{"x":{"filter":"none","vertical":false,"caption":"<caption style=\"caption-side: topleft; text-align = left; color:black;\">Enriched pathways from GO_Cellular_Component_2021<\/caption>","data":[[],[],[],[],[]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":3},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Term","targets":1},{"name":"Overlap","targets":2},{"name":"Adjusted.P.value","targets":3},{"name":"Genes","targets":4}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>print(&quot;GO_Molecular_Function_2021&quot;)</code></pre>
<pre><code>[1] &quot;GO_Molecular_Function_2021&quot;</code></pre>
<pre class="r"><code>db &lt;- &quot;GO_Molecular_Function_2021&quot;
df &lt;- GO_enrichment[[db]]
print(plotEnrich(GO_enrichment[[db]]))</code></pre>
<p><img src="figure/cTWAS_LD_european_munro_colon_transverse7_output.Rmd/GO_Molecular_Function_2021-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>df &lt;- df[df$Adjusted.P.value&lt;0.05,c(&quot;Term&quot;, &quot;Overlap&quot;, &quot;Adjusted.P.value&quot;, &quot;Genes&quot;)]

DT::datatable(df,caption = htmltools::tags$caption( style = &#39;caption-side: topleft; text-align = left; color:black;&#39;,&#39;Enriched pathways from GO_Molecular_Function_2021&#39;),options = list(pageLength = 5) )</code></pre>
<div class="datatables html-widget html-fill-item" id="htmlwidget-38f6575cbe0384702c9e" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-38f6575cbe0384702c9e">{"x":{"filter":"none","vertical":false,"caption":"<caption style=\"caption-side: topleft; text-align = left; color:black;\">Enriched pathways from GO_Molecular_Function_2021<\/caption>","data":[[],[],[],[],[]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Term<\/th>\n      <th>Overlap<\/th>\n      <th>Adjusted.P.value<\/th>\n      <th>Genes<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":5,"columnDefs":[{"className":"dt-right","targets":3},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Term","targets":1},{"name":"Overlap","targets":2},{"name":"Adjusted.P.value","targets":3},{"name":"Genes","targets":4}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":[],"jsHooks":[]}</script>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.2 (2024-10-31)
Platform: aarch64-apple-darwin20
Running under: macOS Sequoia 15.1

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: America/New_York
tzcode source: internal

attached base packages:
[1] stats4    stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] biomaRt_2.60.1            karyoploteR_1.30.0       
 [3] regioneR_1.36.0           htmltools_0.5.8.1        
 [5] DT_0.33                   enrichR_3.2              
 [7] EnsDb.Hsapiens.v86_2.99.0 ensembldb_2.28.1         
 [9] AnnotationFilter_1.28.0   GenomicFeatures_1.56.0   
[11] AnnotationDbi_1.66.0      Biobase_2.64.0           
[13] GenomicRanges_1.56.2      GenomeInfoDb_1.40.1      
[15] IRanges_2.38.1            S4Vectors_0.42.1         
[17] BiocGenerics_0.50.0       ctwas_0.4.18             
[19] data.table_1.16.2         lubridate_1.9.3          
[21] forcats_1.0.0             stringr_1.5.1            
[23] dplyr_1.1.4               purrr_1.0.2              
[25] readr_2.1.5               tidyr_1.3.1              
[27] tibble_3.2.1              ggplot2_3.5.1            
[29] tidyverse_2.0.0          

loaded via a namespace (and not attached):
  [1] later_1.3.2                 BiocIO_1.14.0              
  [3] bitops_1.0-9                filelock_1.0.3             
  [5] XML_3.99-0.17               rpart_4.1.23               
  [7] lifecycle_1.0.4             httr2_1.0.6                
  [9] mixsqp_0.3-54               rprojroot_2.0.4            
 [11] lattice_0.22-6              crosstalk_1.2.1            
 [13] backports_1.5.0             magrittr_2.0.3             
 [15] locuszoomr_0.3.5            Hmisc_5.2-0                
 [17] plotly_4.10.4               sass_0.4.9                 
 [19] rmarkdown_2.29              jquerylib_0.1.4            
 [21] yaml_2.3.10                 httpuv_1.6.15              
 [23] cowplot_1.1.3               DBI_1.2.3                  
 [25] RColorBrewer_1.1-3          abind_1.4-8                
 [27] zlibbioc_1.50.0             pgenlibr_0.3.7             
 [29] biovizBase_1.52.0           RCurl_1.98-1.16            
 [31] nnet_7.3-19                 WriteXLS_6.7.0             
 [33] VariantAnnotation_1.50.0    rappdirs_0.3.3             
 [35] git2r_0.35.0                GenomeInfoDbData_1.2.12    
 [37] ggrepel_0.9.6               irlba_2.3.5.1              
 [39] codetools_0.2-20            DelayedArray_0.30.1        
 [41] xml2_1.3.6                  tidyselect_1.2.1           
 [43] UCSC.utils_1.0.0            farver_2.1.2               
 [45] matrixStats_1.4.1           BiocFileCache_2.12.0       
 [47] base64enc_0.1-3             bamsignals_1.36.0          
 [49] GenomicAlignments_1.40.0    jsonlite_1.8.9             
 [51] Formula_1.2-5               tools_4.4.2                
 [53] progress_1.2.3              Rcpp_1.0.13-1              
 [55] glue_1.8.0                  gridExtra_2.3              
 [57] SparseArray_1.4.8           xfun_0.49                  
 [59] MatrixGenerics_1.16.0       withr_3.0.2                
 [61] fastmap_1.2.0               LDlinkR_1.4.0              
 [63] fansi_1.0.6                 digest_0.6.37              
 [65] timechange_0.3.0            R6_2.5.1                   
 [67] colorspace_2.1-1            dichromat_2.0-0.1          
 [69] RSQLite_2.3.7               utf8_1.2.4                 
 [71] generics_0.1.3              rtracklayer_1.64.0         
 [73] prettyunits_1.2.0           httr_1.4.7                 
 [75] htmlwidgets_1.6.4           S4Arrays_1.4.1             
 [77] whisker_0.4.1               pkgconfig_2.0.3            
 [79] gtable_0.3.6                blob_1.2.4                 
 [81] workflowr_1.7.1             XVector_0.44.0             
 [83] gggrid_0.2-0                ProtGenerics_1.36.0        
 [85] scales_1.3.0                logging_0.10-108           
 [87] png_0.1-8                   knitr_1.49                 
 [89] rstudioapi_0.17.1           tzdb_0.4.0                 
 [91] rjson_0.2.23                checkmate_2.3.2            
 [93] curl_6.0.0                  cachem_1.1.0               
 [95] zoo_1.8-12                  parallel_4.4.2             
 [97] foreign_0.8-87              restfulr_0.0.15            
 [99] pillar_1.9.0                grid_4.4.2                 
[101] vctrs_0.6.5                 promises_1.3.0             
[103] dbplyr_2.5.0                cluster_2.1.6              
[105] htmlTable_2.4.3             evaluate_1.0.1             
[107] cli_3.6.3                   compiler_4.4.2             
[109] bezier_1.1.2                Rsamtools_2.20.0           
[111] rlang_1.1.4                 crayon_1.5.3               
[113] labeling_0.4.3              fs_1.6.5                   
[115] stringi_1.8.4               viridisLite_0.4.2          
[117] BiocParallel_1.38.0         munsell_0.5.1              
[119] Biostrings_2.72.1           lazyeval_0.2.2             
[121] Matrix_1.7-1                BSgenome_1.72.0            
[123] hms_1.1.3                   bit64_4.5.2                
[125] KEGGREST_1.44.1             SummarizedExperiment_1.34.0
[127] memoise_2.0.1               bslib_0.8.0                
[129] bit_4.5.0                  </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
