---
title: "Prostate Cancer ~ Munro 7 fibroblasts QTLs"
author: "Nina Yifan Yang"
date: "`r Sys.Date()`"
output:
  workflowr::wflow_html:
    code_folding: hide
    toc: true
---

```{r echo=FALSE, message=FALSE, warning=FALSE}
library(tidyverse)
library(readr)
library(data.table)
library(stringr)
library(ctwas)
library(GenomicRanges)
library(EnsDb.Hsapiens.v86)
library(enrichR)
library(DT)
library(htmltools)

ens_db <- EnsDb.Hsapiens.v86
weights <- readRDS(system.file("extdata/project_data", "European.munro_fibroblasts7_weights.RDS", package = "ctwas"))
ctwas_res <- readRDS(system.file("extdata/project_data", "European.munro_fibroblasts7_weights.ctwas_noLD_allchrs.RDS", package = "ctwas"))
gwas_n <- 726828
finemap_res <- readRDS("~/Desktop/Prostate Cancer Project/European.munro_fibroblasts_weights.noLD.finemap_res.RDS")
susie_alpha_res <- readRDS("~/Desktop/Prostate Cancer Project/European.munro_fibroblasts_weights.noLD.susie_alpha_res.RDS")
#finemap_res_sub <- subset(finemap_res, group != "SNP" & gene_type == "protein_coding" & susie_pip > 0.8 & !is.na(cs))
```

Here presenting a cTWAS results report based on real data analysis. The analyzed trait is prostate cancer, the prediction models are Munro et.al multimodal 7 QTLs.

## Analysis Settings

### Input Data

* GWAS Z-scores

The summary statistics for prostate cancer are downloaded from https://www.ebi.ac.uk/gwas/downloads/summary-statistics, using dataset ID: GCST90274714.

The sample size is

```{r echo=FALSE}
print("gwas_n = 726828")
```

* Prediction models

The prediction models used in this analysis are Munro et.al 7 QTLs. Source data: https://www.dropbox.com/scl/fo/c88y8wu8u6beaxar3bs7g/h?rlkey=d2atycui3uyq83igk4hipu8ln&dl=0.

* Reference data

We did include ukb38 whole genome LD reference in the analysis.

### Data processing and harmonization

We map the reference SNPs to regions following the instructions from the cTWAS tutorial.

When processing z-scores, we exclude multi-allelic and strand-ambiguous variants by setting `drop_multiallelic = TRUE` and `drop_strand_ambig = TRUE`.

The process can be divided into steps below, users can expand the code snippets below to view the exact code used.

* Input and output settings

```{r echo=TRUE, eval=FALSE}
weight_files <- c("/project2/xinhe/shared_data/multigroup_ctwas/weights/Munro_alt_polyA_PredictDB/apa_Cells_Cultured_fibroblasts.db",
                  "/project2/xinhe/shared_data/multigroup_ctwas/weights/Munro_alt_stability_PredictDB/Cells_Cultured_fibroblasts.db",
                  "/project2/xinhe/shared_data/multigroup_ctwas/weights/Munro_alt_tss_PredictDB/tss_Cells_Cultured_fibroblasts.db",
                  "/project2/xinhe/shared_data/multigroup_ctwas/weights/Munro_expression_PredictDB/expression_Cells_Cultured_fibroblasts.db",
                  "/project2/xinhe/shared_data/multigroup_ctwas/weights/Munro_isoform_PredictDB/isoform_Cells_Cultured_fibroblasts.db",
                  "/project2/xinhe/shared_data/multigroup_ctwas/weights/Munro_splicing_PredictDB/splicing_Cells_Cultured_fibroblasts.db",
                  "/project2/xinhe/shared_data/multigroup_ctwas/weights/Munro_stability_PredictDB/stability_Cells_Cultured_fibroblasts.db")

z_snp <- vroom::vroom("/project2/xinhe/yifany/GCST90274714.mod.tsv.gz", col_names = TRUE, show_col_types = FALSE)
z_snp$Z <- z_snp$beta / z_snp$standard_error

# gwas_n <- as.numeric(names(sort(table(z_snp$base_pair_location), decreasing = TRUE)[1]))
# cat("gwas_n =", gwas_n, "\n")
gwas_n <- 726828

z_snp <- z_snp[, c("rsid", "other_allele", "effect_allele", "Z")]
colnames(z_snp) <- c("id", "A1", "A2", "z")

region_file <- system.file("extdata/ldetect", "EUR.b38.ldetect.regions.RDS", package = "ctwas")
region_info <- readRDS(region_file)

# output dir
outputdir <- "/project2/xinhe/yifany/multigroup_cTWAS"
dir.create(outputdir, showWarnings=F, recursive=T)

# Number of cores
ncore <- 5
```

* Preprocessing GWAS

```{r echo=TRUE, eval=FALSE}
### Preprocess SNP_map and LD_map
genome_version <- "b38"
region_file <- system.file("extdata/ldetect", paste0("EUR.", genome_version, ".ldetect.regions.RDS"), package = "ctwas")
region_info <- readRDS(region_file)

LD_dir <- "/project2/mstephens/wcrouse/UKB_LDR_0.1"

LD_filestem <- sprintf("ukb_%s_0.1_chr%s.R_snp.%s_%s", genome_version, region_info$chrom, region_info$start, region_info$stop)

region_metatable <- region_info
region_metatable$LD_file <- file.path(LD_dir, paste0(LD_filestem, ".RDS"))
region_metatable$SNP_file <- file.path(LD_dir, paste0(LD_filestem, ".Rvar"))

res <- create_snp_LD_map(region_metatable)
region_info <- res$region_info
snp_map <- res$snp_map
LD_map <- res$LD_map

z_snp <- preprocess_z_snp(z_snp, snp_map, 
                          drop_multiallelic = TRUE, 
                          drop_strand_ambig = TRUE)
```

* Preprocessing weights

```{r echo=TRUE, eval=FALSE}
weights_fibroblasts_apaQTL <- preprocess_weights(weight_files[1],
                                              region_info,
                                              gwas_snp_ids = z_snp$id,
                                              snp_map = snp_map,
                                              LD_map = LD_map,
                                              type = "apaQTL",
                                              context = "fibroblasts",
                                              weight_format = "PredictDB",
                                              drop_strand_ambig = TRUE,
                                              scale_predictdb_weights = TRUE,
                                              load_predictdb_LD = FALSE,
                                              filter_protein_coding_genes = TRUE,
                                              ncore = 5)

weights_fibroblasts_astQTL <- preprocess_weights(weight_files[2],
                                              region_info,
                                              gwas_snp_ids = z_snp$id,
                                              snp_map = snp_map,
                                              LD_map = LD_map,
                                              type = "astQTL",
                                              context = "fibroblasts",
                                              weight_format = "PredictDB",
                                              drop_strand_ambig = TRUE,
                                              scale_predictdb_weights = TRUE,
                                              load_predictdb_LD = FALSE,
                                              filter_protein_coding_genes = TRUE,
                                              ncore = 5)

weights_fibroblasts_tssQTL <- preprocess_weights(weight_files[3],
                                              region_info,
                                              gwas_snp_ids = z_snp$id,
                                              snp_map = snp_map,
                                              LD_map = LD_map,
                                              type = "tssQTL",
                                              context = "fibroblasts",
                                              weight_format = "PredictDB",
                                              drop_strand_ambig = TRUE,
                                              scale_predictdb_weights = TRUE,
                                              load_predictdb_LD = FALSE,
                                              filter_protein_coding_genes = TRUE,
                                              ncore = 5)

weights_fibroblasts_eQTL <- preprocess_weights(weight_files[4],
                                            region_info,
                                            gwas_snp_ids = z_snp$id,
                                            snp_map = snp_map,
                                            LD_map = LD_map,
                                            type = "munro_eQTL",
                                            context = "fibroblasts",
                                            weight_format = "PredictDB",
                                            drop_strand_ambig = TRUE,
                                            scale_predictdb_weights = TRUE,
                                            load_predictdb_LD = FALSE,
                                            filter_protein_coding_genes = TRUE,
                                            ncore = 5)

weights_fibroblasts_isoQTL <- preprocess_weights(weight_files[5],
                                              region_info,
                                              gwas_snp_ids = z_snp$id,
                                              snp_map = snp_map,
                                              LD_map = LD_map,
                                              type = "isoQTL",
                                              context = "fibroblasts",
                                              weight_format = "PredictDB",
                                              drop_strand_ambig = TRUE,
                                              scale_predictdb_weights = TRUE,
                                              load_predictdb_LD = FALSE,
                                              filter_protein_coding_genes = TRUE,
                                              ncore = 5)

weights_fibroblasts_sQTL <- preprocess_weights(weight_files[6],
                                            region_info,
                                            gwas_snp_ids = z_snp$id,
                                            snp_map = snp_map,
                                            LD_map = LD_map,
                                            type = "munro_sQTL",
                                            context = "fibroblasts",
                                            weight_format = "PredictDB",
                                            drop_strand_ambig = TRUE,
                                            scale_predictdb_weights = TRUE,
                                            load_predictdb_LD = FALSE,
                                            filter_protein_coding_genes = TRUE,
                                            ncore = 5)

weights_fibroblasts_stQTL <- preprocess_weights(weight_files[7],
                                             region_info,
                                             gwas_snp_ids = z_snp$id,
                                             snp_map = snp_map,
                                             LD_map = LD_map,
                                             type = "stQTL",
                                             context = "fibroblasts",
                                             weight_format = "PredictDB",
                                             drop_strand_ambig = TRUE,
                                             scale_predictdb_weights = TRUE,
                                             load_predictdb_LD = FALSE,
                                             filter_protein_coding_genes = TRUE,
                                             ncore = 5)

weights <- c(
  weights_fibroblasts_apaQTL,
  weights_fibroblasts_astQTL,
  weights_fibroblasts_tssQTL,
  weights_fibroblasts_eQTL,
  weights_fibroblasts_isoQTL,
  weights_fibroblasts_sQTL,
  weights_fibroblasts_stQTL)
```

## Running cTWAS analysis

We use the ctwas main function `ctwas_sumstats` to run the cTWAS analysis with LD. For more details on this function, refer to the cTWAS tutorial: https://xinhe-lab.github.io/multigroup_ctwas/articles/running_ctwas_analysis.html

Users can expand the code snippets below to view the exact code used.

```{r echo=TRUE, eval=FALSE}
ctwas_res <- ctwas_sumstats_noLD(z_snp,
                                 weights,
                                 region_info,
                                 snp_map,
                                 thin = 0.1, 
                                 ncore = 5)
```

## Parameter estimation

`ctwas_res` is the object contains the outputs of cTWAS

We extract the estimated parameters by `param <- ctwas_res$param`

We make plots using the function `make_convergence_plots(param, gwas_n)` to see how estimated parameters converge during the execution of the program:

```{r echo=FALSE, eval=TRUE, fig.height=6, fig.width=10}
param <- ctwas_res$param
make_convergence_plots(param, gwas_n)
```

These plots show the estimated prior inclusion probability, prior effect size variance, enrichment and proportion of variance explained (PVE) over the iterations of parameter estimation. The enrichment is defined as the ratio of the prior inclusion probability of molecular traits over the prior inclusion probability of variants. We generally expect molecular traits to have higher prior inclusion probability than variants. Enrichment values typically range from 20 - 100 for expression traits.

Then, we use `summarize_param(param, gwas_n)` to obtain estimated parameters (from the last iteration) and to compute the PVE by variants and molecular traits.

```{r echo=FALSE, eval=TRUE}
ctwas_parameters <- summarize_param(param, gwas_n)

print("The number of genes/introns/SNPs used in the analysis is:")
ctwas_parameters$group_size
```

`ctwas_parameters$prop_heritability` contains the proportion of heritability mediated by molecular traits and variants, we visualize it using pie chart.

```{r piechart, echo=TRUE, eval=TRUE, fig.height=7, fig.width=7}
ctwas_parameters <- summarize_param(param, gwas_n)
data <- data.frame(
  category = names(ctwas_parameters$prop_heritability),
  percentage = ctwas_parameters$prop_heritability
)
# Calculate percentage labels for the chart
data$percentage_label <- paste0(round(data$percentage * 100), "%")

ggplot(data, aes(x = "", y = percentage, fill = category)) +
  geom_bar(stat = "identity", width = 1) +
  coord_polar("y", start = 0) +
  theme_void() +  # Remove background and axes
  geom_text(aes(label = percentage_label), 
            position = position_stack(vjust = 0.5)) +
  scale_fill_brewer(palette = "Pastel1") +
  labs(fill = "Category") +
  ggtitle("Percent of heritability")
```

## Diagnosis plots

For all genes analyzed, we compare the z-scores and fine-mapping PIPs"

```{r echo=TRUE, eval=TRUE}
ggplot(data = finemap_res[finemap_res$type!="SNP",], aes(x = abs(z), y = susie_pip)) +
  geom_point() +
  labs(x = "abs(z-scores)", y = "PIPs") +
  theme_minimal()
```

## Fine-mapping results

We process the fine-mapping results here.

We first add gene annotations to cTWAS results.

`mapping_table` is a combination of Munro et.al 7 QTLs.

```{r echo=TRUE, eval=FALSE}
finemap_res <- anno_finemap_res(finemap_res,
                                snp_map = snp_map,
                                mapping_table = mapping_table,
                                add_gene_annot = TRUE,
                                map_by = "molecular_id",
                                drop_unmapped = TRUE,
                                add_position = TRUE,
                                use_gene_pos = "mid")
```

The annotated fine-mapping results, ones within credible sets are shown

```{r echo=TRUE, eval=TRUE}
finemap_res_show <- subset(finemap_res, group != "SNP" & gene_type == "protein_coding" & susie_pip > 0.8 & !is.na(cs))

DT::datatable(finemap_res_show,caption = htmltools::tags$caption( style = 'caption-side: topleft; text-align = left; color:black;','The annotated fine-mapping results, ones within credible sets are shown'),options = list(pageLength = 5) )
```

Next, we compute gene combined-PIPs across different types of molecular traits, where only genes within credible sets are shown.

```{r echo=TRUE, eval=FALSE}
susie_alpha_res <- anno_susie_alpha_res(susie_alpha_res,
                                        mapping_table = mapping_table,
                                        map_by = "molecular_id",
                                        drop_unmapped = TRUE)
```

```{r echo=TRUE, eval=TRUE}
combined_pip_by_group <- combine_gene_pips(susie_alpha_res,
                                          group_by = "gene_name",
                                          by = "group",
                                          method = "combine_cs",
                                          filter_cs = TRUE,
                                          include_cs_id = TRUE)
combined_pip_by_group$sQTL_pip_partition <- sapply(combined_pip_by_group$gene_name, function(gene) {
  # Find rows in finemap_res_show matching the gene_name
  matching_rows <- finemap_res_show %>%
    dplyr::filter(gene_name == gene, type == "sQTL")  # Match gene_name and filter by type == "sQTL"
  # If no matching rows, return NA
  if (nrow(matching_rows) == 0) {
    return(NA)
  }
  # Create the desired string format: molecular_id-round(susie_pip, digits = 4)
  paste(matching_rows$molecular_id, ":PIP=", round(matching_rows$susie_pip, digits = 4), sep = "", collapse = ", ")
})

combined_pip_table <- subset(combined_pip_by_group, combined_pip > 0.8)
DT::datatable(combined_pip_table, caption = htmltools::tags$caption(
                style = 'caption-side: topleft; text-align = left; color:black;',
                'Gene PIPs, only genes within credible sets are shown'),
              options = list(pageLength = 5))
```

## Ideogram

```{r Ideogram, eval=TRUE, echo=FALSE, fig.height=28, fig.width=40, message=FALSE, warning=FALSE}
library(karyoploteR)
library(biomaRt)
library(regioneR)
gene.symbols <- combined_pip_table$gene_name
ensembl <- useEnsembl(biomart="ensembl", dataset="hsapiens_gene_ensembl")
genes <- toGRanges(getBM(attributes=c('chromosome_name', 'start_position', 'end_position', 'hgnc_symbol'), filters = 'hgnc_symbol', values =gene.symbols, mart = ensembl))
seqlevelsStyle(genes) <- "UCSC"

kp <- plotKaryotype(genome="hg38", chromosomes="autosomal")
kpPlotMarkers(kp, data=genes, labels=genes$hgnc_symbol, #text.orientation = "horizontal",
              r1=0.7, cex=0.8, adjust.label.position = TRUE)
```

## Locus plots (ONLY showing top 10 here)

```{r locus plot, echo=TRUE, eval=TRUE, fig.height=6, fig.width=10}
make_locusplot(finemap_res,
               region_id = "3_47685722_49279539",
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = "cs",
               color_pip_by = "cs")

make_locusplot(finemap_res,
               region_id = "10_101189482_102620653",
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = "cs",
               color_pip_by = "cs")

make_locusplot(finemap_res,
               region_id = "20_25363595_33027017",
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = "cs",
               color_pip_by = "cs")

make_locusplot(finemap_res,
               region_id = "4_93312148_95261037",
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = "cs",
               color_pip_by = "cs")

make_locusplot(finemap_res,
               region_id = "3_153538782_154996429",
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = "cs",
               color_pip_by = "cs")

make_locusplot(finemap_res,
               region_id = "19_35978393_37036131",
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = "cs",
               color_pip_by = "cs")

make_locusplot(finemap_res,
               region_id = "17_82076532_83247139",
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = "cs",
               color_pip_by = "cs")

make_locusplot(finemap_res,
               region_id = "12_108632125_109898914",
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = "cs",
               color_pip_by = "cs")

make_locusplot(finemap_res,
               region_id = "9_33578336_34642246",
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = "cs",
               color_pip_by = "cs")

make_locusplot(finemap_res,
               region_id = "11_46984586_49844498",
               ens_db = ens_db,
               weights = weights,
               highlight_pip = 0.8,
               filter_protein_coding_genes = TRUE,
               filter_cs = TRUE,
               color_pval_by = "cs",
               color_pip_by = "cs")
```

## GSEA-Gene Ontology

```{r pathway setup, echo=TRUE, eval=TRUE, warning=FALSE, message=FALSE}
dbs <- c("GO_Biological_Process_2021", "GO_Cellular_Component_2021", "GO_Molecular_Function_2021")
genes <- combined_pip_by_group$gene_name[combined_pip_by_group$combined_pip >0.8]

#number of genes for gene set enrichment
sprintf("The number of genes used in enrichment analysis = %s", length(genes))
```

```{r GO enrichment, echo=TRUE, eval=TRUE, warning=FALSE, message=FALSE}
GO_enrichment <- enrichr(genes, dbs)
```

```{r figure title1, echo=TRUE, eval=TRUE}
print("GO_Biological_Process_2021")
```

```{r GO_Biological_Process_2021, echo=TRUE, eval=TRUE}
db <- "GO_Biological_Process_2021"
df <- GO_enrichment[[db]]
print(plotEnrich(GO_enrichment[[db]]))
```

```{r echo=TRUE, eval=TRUE}
df <- df[df$Adjusted.P.value<0.05,c("Term", "Overlap", "Adjusted.P.value", "Genes")]

DT::datatable(df,caption = htmltools::tags$caption( style = 'caption-side: topleft; text-align = left; color:black;','Enriched pathways from GO_Biological_Process_2021'),options = list(pageLength = 5) )
```

```{r figure title2, echo=TRUE, eval=TRUE}
print("GO_Cellular_Component_2021")
```

```{r GO_Cellular_Component_2021, echo=TRUE, eval=TRUE}
db <- "GO_Cellular_Component_2021"
df <- GO_enrichment[[db]]
print(plotEnrich(GO_enrichment[[db]]))
```

```{r echo=TRUE, eval=TRUE}
df <- df[df$Adjusted.P.value<0.05,c("Term", "Overlap", "Adjusted.P.value", "Genes")]

DT::datatable(df,caption = htmltools::tags$caption( style = 'caption-side: topleft; text-align = left; color:black;','Enriched pathways from GO_Cellular_Component_2021'),options = list(pageLength = 5) )
```

```{r figure title3, echo=TRUE, eval=TRUE}
print("GO_Molecular_Function_2021")
```

```{r GO_Molecular_Function_2021, echo=TRUE, eval=TRUE}
db <- "GO_Molecular_Function_2021"
df <- GO_enrichment[[db]]
print(plotEnrich(GO_enrichment[[db]]))
```

```{r echo=TRUE, eval=TRUE}
df <- df[df$Adjusted.P.value<0.05,c("Term", "Overlap", "Adjusted.P.value", "Genes")]

DT::datatable(df,caption = htmltools::tags$caption( style = 'caption-side: topleft; text-align = left; color:black;','Enriched pathways from GO_Molecular_Function_2021'),options = list(pageLength = 5) )
```
